---
title: "Hölzer Lab - Home"
layout: homelay
excerpt: "Hölzer Lab - Bioinformatics."
sitemap: false
permalink: /
---

We are a fresh, small bioinformatics group at the [FSU Jena](https://www.uni-jena.de/en/start.html?). 

We focus on the development of algorithms and tools for handling large sequencing data sets. We apply our computational methods to *de novo* genome and transcriptome assembly, the comprehensive analysis of RNA-Seq data, and the identification of pathogens such as viruses.

We strive to combine well-established and novel computational approaches to analyze and visualize large amounts of 
short- and long-read sequencing data.
<!-- in a comprehensive and handsome way to allow other researchers to understand, interpret and dilate the data.-->

We work closely with the [RNA Bioinformatics/High-Throughput](https://www.rna.uni-jena.de/) group of Manja Marz and we are a member of the [European Virus Bioinformatics Center](http://evbc.uni-jena.de/). 

<br>
#### Research Interests

* Bioinformatics
* Transcriptomics & comparative genomics
* _de novo_ assembly & annotation
* Microbes & viruses
* Metagenomics & -transcriptomics
* Oxford Nanopore Sequencing
* Non-coding RNA & RNA secondary structure
* Automation & visualization

[__A paper per day challenge__](https://hoelzer-lab.github.io/papers)

<!--
<br><br>
![](/images/black_box.svg)
![](/images/gatc_resistance.png)-->


<br><br><br><br><br><br>
[![](/images/logos.svg)](/support)


<!--
We apply our computational methods to de novo genome assembly, cancer genomics and (most recently) infectious disease. We collaborate closely with [Nick Loman's group at the University of Birmingham](http://lab.loman.net/), [Winston Timp's lab at Johns Hopkins University](http://www.timplab.org/) and [Conrad Nieduszynski at the University of Oxford](http://www.path.ox.ac.uk/content/conrad-nieduszynski).
-->