---
title: "Hölzer Team - Home"
layout: homelay
excerpt: "Hölzer Team - Bioinformatics."
sitemap: false
permalink: /
---

In 2018, we formed a bioinformatics team focusing on microbial transcriptomics and metagenomics in collaboration with the [RNA Bioinformatics](https://www.rna.uni-jena.de/) group at the [FSU Jena](https://www.uni-jena.de/en/start.html?).

With Martin's move to the [bioinformatics department of the Robert Koch Institute](https://www.rki.de/EN/Content/Institute/DepartmentsUnits/MF/MF1/mf1_node.html) in 2020, the team will no longer exist in its original form, but the science will continue at the RKI. 

We develop novel tools and workflows for handling large amounts of sequence data. We apply computational methods to automate *de novo* (meta-)genome and transcriptome assembly, analysis and annotation of RNA-Seq data, identification and characterization of pathogens such as viruses and bacteria from environmental samples and (more recently) molecular monitoring of infectious diseases.

We strive to combine well-established and novel computational approaches into reproducible workflows to analyze and visualize large amounts of short- and long-read sequencing data.
<!-- in a comprehensive and handsome way to allow other researchers to understand, interpret and dilate the data.-->

<!--We work closely with the [RNA Bioinformatics/High-Throughput](https://www.rna.uni-jena.de/) group of Manja Marz and We are a member of the [European Virus Bioinformatics Center](http://evbc.uni-jena.de/). -->

<br>
<div class="row"><div class="col-sm-6 clearfix">

#### Research interests

* Microbial bioinformatics
* _de novo_ assembly & annotation
* Transcriptomics & comparative genomics
* Metagenomics & -transcriptomics
* Real-time nanopore sequencing
* Workflow management & reproducibility
* Automation & visualization

</div><div class="col-sm-6 clearfix">
<!--[![GR cover art](/images/gr_cover.gif)](https://genome.cshlp.org/content/29/9.cover-expansion)-->
[![ONT hybrid MAGs](/images/hybrid.svg)](https://www.biorxiv.org/content/10.1101/2019.12.18.880807v1)
</div></div>


<!--[__A paper per day challenge__](https://hoelzer-lab.github.io/papers)-->

<!--
<br><br>
![](/images/black_box.svg)
![](/images/gatc_resistance.png)-->


<br><br><br><br><br><br>
[![](/images/logos.svg)](/support)


<!--
We apply our computational methods to de novo genome assembly, cancer genomics and (most recently) infectious disease. We collaborate closely with [Nick Loman's group at the University of Birmingham](http://lab.loman.net/), [Winston Timp's lab at Johns Hopkins University](http://www.timplab.org/) and [Conrad Nieduszynski at the University of Oxford](http://www.path.ox.ac.uk/content/conrad-nieduszynski).
-->