<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html>
  <head>

    <style type='text/css'>
  	details{
		cursor:pointer;
		}
    </style>    
    
    <title>The dark art of de novo transcriptome assembly: a comprehensive comparison</title>
    <link rel="stylesheet" href="../src/css/style.css" type="text/css" media="print, projection, screen" />
    <link rel="stylesheet" href="../src/css/jq.css" type="text/css" media="print, projection, screen" />
    
    <script type="text/javascript" src="../src/jquery/jquery-latest.js"></script> 
    <script type="text/javascript" src="../src/jquery/jquery.tablesorter.js"></script>
    <script type="text/javascript" src="../src/jquery/jquery.tablesorter.staticrow.min.js"></script>

    <script src="http://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.14/require.min.js"></script> <!--NYAPLOT-->

		<script type="text/javascript" src="src/details-shim-master/details-shim.min.js">
		</script><link rel="stylesheet" type="text/css" href="src/details-shim-master/details-shim.min.css">
    
    <script type="text/javascript" language="javascript" class="init">
   if(window['d3'] === undefined ||
      window['Nyaplot'] === undefined){
       var path = {"d3":"http://d3js.org/d3.v3.min","downloadable":"http://cdn.rawgit.com/domitry/d3-downloadable/master/d3-downloadable"};

       var shim = {"d3":{"exports":"d3"},"downloadable":{"exports":"downloadable"}};

       require.config({paths: path, shim:shim});

   require(['d3'], function(d3){window['d3']=d3;console.log('finished loading d3');require(['downloadable'], function(downloadable){window['downloadable']=downloadable;console.log('finished loading downloadable');

	var script = d3.select("head")
	    .append("script")
	    .attr("src", "http://cdn.rawgit.com/domitry/Nyaplotjs/master/release/nyaplot.js")
	    .attr("async", true);

	script[0][0].onload = script[0][0].onreadystatechange = function(){

	    var event = document.createEvent("HTMLEvents");
	    event.initEvent("load_nyaplot",false,false);
	    window.dispatchEvent(event);
	    console.log('Finished loading Nyaplotjs');
	};
});});
}

	$(function() {
	  $('a[href*=#]:not([href=#])').click(function() {
	    if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
	      var target = $(this.hash);
	      target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
	      if (target.length) {
	        $('html,body').animate({
	          scrollTop: target.offset().top
	        }, 500);
	        return false;
	      }
	    }
	  });
	});

$(document).ready(function() {

		var headers = {},
		$table = $('#samplesTab'), i,
		l = $table.find('thead th').length,
		sortcolumns = [];
		for (i = 0; i < l; i++) {
			if ($.inArray(i, sortcolumns) < 0) {
				headers[i] = { sorter: false }
			}
		}
		$("#samplesTab").tablesorter({widgets: ['zebra']});
		$("#dataTab").tablesorter({widgets: ['zebra']});
$("#assemblerTab").tablesorter({widgets: ['zebra']});
$("#executionTab").tablesorter({widgets: ['zebra']});
$("#hisatTab").tablesorter({widgets: ['zebra']});
$("#quastTab").tablesorter({widgets: ['zebra']});
$("#transrateTab").tablesorter({widgets: ['zebra']});
$("#exn50Tab").tablesorter({widgets: ['zebra']});
$("#buscoTab").tablesorter({widgets: ['zebra']});
$("#detonateTab1").tablesorter({widgets: ['zebra']});
$("#detonateTab2").tablesorter({widgets: ['zebra']});
$("#detonateTab3").tablesorter({widgets: ['zebra']});
$("#detonateTab4").tablesorter({widgets: ['zebra']});
$("#detonateTab5").tablesorter({widgets: ['zebra']});
$("#detonateTab6").tablesorter({widgets: ['zebra']});
$("#detonateTab7").tablesorter({widgets: ['zebra']});
$("#detonateTab8").tablesorter({widgets: ['zebra']});
$("#detonateTab9").tablesorter({widgets: ['zebra']});
$("#metricsTab1").tablesorter({widgets: ['zebra']});
$("#metricsTab2").tablesorter({widgets: ['zebra']});
$("#metricsTab3").tablesorter({widgets: ['zebra']});
$("#metricsTab4").tablesorter({widgets: ['zebra']});
$("#metricsTab5").tablesorter({widgets: ['zebra']});
$("#metricsTab6").tablesorter({widgets: ['zebra']});
$("#metricsTab7").tablesorter({widgets: ['zebra']});
$("#metricsTab8").tablesorter({widgets: ['zebra']});
$("#metricsTab9").tablesorter({widgets: ['zebra']});
$("#runtimeTab").tablesorter({widgets: ['zebra']});
$("#normscoreTab").tablesorter({widgets: ['zebra']});
<!--PLACEHOLDER-->
});
</script>
</head>
<body>
<div id="banner">
<h1 id="top"><em>De novo transcriptome assembly</em>: A comprehensive cross-species comparison of<br>short-read RNA-Seq assemblers</h1>
<h2>Supplementary Data</b>
<h3>Martin H&ouml;lzer and Manja Marz</h3>
</div>


<div id="main"><a name="contents"></a>
<h1>Contents</h1>
<ul><li>S1: <a href="#data">Data sets and preprocessing</a></li> <li>S2: <a href="#assembler">Assembly tools</a></li> <li>S3: <a href="#execution">Executed assembly commands</a></li> <li>S4: <a href="#hisat">HISAT2 re-mapping rate</a></li> <li>S5: <a href="#quast">RNAQuast statistics</a></li> <li>S6: <a href="#transrate">TransRate</a></li> <li>S7: <a href="#exn50">ExN50</a></li> <li>S8: <a href="#busco">BUSCO</a></li> <li>S9: <a href="#detonate">DETONATE</a></li> <li>S10: <a href="#metrics">Selected main metrics</a></li> <li>S11: <a href="#runtime">Runtime and memory consumption</a></li> <li>S12: <a href="#normscore">(0,1)-normalized scores per data set and metric</a></li> </ul><br><br><p>All <i>big</i> data files (processed read data, assemblies, ...) as well as execution commands and Blast results are also available at the 
    <a href="https://doi.org/10.17605/OSF.IO/5ZDX4">Open Science Framework</a> under accession <a href="https://doi.org/10.17605/OSF.IO/5ZDX4">doi.org/10.17605/OSF.IO/5ZDX4</a>.</p><a name="data">
<h1>S1: Data sets and preprocessing</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>Overview of all data sets used for assembly. All reads were quality checked with FASTQC and Prinseq prior assembly. Shown are read statistics before and after trimming. If a data set was sequenced by preserving the strand specificity (ss), this is indicated by the direction of the sequenced read (pairs): F -- forward, R -- reverse.</p>
<table id="dataTab" class="tablesorter" border="0" cellpadding="0" cellspacing="1">
<thead>
<tr>
<th>ID</th><th>Species</th><th>Domain</th><th>Encoding</th><th>Type</th><th>Stranded</th><th># Reads (raw)</th><th>Read length (raw)</th><th>%GC (raw)</th><th>FastQC (raw)</th><th># Reads (trimmed)</th><th>Read length (trimmed)</th><th>%GC (trimmed)</th><th>FastQC (trimmed)</th></tr></thead><tbody><tr><td>ECO</td><td>Escherichia coli</td><td>Bacteria</td><td> Illumina 1.9</td><td>single-end</td><td>ss F</td><td>7943130</td><td>94</td><td>40%</td><td><a href="data/fastqc/raw/eco/eco_fastqc.html">SINGLE</a></td><td>6643709</td><td>25-94</td><td>50%</td><td><a href="data/fastqc/trimmed/eco/eco_fastqc.html">SINGLE</a></td></tr>
<tr><td>CAL3</td><td>Candida albicans</td><td>Fungi</td><td> Illumina 1.9</td><td>paired-end</td><td>not</td><td>11576932</td><td>51</td><td>38%</td><td><a href="data/fastqc/raw/cal3/cal3_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/cal3/cal3_r2_fastqc.html">REVERSE</a></td><td>11168529</td><td>25-51</td><td>38%</td><td><a href="data/fastqc/trimmed/cal3/cal3_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/cal3/cal3_2_fastqc.html">REVERSE</a></td></tr>
<tr><td>ATH</td><td>Arabidopsis thaliana</td><td>Plant</td><td> Illumina 1.9</td><td>single-end</td><td>not</td><td>16911774</td><td>30-101</td><td>46%</td><td><a href="data/fastqc/raw/ath/ath_fastqc.html">SINGLE</a></td><td>16828036</td><td>25-101</td><td>46%</td><td><a href="data/fastqc/trimmed/ath/ath_fastqc.html">SINGLE</a></td></tr>
<tr><td>MMU</td><td>Mus musculus</td><td>Mammal</td><td> Illumina 1.9</td><td>paired-end</td><td>ss FR</td><td>52645238</td><td>76</td><td>48%</td><td><a href="data/fastqc/raw/mmu/mmu_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/mmu/mmu_r2_fastqc.html">REVERSE</a></td><td>43321537</td><td>25-76</td><td>48%</td><td><a href="data/fastqc/trimmed/mmu/mmu_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/mmu/mmu_2_fastqc.html">REVERSE</a></td></tr>
<tr><td>HSA</td><td>Homo sapiens</td><td>Mammal</td><td> Illumina 1.9</td><td>paired-end</td><td>ss FR</td><td>97548052</td><td>101</td><td>53%</td><td><a href="data/fastqc/raw/hsa/hsa_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/hsa/hsa_r2_fastqc.html">REVERSE</a></td><td>96093116</td><td>25-101</td><td>52%</td><td><a href="data/fastqc/trimmed/hsa/hsa_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/hsa/hsa_2_fastqc.html">REVERSE</a></td></tr>
<tr><td>EBOV_HSA_3H</td><td>Homo sapiens + EBOV 3h</td><td>Mammal+Virus</td><td> Illumina 1.9</td><td>paired-end</td><td>not</td><td>17203785</td><td>20-100</td><td>48%</td><td><a href="data/fastqc/raw/ebov_hsa_3h/ebov_hsa_3h_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/ebov_hsa_3h/ebov_hsa_3h_r2_fastqc.html">REVERSE</a></td><td>15649763</td><td>25-100</td><td>47%</td><td><a href="data/fastqc/trimmed/ebov_hsa_3h/ebov_hsa_3h_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/ebov_hsa_3h/ebov_hsa_3h_2_fastqc.html">REVERSE</a></td></tr>
<tr><td>EBOV_HSA_7H</td><td>Homo sapiens + EBOV 7h</td><td>Mammal+Virus</td><td> Illumina 1.9</td><td>paired-end</td><td>not</td><td>24688523</td><td>20-100</td><td>47%</td><td><a href="data/fastqc/raw/ebov_hsa_7h/ebov_hsa_7h_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/ebov_hsa_7h/ebov_hsa_7h_r2_fastqc.html">REVERSE</a></td><td>22268601</td><td>25-100</td><td>47%</td><td><a href="data/fastqc/trimmed/ebov_hsa_7h/ebov_hsa_7h_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/ebov_hsa_7h/ebov_hsa_7h_2_fastqc.html">REVERSE</a></td></tr>
<tr><td>EBOV_HSA_23H</td><td>Homo sapiens + EBOV 23h</td><td>Mammal+Virus</td><td> Illumina 1.9</td><td>paired-end</td><td>not</td><td>26470155</td><td>20-100</td><td>46%</td><td><a href="data/fastqc/raw/ebov_hsa_23h/ebov_hsa_23h_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/ebov_hsa_23h/ebov_hsa_23h_r2_fastqc.html">REVERSE</a></td><td>23930250</td><td>25-100</td><td>46%</td><td><a href="data/fastqc/trimmed/ebov_hsa_23h/ebov_hsa_23h_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/ebov_hsa_23h/ebov_hsa_23h_2_fastqc.html">REVERSE</a></td></tr>
<tr><td>HSA_FLUX</td><td>Homo sapiens simulated</td><td>Mammal-simulated</td><td> Illumina 1.9</td><td>paired-end</td><td>not</td><td>60002178</td><td>3-100</td><td>50%</td><td><a href="data/fastqc/raw/hsa_flux/hsa_flux_r1_fastqc.html">FORWARD</a> | <a href="data/fastqc/raw/hsa_flux/hsa_flux_r2_fastqc.html">REVERSE</a></td><td>56962181</td><td>25-100</td><td>49%</td><td><a href="data/fastqc/trimmed/hsa_flux/hsa_flux_1_fastqc.html">FORWARD</a> | <a href="data/fastqc/trimmed/hsa_flux/hsa_flux_2_fastqc.html">REVERSE</a></td></tr>
</tbody></table><br><br><br>

<a name="assembler">
<h1>S2: Assembly tools</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>Overview about the evaluated assembly tools. We obtained
the most recent versions in November 2018. We chose Oases and Trans-ABySS as two transcriptome
assemblers build on top of the genome assemblers Velvet and ABySS, respectively. To achieve an
assembly with Oases or Trans-ABySS one first has to run the underlying genome assembly tool
with a range of different k-mers. SOAPdenovo-Trans, build on the principles of SOAPdenovo2,
can be used as a stand alone tool for <i>de novo</i> transcriptome assembly based on single k-mer values.
Trinity was also especially designed for transcriptome assembly, using only a fixed k-mer value of 25.
All these four transcriptome assemblers are designed in the scope of working with RNA-Seq data
and are based on de Bruijn graph algorithms. In contrast the Mira assembler works on overlap-
graphs and therefore uses no k-mer approach to decompose reads before assembly. It can be run
in EST mode to deal with the special requirements of RNA-Seq data. Next to this transcriptome
assemblers we also chose one <i>de novo</i> genome assembly tool, SPAdes, originally developed for
smaller bacterial-size genomes and single-cell data and also based on de Bruijn graph and multiple
k-mer values. MK – Yes, if the tool has an build in multiple kmer approach and automatically
merges the output of different kmer runs. 
<sup>a</sup>Oases was used on top of the <i>de novo</i> genome assembler Velvet (v1.2.10).
<sup>b</sup>SPAdes, originally designed as a <i>de novo</i> genome
assembler for single-cell data, was used in RNA-Seq modus (-rna) and single-cell modus (-sc),
respectively. 
<sup>c</sup>When running SPAdes in RNA-Seq modus, two k-mer values are used.</p><table id="assemblerTab" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead>
<tr>
<th>Assembler</th><th>Version</th><th>Multiple k-mer mode</th><th>PMID</th></tr></thead><tbody>
<tr><td>Trinity</td><td>2.8.4</td><td>no</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21572440">21572440</a></td></tr>
<tr><td>Trans-ABySS</td><td>2.0.1</td><td>yes</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20935650">20935650</a></td></tr>
<tr><td>SOAPdenovo-Trans</td><td>1.03</td><td>no</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24532719">24532719</a></td></tr>
<tr><td>Oases<sup>a</sup></td><td>0.2.08</td><td>yes</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22368243">22368243</a></td></tr>
<tr><td>IDBA-Tran</td><td>1.1.1</td><td>yes</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23813001">23813001</a></td></tr>
<tr><td>Bridger</td><td>v2014-12-01</td><td>no</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25723335">25723335</a></td></tr>
<tr><td>BinPacker</td><td>1.0</td><td>no</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26894997">26894997</a></td></tr>
<tr><td>Shannon</td><td>0.0.2</td><td>no</td><td><a href="http://biorxiv.org/content/early/2016/02/09/039230">bioRxiv</a></td></tr>
<tr><td>SPAdes-sc<sup>b</sup></td><td>3.13.0</td><td>yes</td><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22506599">22506599</a></td></tr>
<tr><td>SPAdes-rna<sup>b</sup></td><td>3.13.0</td><td>yes<sup>c</sup></td><td><a href="https://www.biorxiv.org/content/early/2018/09/18/420208">bioRxiv</a></td></tr>
</tbody></table><br><br><br>

<a name="execution">
<h1>S3: Executed assembly commands</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>In the following we provide all executed commands that were used to run the different assemblies.
Each assembler listed in <a href="#assembler">Tab. S2</a> was run on each data set listed in <a href="#data">Tab. S1</a>. For details like used k-mers and strand-specificity download the readme files or expand the view for the corresponding data set.
 The command files still include the execution command of MIRA, however the output was not used in the comparison because MIRA performed worse on all data sets (very short contigs, almost no homology to known transcripts, bad re-mapping rate, ... see manuscript).</p><details><summary>Assembly of Escherichia coli <a href="data/commands/eco/eco.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## Eco1 1x 94bp, strand specific F
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=eco
</br>
R1=eco1_formated.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=35
</br>
kmer3=45
</br>
kmer4=55
</br>
kmer5=65
</br>

</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --single $R1 --CPU $THREADS --output $DIR/trinity/ --SS_lib_type F
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --se $R1 --SS --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink $kmer1 --maxk $kmer5 --SS --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fa
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/$PROJECT.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=94" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=94" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q="$R1 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with default settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/$PROJECT.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades
</br>
rnaspades.py --s1 $R1 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc --s1 $R1 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=10
</br>
type=-short
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 -strand_specific " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --filter $R1 $DIR/idba-tran/r1.fasta
</br>
idba_tran -r $DIR/idba-tran/r1.fasta -o $DIR/idba-tran/ --mink $kmer1 --maxk $kmer5 --step 10 --num_threads $THREADS
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fa
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --single $R1 --SS_lib_type F --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --single $R1 --SS_lib_type F --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
cd $OUT
</br>
BinPacker -s fq -p single -u $R1 -m F -o $OUT
</br>
cd ..
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --single $R1 --ss
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = se" >> $DIR/mira/manifest.config
</br>
echo "data = $R1" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
#echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
ln -s $DIR/mira/"$PROJECT"_assembly/"$PROJECT"_d_results/"$PROJECT"_out.unpadded.fasta $FINAL_DIR/mira.fasta
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Candida albicans <a href="data/commands/cal3/cal3.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## CAL,  2x 51bp not strand-specific
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=cal3
</br>
R1=cal3_1_formated.fastq
</br>
R2=cal3_2_formated.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=21
</br>
kmer2=27
</br>
kmer3=33
</br>
kmer4=39
</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
transabyss-merge --threads $THREADS --mink $kmer1 --maxk $kmer4 --SS --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/test.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=51" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=51" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/test.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>

</br>
##ATTENTION CHANGED KMER BECAUS DEFAULT 55 IS TO BIG
</br>
mkdir -p $DIR/rnaspades
</br>
rnaspades.py -k 25 --pe1-1 $R1 --pe1-2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc --pe1-1 $R1 --pe1-2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
##################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer4 + 1))
</br>
k_step=6
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink $kmer1 --maxk $kmer4 --step 6 --num_threads $THREADS
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -o $OUT
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
ln -s $DIR/mira/"$PROJECT"_assembly/"$PROJECT"_d_results/"$PROJECT"_out.unpadded.fasta $FINAL_DIR/mira.fasta
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Arabidopsis thaliana <a href="data/commands/ath/ath.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## Arabidopsis 1x 100bp, not strand specific
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=ath
</br>
R1=ath_formated.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=35
</br>
kmer3=45
</br>
kmer4=55
</br>
kmer5=65
</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --single $R1 --CPU $THREADS --output $DIR/trinity/
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --se $R1 --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink $kmer1 --maxk $kmer5 --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/$PROJECT.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=100" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=100" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq single end files" >> $CONFIG
</br>
echo "q="$R1 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with default settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/$PROJECT.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades
</br>
rnaspades.py --s1 $R1 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc --s1 $R1 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=10
</br>
type=-short
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --filter $R1 $DIR/idba-tran/r1.fasta
</br>
idba_tran -r $DIR/idba-tran/r1.fasta -o $DIR/idba-tran/ --mink $kmer1 --maxk $kmer5 --step 10 --num_threads $THREADS
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --single $R1 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --single $R1 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
cd $OUT
</br>
BinPacker -s fq -p single -u $R1 -o $OUT
</br>
cd ..
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --single $R1
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = se" >> $DIR/mira/manifest.config
</br>
echo "data = $R1" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
ln -s $DIR/mira/"$PROJECT"_assembly/"$PROJECT"_d_results/"$PROJECT"_out.unpadded.fasta $FINAL_DIR/mira.fasta
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Mus musculus <a href="data/commands/mmu/mmu.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## MMU TRINITY TEST DATA, 2x 76bp, strand specific RF
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=mmu
</br>
R1=mmu_1_formated.fastq
</br>
R2=mmu_2_formated.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=35
</br>
kmer3=45
</br>
kmer4=55
</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/ --SS_lib_type FR
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --SS --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
transabyss-merge --threads $THREADS --mink $kmer1 --maxk $kmer4 --SS --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/test.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=76" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=76" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=1" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer4
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/test.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades
</br>
rnaspades.py --pe1-1 $R1 --pe1-2 $R2 --pe1-fr -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc --pe1-1 $R1 --pe1-2 $R2 --pe1-fr -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
##################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer4 + 1))
</br>
k_step=10
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 -strand_specific -separate " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
R1=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/mmu_1_formated_reversed.fastq
</br>
R2=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/mmu_2_formated_reversed.fastq
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink $kmer1 --maxk $kmer4 --step 10 --num_threads $THREADS
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
R1=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/mmu_1_formated.fastq
</br>
R2=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/mmu_2_formated.fastq
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
#Error! try unstranded mode!
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -m RF -o $OUT
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2 --ss
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
#    if @type == :SE
</br>
#      config << "# defining illumina single end reads"
</br>
#      config << "readgroup = se"
</br>
#      config << "data = #{@reads}"
</br>
#      config << "technology = #{@technology}"
</br>
#    end
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
#echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = <--- --->" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
ln -s $DIR/mira/"$PROJECT"_assembly/"$PROJECT"_d_results/"$PROJECT"_out.unpadded.fasta $FINAL_DIR/mira.fasta
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Homo sapiens <a href="data/commands/hsa/hsa.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## HSA SRA, 2x 100bp, strand specific RF
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=hsa
</br>
R1=hsa_1.fastq
</br>
R2=hsa_2.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=35
</br>
kmer3=45
</br>
kmer4=55
</br>
kmer5=65
</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/ --SS_lib_type FR
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --SS --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink $kmer1 --maxk $kmer5 --SS --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/test.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=100" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=100" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=1" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/test.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades
</br>
rnaspades.py --pe1-1 $R1 --pe1-2 $R2 --pe1-fr -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc --pe1-1 $R1 --pe1-2 $R2 --pe1-fr -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=10
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 -strand_specific -separate " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
R1=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/hsa_1_reversed.fastq
</br>
R2=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/hsa_2_reversed.fastq
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink $kmer1 --maxk $kmer5 --step 10 --num_threads $THREADS
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
R1=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/hsa_1.fastq
</br>
R2=/mnt/dessertlocal/projects/transcriptome_assembly/data/prinseq/hsa_2.fastq
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
# Error, try unstranded
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -m RF -o $OUT
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python /mnt/prostlocal/programs/shannon/0.0.2/shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2 --ss
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
#    if @type == :SE
</br>
#      config << "# defining illumina single end reads"
</br>
#      config << "readgroup = se"
</br>
#      config << "data = #{@reads}"
</br>
#      config << "technology = #{@technology}"
</br>
#    end
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
#echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = <--- --->" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
ln -s $DIR/mira/"$PROJECT"_assembly/"$PROJECT"_d_results/"$PROJECT"_out.unpadded.fasta $FINAL_DIR/mira.fasta
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Homo sapiens + EBOV 3h <a href="data/commands/ebov_hsa_3h/ebov_hsa_3h.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## EBOV HSA 3h
</br>
## paired-end, not strand specific, 100bp
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=ebov_hsa_3h
</br>
R1=ebov_hsa_3h_1.fastq
</br>
R2=ebov_hsa_3h_2.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=29
</br>
kmer3=33
</br>
kmer4=37
</br>
kmer5=41
</br>

</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink 25 --maxk 41 --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/test.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=100" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=100" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/test.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades/
</br>
rnaspades.py -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=4
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 -separate " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink 25 --maxk 41 --step 4
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
cd $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -o $DIR/binpacker
</br>
cd ..
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
#    if @type == :SE
</br>
#      config << "# defining illumina single end reads"
</br>
#      config << "readgroup = se"
</br>
#      config << "data = #{@reads}"
</br>
#      config << "technology = #{@technology}"
</br>
#    end
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
###################################################################################
</br>

</br>
</br></br></code></details>
<details><summary>Assembly of Homo sapiens + EBOV 7h <a href="data/commands/ebov_hsa_7h/ebov_hsa_7h.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## EBOV HSA 7h
</br>
## paired-end, not strand specific, 100bp
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=ebov_hsa_7h
</br>
R1=ebov_hsa_7h_1.fastq
</br>
R2=ebov_hsa_7h_2.fastq
</br>
THREADS=48
</br>
MEM=400
</br>

</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=29
</br>
kmer3=33
</br>
kmer4=37
</br>
kmer5=41
</br>

</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink 25 --maxk 41 --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/test.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=100" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=100" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/test.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades/
</br>
rnaspades.py -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=4
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 -separate " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink 25 --maxk 41 --step 4
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
cd $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -o $DIR/binpacker
</br>
cd ..
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
#    if @type == :SE
</br>
#      config << "# defining illumina single end reads"
</br>
#      config << "readgroup = se"
</br>
#      config << "data = #{@reads}"
</br>
#      config << "technology = #{@technology}"
</br>
#    end
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Homo sapiens + EBOV 23h <a href="data/commands/ebov_hsa_23h/ebov_hsa_23h.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## EBOV HSA 23h
</br>
## paired-end, not strand specific, 100bp
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=ebov_hsa_23h
</br>
R1=ebov_hsa_23h_1.fastq
</br>
R2=ebov_hsa_23h_2.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=29
</br>
kmer3=33
</br>
kmer4=37
</br>
kmer5=41
</br>

</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink 25 --maxk 41 --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/test.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=100" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=100" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/test.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades/
</br>
rnaspades.py -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=4
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 -separate " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink 25 --maxk 41 --step 4
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
cd $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -o $DIR/binpacker
</br>
cd ..
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
#    if @type == :SE
</br>
#      config << "# defining illumina single end reads"
</br>
#      config << "readgroup = se"
</br>
#      config << "data = #{@reads}"
</br>
#      config << "technology = #{@technology}"
</br>
#    end
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
###################################################################################
</br>
</br></br></code></details>
<details><summary>Assembly of Homo sapiens simulated <a href="data/commands/hsa_flux/hsa_flux.readme.sh">(download script)</a> </summary><code></br>#######################################
</br>
## HSA FLUX SIMULATED, 2x100bp, not strand specific
</br>
#######################################
</br>

</br>
##PARAMETERS SHARED
</br>
PROJECT=hsa_flux
</br>
R1=hsa_flux_1.fastq
</br>
R2=hsa_flux_2.fastq
</br>
THREADS=48
</br>
MEM=400
</br>
DIR=~/$PROJECT
</br>
FINAL_DIR=~/$PROJECT/final
</br>

</br>
mkdir -p $DIR
</br>
mkdir -p $FINAL_DIR
</br>

</br>
kmer1=25
</br>
kmer2=35
</br>
kmer3=45
</br>
kmer4=55
</br>
kmer5=65
</br>

</br>

</br>
###################################################################################
</br>
## TRINITY
</br>
mkdir -p $DIR/trinity
</br>
Trinity --seqType fq --max_memory $MEM"G" --left $R1 --right $R2 --CPU $THREADS --output $DIR/trinity/
</br>
ln -s $DIR/trinity/Trinity.fasta $FINAL_DIR/trinity.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## TRANS-ABYSS
</br>
mkdir -p $DIR/transabyss
</br>

</br>
for kmer in $kmer1 $kmer2 $kmer3 $kmer4 $kmer5; do
</br>
	name=k${kmer}
</br>
	assemblydir=$DIR/transabyss/${name}
</br>
  mkdir -p $assemblydir
</br>
	finalassembly=${assemblydir}/${name}-final.fa
</br>
	transabyss -k ${kmer} --pe $R1 $R2 --outdir ${assemblydir} --name ${name} --threads $THREADS
</br>
done
</br>

</br>
mergedassembly=$DIR/transabyss/merged.fa
</br>
finalassembly1=$DIR/transabyss/k"$kmer1"/k"$kmer1"-final.fa
</br>
finalassembly2=$DIR/transabyss/k"$kmer2"/k"$kmer2"-final.fa
</br>
finalassembly3=$DIR/transabyss/k"$kmer3"/k"$kmer3"-final.fa
</br>
finalassembly4=$DIR/transabyss/k"$kmer4"/k"$kmer4"-final.fa
</br>
finalassembly5=$DIR/transabyss/k"$kmer5"/k"$kmer5"-final.fa
</br>
transabyss-merge --threads $THREADS --mink $kmer1 --maxk $kmer5 --prefixes k${kmer1}. k${kmer2}. k${kmer3}. k${kmer4}. k${kmer5}. --out ${mergedassembly} ${finalassembly1} ${finalassembly2} ${finalassembly3} ${finalassembly4} ${finalassembly5}
</br>
ln -s $DIR/transabyss/merged.fa $FINAL_DIR/trans-abyss.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SOAP-TRANS
</br>
mkdir -p $DIR/soap-trans
</br>
CONFIG=$DIR/soap-trans/$PROJECT.config
</br>
touch $CONFIG
</br>
echo '#maximal read length' > $CONFIG
</br>
echo "max_rd_len=100" >> $CONFIG
</br>
echo "[LIB]" >> $CONFIG
</br>
echo "#maximal read length in this lib" >> $CONFIG
</br>
echo "rd_len_cutoff=100" >> $CONFIG
</br>
#echo "#average insert size" >> $CONFIG
</br>
#echo "avg_ins=200" >> $CONFIG
</br>
echo "#if sequence needs to be reversed" >> $CONFIG
</br>
echo "reverse_seq=0" >> $CONFIG
</br>
echo "#in which part(s) the reads are used" >> $CONFIG
</br>
echo "asm_flags=3" >> $CONFIG
</br>
#echo "q=#{@reads}"  >> $CONFIG #if @type == :SE
</br>
echo "#fastq paired end files" >> $CONFIG
</br>
echo "q1="$R1 >> $CONFIG
</br>
echo "q2="$R2 >> $CONFIG
</br>

</br>
kmer_min=$kmer1
</br>
kmer_max=$kmer5
</br>

</br>
# run soap with defualt settings
</br>
mkdir -p $DIR/soap-trans/default
</br>
SOAPdenovo-Trans-127mer all -s $DIR/soap-trans/$PROJECT.config -o $DIR/soap-trans/default/k23 -p $THREADS
</br>
ln -s $DIR/soap-trans/default/k23.scafSeq $FINAL_DIR/soap-trans-default.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## RNA-SPADES
</br>
mkdir -p $DIR/rnaspades
</br>
rnaspades.py -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/rnaspades/
</br>
ln -s $DIR/rnaspades/transcripts.fasta $FINAL_DIR/rna-spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SPADES
</br>
mkdir -p $DIR/spades
</br>
spades.py --sc -1 $R1 -2 $R2 -t $THREADS -m $MEM --cov-cutoff auto -o $DIR/spades/
</br>
ln -s $DIR/spades/scaffolds.fasta $FINAL_DIR/spades.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## OASES
</br>
k_min=$kmer1
</br>
k_max=$((kmer5 + 1))
</br>
k_step=10
</br>
type=-shortPaired
</br>

</br>
mkdir -p $DIR/oases
</br>
oases_pipeline.py -m $k_min -M $k_max -s $k_step -o $DIR/oases/ -d " $type -fastq $R1 $R2 -separate " -c
</br>
ln -s $DIR/oases/Merged/transcripts.fa $FINAL_DIR/oases.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## IDBA-TRANS
</br>
###this tools assume the paired-end reads are in order (->, <-). If your data is in order (<-, ->), please convert it by yourself.
</br>
mkdir $DIR/idba-tran
</br>
fq2fa --merge --filter $R1 $R2 $DIR/idba-tran/r12.fasta
</br>
idba_tran -r $DIR/idba-tran/r12.fasta -o $DIR/idba-tran/ --mink $kmer1 --maxk $kmer5 --step 10 --num_threads $THREADS
</br>
ln -s $DIR/idba-tran/contig.fa $FINAL_DIR/idba-tran.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BRIDGER
</br>
OUT=$DIR/bridger/
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT
</br>
Bridger.pl --seqType fq --left $R1 --right $R2 --CPU $THREADS -o $OUT # start second time, because first run crashs at some point and then finishs
</br>
ln -s $OUT/Bridger.fasta $FINAL_DIR/bridger.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## BINPACKER
</br>
OUT=$DIR/binpacker/
</br>
mkdir -p $OUT
</br>
BinPacker -s fq -p pair -l $R1 -r $R2 -o $OUT
</br>
ln -s $OUT/BinPacker_Out_Dir/BinPacker.fa $FINAL_DIR/binpacker.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## SHANNON
</br>
mkdir $DIR/shannon
</br>
python shannon.py -p $THREADS -o $DIR/shannon --left $R1 --right $R2
</br>
ln -s $DIR/shannon/shannon.fasta $FINAL_DIR/shannon.fasta
</br>
###################################################################################
</br>

</br>
###################################################################################
</br>
## MIRA
</br>
mkdir $DIR/mira
</br>
touch $DIR/mira/manifest.config
</br>
echo "project = $PROJECT" > $DIR/mira/manifest.config
</br>
echo "job = est,denovo,accurate" >> $DIR/mira/manifest.config
</br>
echo "parameters = -DI:trt=/mnt/dessertlocal/projects/transcriptome_assembly/tmp -NW:cnfs=no -NW:cmrnl=no -SK:mmhr=1" >> $DIR/mira/manifest.config
</br>
#    if @type == :SE
</br>
#      config << "# defining illumina single end reads"
</br>
#      config << "readgroup = se"
</br>
#      config << "data = #{@reads}"
</br>
#      config << "technology = #{@technology}"
</br>
#    end
</br>
echo "# defining illumina paired end reads" >> $DIR/mira/manifest.config
</br>
echo "readgroup = pe" >> $DIR/mira/manifest.config
</br>
echo "data = $R1 $R2" >> $DIR/mira/manifest.config
</br>
echo "technology = solexa" >>  $DIR/mira/manifest.config
</br>
echo "template_size = 50 1000 autorefine" >>  $DIR/mira/manifest.config
</br>
echo "segment_placement = ---> <---" >>  $DIR/mira/manifest.config
</br>

</br>
mira -c $DIR/mira -t $THREADS $DIR/mira/manifest.config
</br>
ln -s $DIR/mira/"$PROJECT"_assembly/"$PROJECT"_d_results/"$PROJECT"_out.unpadded.fasta $FINAL_DIR/mira.fasta
</br>
###################################################################################
</br>

</br>

</br>

</br>

</br>

</br>

</br>

</br>
</br></br></code></details>
</br></br></br><a name="hisat">
<h1>S4: HISAT2 re-mapping rate</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>We mapped the quality-controlled reads back to their assembled contigs, to determine the amount of reads that were included in the assembly process by each tool.</p><div id='vis-dc408071-3358-45d5-8963-c3683ef1c5c0'></div>
<script>
(function(){
    var render = function(){
        var model = {"panes":[{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"c76bde32-d51d-4ad0-841f-f043aa711f28"}],"options":{"x_label":"Escherichia coli","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"22dcff1f-a93a-4fee-8a0d-59166b9d98ab"}],"options":{"x_label":"Candida albicans","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"d6214d83-e9b2-4822-a621-308f675e0f8e"}],"options":{"x_label":"Arabidopsis thaliana","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"0fe1b9be-43b4-4e2a-b7aa-9e911524cf4d"}],"options":{"x_label":"Mus musculus","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"a8411362-d806-45ee-99d7-d01dc93afdbf"}],"options":{"x_label":"H. sapiens","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"e3ef5187-e773-46fe-927a-c4f0205ccb60"}],"options":{"x_label":"H. sapiens+EBOV 3h","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"d5f6f14a-5597-40b4-bf7e-7c7876f327e3"}],"options":{"x_label":"H. sapiens+EBOV 7h","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"b91888b2-502e-4fc0-a5e2-89c381b29e35"}],"options":{"x_label":"H. sapiens+EBOV 23h","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"mapping_rate","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"6f76879b-38a1-4e90-8505-e5a34a2e3027"}],"options":{"x_label":"H. sapiens simulated","y_label":"overall mapped reads [%]","legend":true,"yrange":[0,100],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans-ABySS","SOAPdenovo-Trans","Bridger","BinPacker","IDBA-Tran","Shannon","SPAdes-sc","SPAdes-rna"]}}],"data":{"c76bde32-d51d-4ad0-841f-f043aa711f28":[{"tool":"Trinity","mapping_rate":77.01},{"tool":"Oases","mapping_rate":49.16},{"tool":"Trans-ABySS","mapping_rate":95.67},{"tool":"SOAPdenovo-Trans","mapping_rate":56.62},{"tool":"Bridger","mapping_rate":87.35},{"tool":"BinPacker","mapping_rate":71.09},{"tool":"IDBA-Tran","mapping_rate":34.31},{"tool":"Shannon","mapping_rate":76.69},{"tool":"SPAdes-sc","mapping_rate":88.04},{"tool":"SPAdes-rna","mapping_rate":88.9}],"22dcff1f-a93a-4fee-8a0d-59166b9d98ab":[{"tool":"Trinity","mapping_rate":97.32},{"tool":"Oases","mapping_rate":93.61},{"tool":"Trans-ABySS","mapping_rate":98.45},{"tool":"SOAPdenovo-Trans","mapping_rate":95.12},{"tool":"Bridger","mapping_rate":96.72},{"tool":"BinPacker","mapping_rate":96.66},{"tool":"IDBA-Tran","mapping_rate":86.34},{"tool":"Shannon","mapping_rate":96.51},{"tool":"SPAdes-sc","mapping_rate":97.51},{"tool":"SPAdes-rna","mapping_rate":97.43}],"d6214d83-e9b2-4822-a621-308f675e0f8e":[{"tool":"Trinity","mapping_rate":95.71},{"tool":"Oases","mapping_rate":91.93},{"tool":"Trans-ABySS","mapping_rate":97.55},{"tool":"SOAPdenovo-Trans","mapping_rate":85.31},{"tool":"Bridger","mapping_rate":90.64},{"tool":"BinPacker","mapping_rate":67.15},{"tool":"IDBA-Tran","mapping_rate":89.04},{"tool":"Shannon","mapping_rate":51.53},{"tool":"SPAdes-sc","mapping_rate":94.29},{"tool":"SPAdes-rna","mapping_rate":93.67}],"0fe1b9be-43b4-4e2a-b7aa-9e911524cf4d":[{"tool":"Trinity","mapping_rate":92.14},{"tool":"Oases","mapping_rate":89.29},{"tool":"Trans-ABySS","mapping_rate":93.96},{"tool":"SOAPdenovo-Trans","mapping_rate":90.98},{"tool":"Bridger","mapping_rate":91.66},{"tool":"BinPacker","mapping_rate":54.31},{"tool":"IDBA-Tran","mapping_rate":70.64},{"tool":"Shannon","mapping_rate":86.6},{"tool":"SPAdes-sc","mapping_rate":93.83},{"tool":"SPAdes-rna","mapping_rate":96.86}],"a8411362-d806-45ee-99d7-d01dc93afdbf":[{"tool":"Trinity","mapping_rate":91.9},{"tool":"Oases","mapping_rate":88.04},{"tool":"Trans-ABySS","mapping_rate":98.34},{"tool":"SOAPdenovo-Trans","mapping_rate":89.93},{"tool":"Bridger","mapping_rate":86.83},{"tool":"BinPacker","mapping_rate":72.6},{"tool":"IDBA-Tran","mapping_rate":64.61},{"tool":"Shannon","mapping_rate":84.27},{"tool":"SPAdes-sc","mapping_rate":92.04},{"tool":"SPAdes-rna","mapping_rate":95.95}],"e3ef5187-e773-46fe-927a-c4f0205ccb60":[{"tool":"Trinity","mapping_rate":90.73},{"tool":"Oases","mapping_rate":86.92},{"tool":"Trans-ABySS","mapping_rate":93.9},{"tool":"SOAPdenovo-Trans","mapping_rate":89.8},{"tool":"Bridger","mapping_rate":89.95},{"tool":"BinPacker","mapping_rate":36.6},{"tool":"IDBA-Tran","mapping_rate":78.41},{"tool":"Shannon","mapping_rate":90.23},{"tool":"SPAdes-sc","mapping_rate":91.41},{"tool":"SPAdes-rna","mapping_rate":93.83}],"d5f6f14a-5597-40b4-bf7e-7c7876f327e3":[{"tool":"Trinity","mapping_rate":92.58},{"tool":"Oases","mapping_rate":85.78},{"tool":"Trans-ABySS","mapping_rate":94.22},{"tool":"SOAPdenovo-Trans","mapping_rate":90.52},{"tool":"Bridger","mapping_rate":89.14},{"tool":"BinPacker","mapping_rate":84.63},{"tool":"IDBA-Tran","mapping_rate":76.39},{"tool":"Shannon","mapping_rate":90.99},{"tool":"SPAdes-sc","mapping_rate":91.94},{"tool":"SPAdes-rna","mapping_rate":93.4}],"b91888b2-502e-4fc0-a5e2-89c381b29e35":[{"tool":"Trinity","mapping_rate":94.39},{"tool":"Oases","mapping_rate":70.05},{"tool":"Trans-ABySS","mapping_rate":95.49},{"tool":"SOAPdenovo-Trans","mapping_rate":93.1},{"tool":"Bridger","mapping_rate":92.43},{"tool":"BinPacker","mapping_rate":81.57},{"tool":"IDBA-Tran","mapping_rate":48.37},{"tool":"Shannon","mapping_rate":31.29},{"tool":"SPAdes-sc","mapping_rate":91.15},{"tool":"SPAdes-rna","mapping_rate":93.12}],"6f76879b-38a1-4e90-8505-e5a34a2e3027":[{"tool":"Trinity","mapping_rate":96.39},{"tool":"Oases","mapping_rate":73.26},{"tool":"Trans-ABySS","mapping_rate":99.56},{"tool":"SOAPdenovo-Trans","mapping_rate":91.68},{"tool":"Bridger","mapping_rate":94.02},{"tool":"BinPacker","mapping_rate":93.03},{"tool":"IDBA-Tran","mapping_rate":85.34},{"tool":"Shannon","mapping_rate":30.77},{"tool":"SPAdes-sc","mapping_rate":97.02},{"tool":"SPAdes-rna","mapping_rate":96.72}]},"extension":[]}
        var id_name = '#vis-dc408071-3358-45d5-8963-c3683ef1c5c0';
        Nyaplot.core.parse(model, id_name);

        require(['downloadable'], function(downloadable){
          var svg = d3.select(id_name).select("svg");
	  if(!svg.empty())
	    svg.call(downloadable().filename('fig'));
	});
    };
    if(window['Nyaplot']==undefined){
        window.addEventListener('load_nyaplot', render, false);
	return;
    } else {
       render();
    }
})();
</script>
<a name="quast">
<h1>S5: RNAQuast statistics</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>Overview of RNA-Quast statistical output. Please click on the corresponding link to get a short summary of the assembly comparisons or to observe the full output.</p><ul>
<li>Escherichia coli: <a href="data/rna-quast/eco/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/eco/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/eco/index.html">full report + plots (html)</a></li><li>Candida albicans: <a href="data/rna-quast/cal3/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/cal3/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/cal3/index.html">full report + plots (html)</a></li><li>Arabidopsis thaliana: <a href="data/rna-quast/ath/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/ath/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/ath/index.html">full report + plots (html)</a></li><li>Mus musculus: <a href="data/rna-quast/mmu/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/mmu/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/mmu/index.html">full report + plots (html)</a></li><li>Homo sapiens: <a href="data/rna-quast/hsa/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/hsa/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/hsa/index.html">full report + plots (html)</a></li><li>Homo sapiens + EBOV 3h: <a href="data/rna-quast/ebov_hsa_3h/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/ebov_hsa_3h/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/ebov_hsa_3h/index.html">full report + plots (html)</a></li><li>Homo sapiens + EBOV 7h: <a href="data/rna-quast/ebov_hsa_7h/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/ebov_hsa_7h/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/ebov_hsa_7h/index.html">full report + plots (html)</a></li><li>Homo sapiens + EBOV 23h: <a href="data/rna-quast/ebov_hsa_23h/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/ebov_hsa_23h/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/ebov_hsa_23h/index.html">full report + plots (html)</a></li><li>Homo sapiens simulated: <a href="data/rna-quast/hsa_flux/short_report.pdf">short report (pdf)</a> | <a href="data/rna-quast/hsa_flux/short_report.tsv">short report (tsv)</a> | <a href="data/rna-quast/hsa_flux/index.html">full report + plots (html)</a></li></ul>
</br></br><br><a name="transrate">
<h1>S6: TransRate</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>TransRate: reference-free quality assessment of <i>de novo</i> transcriptome assemblies (<a target="blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27252236">27252236</a>)</p><ul>
<li>TransRate results for Escherichia coli: <a href="data/transrate/eco/assemblies.csv">download (CSV)</a> | <a href="data/transrate/eco/index.html">show (HTML)</a></li>
<li>TransRate results for Candida albicans: <a href="data/transrate/cal3/assemblies.csv">download (CSV)</a> | <a href="data/transrate/cal3/index.html">show (HTML)</a></li>
<li>TransRate results for Arabidopsis thaliana: <a href="data/transrate/ath/assemblies.csv">download (CSV)</a> | <a href="data/transrate/ath/index.html">show (HTML)</a></li>
<li>TransRate results for Mus musculus: <a href="data/transrate/mmu/assemblies.csv">download (CSV)</a> | <a href="data/transrate/mmu/index.html">show (HTML)</a></li>
<li>TransRate results for Homo sapiens: <a href="data/transrate/hsa/assemblies.csv">download (CSV)</a> | <a href="data/transrate/hsa/index.html">show (HTML)</a></li>
<li>TransRate results for Homo sapiens + EBOV 3h: <a href="data/transrate/ebov_hsa_3h/assemblies.csv">download (CSV)</a> | <a href="data/transrate/ebov_hsa_3h/index.html">show (HTML)</a></li>
<li>TransRate results for Homo sapiens + EBOV 7h: <a href="data/transrate/ebov_hsa_7h/assemblies.csv">download (CSV)</a> | <a href="data/transrate/ebov_hsa_7h/index.html">show (HTML)</a></li>
<li>TransRate results for Homo sapiens + EBOV 23h: <a href="data/transrate/ebov_hsa_23h/assemblies.csv">download (CSV)</a> | <a href="data/transrate/ebov_hsa_23h/index.html">show (HTML)</a></li>
<li>TransRate results for Homo sapiens simulated: <a href="data/transrate/hsa_flux/assemblies.csv">download (CSV)</a> | <a href="data/transrate/hsa_flux/index.html">show (HTML)</a></li>
</ul></br></br></br>
<a name="exn50">
<h1>S7: ExN50</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>An alternative to the standard contig Nx (e.g. N50) statistic that can be more appropriate for transcriptome assembly is the ExN50 statistic. 
    The N50 statistic is computed as usual but limited to the top most highly expressed transcripts that represent x% of the total normalized expression data. 
    Therefore, we first calculated transcript abundances using the alignment-free quantification tool <a href="https://combine-lab.github.io/salmon/">Salmon</a>
    and subsequently used the <a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki/Transcriptome-Contig-Nx-and-ExN50-stats">Trinity utilities</a> to 
    calculate ExN50 values. We report the <b>Ex90N50</b> metric as one of our main metrics.</p><ul>
<div id='vis-699ad978-f6fb-496e-8725-f7dc8a4ab9e6'></div>
<script>
(function(){
    var render = function(){
        var model = {"panes":[{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"2f2c7da7-c6f7-4c37-b53e-ce2818eaea77"}],"options":{"x_label":"Escherichia coli","y_label":"Ex90N50 value","legend":true,"yrange":[0,742.5000000000001],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"4cc2b2d1-f2db-4653-8472-a885ddf58f42"}],"options":{"x_label":"Candida albicans","y_label":"Ex90N50 value","legend":true,"yrange":[0,2299.0],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"d3ef7210-5778-49bb-a924-87f7c566b4b1"}],"options":{"x_label":"Arabidopsis thaliana","y_label":"Ex90N50 value","legend":true,"yrange":[0,1819.4],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"e31acfed-e4bd-48d1-b4e1-29d14079bc43"}],"options":{"x_label":"Mus musculus","y_label":"Ex90N50 value","legend":true,"yrange":[0,4195.400000000001],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"a76330ce-8cee-46a4-aa06-824130bc0e45"}],"options":{"x_label":"H. sapiens","y_label":"Ex90N50 value","legend":true,"yrange":[0,2619.1000000000004],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"8875f224-033c-4835-afc9-02f4dc136fe5"}],"options":{"x_label":"H. sapiens+EBOV 3h","y_label":"Ex90N50 value","legend":true,"yrange":[0,3883.0000000000005],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"16f64314-e833-4422-b615-d9727b1d2624"}],"options":{"x_label":"H. sapiens+EBOV 7h","y_label":"Ex90N50 value","legend":true,"yrange":[0,4140.400000000001],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"24d14be3-16ba-4a74-ad37-8b5d62e60b61"}],"options":{"x_label":"H. sapiens+EBOV 23h","y_label":"Ex90N50 value","legend":true,"yrange":[0,4361.5],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"ex90n50","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"cf0c8708-5d50-4ab0-8c27-8c8f66041c1d"}],"options":{"x_label":"H. sapiens simulated","y_label":"Ex90N50 value","legend":true,"yrange":[0,3646.5000000000005],"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["Trinity","Oases","Trans_ABySS","SOAPdenovo_Trans","Bridger","BinPacker","IDBA_Tran","Shannon","SPAdes_sc","SPAdes_rna"]}}],"data":{"2f2c7da7-c6f7-4c37-b53e-ce2818eaea77":[{"tool":"Trinity","ex90n50":369},{"tool":"Oases","ex90n50":229},{"tool":"Trans_ABySS","ex90n50":177},{"tool":"SOAPdenovo_Trans","ex90n50":173},{"tool":"Bridger","ex90n50":393},{"tool":"BinPacker","ex90n50":675},{"tool":"IDBA_Tran","ex90n50":304},{"tool":"Shannon","ex90n50":363},{"tool":"SPAdes_sc","ex90n50":111},{"tool":"SPAdes_rna","ex90n50":245}],"4cc2b2d1-f2db-4653-8472-a885ddf58f42":[{"tool":"Trinity","ex90n50":1862},{"tool":"Oases","ex90n50":1588},{"tool":"Trans_ABySS","ex90n50":1664},{"tool":"SOAPdenovo_Trans","ex90n50":1374},{"tool":"Bridger","ex90n50":1923},{"tool":"BinPacker","ex90n50":1876},{"tool":"IDBA_Tran","ex90n50":1257},{"tool":"Shannon","ex90n50":1321},{"tool":"SPAdes_sc","ex90n50":2090},{"tool":"SPAdes_rna","ex90n50":2034}],"d3ef7210-5778-49bb-a924-87f7c566b4b1":[{"tool":"Trinity","ex90n50":1451},{"tool":"Oases","ex90n50":1305},{"tool":"Trans_ABySS","ex90n50":940},{"tool":"SOAPdenovo_Trans","ex90n50":449},{"tool":"Bridger","ex90n50":1636},{"tool":"BinPacker","ex90n50":1452},{"tool":"IDBA_Tran","ex90n50":805},{"tool":"Shannon","ex90n50":1654},{"tool":"SPAdes_sc","ex90n50":1030},{"tool":"SPAdes_rna","ex90n50":1239}],"e31acfed-e4bd-48d1-b4e1-29d14079bc43":[{"tool":"Trinity","ex90n50":420},{"tool":"Oases","ex90n50":2044},{"tool":"Trans_ABySS","ex90n50":530},{"tool":"SOAPdenovo_Trans","ex90n50":2796},{"tool":"Bridger","ex90n50":3100},{"tool":"BinPacker","ex90n50":2715},{"tool":"IDBA_Tran","ex90n50":1672},{"tool":"Shannon","ex90n50":2067},{"tool":"SPAdes_sc","ex90n50":3814},{"tool":"SPAdes_rna","ex90n50":3418}],"a76330ce-8cee-46a4-aa06-824130bc0e45":[{"tool":"Trinity","ex90n50":326},{"tool":"Oases","ex90n50":666},{"tool":"Trans_ABySS","ex90n50":441},{"tool":"SOAPdenovo_Trans","ex90n50":711},{"tool":"Bridger","ex90n50":1370},{"tool":"BinPacker","ex90n50":2381},{"tool":"IDBA_Tran","ex90n50":708},{"tool":"Shannon","ex90n50":1324},{"tool":"SPAdes_sc","ex90n50":1186},{"tool":"SPAdes_rna","ex90n50":782}],"8875f224-033c-4835-afc9-02f4dc136fe5":[{"tool":"Trinity","ex90n50":2753},{"tool":"Oases","ex90n50":2461},{"tool":"Trans_ABySS","ex90n50":1947},{"tool":"SOAPdenovo_Trans","ex90n50":2372},{"tool":"Bridger","ex90n50":2676},{"tool":"BinPacker","ex90n50":3530},{"tool":"IDBA_Tran","ex90n50":1924},{"tool":"Shannon","ex90n50":2236},{"tool":"SPAdes_sc","ex90n50":2735},{"tool":"SPAdes_rna","ex90n50":1603}],"16f64314-e833-4422-b615-d9727b1d2624":[{"tool":"Trinity","ex90n50":2839},{"tool":"Oases","ex90n50":3003},{"tool":"Trans_ABySS","ex90n50":2039},{"tool":"SOAPdenovo_Trans","ex90n50":2385},{"tool":"Bridger","ex90n50":2745},{"tool":"BinPacker","ex90n50":3764},{"tool":"IDBA_Tran","ex90n50":1967},{"tool":"Shannon","ex90n50":2328},{"tool":"SPAdes_sc","ex90n50":2766},{"tool":"SPAdes_rna","ex90n50":1647}],"24d14be3-16ba-4a74-ad37-8b5d62e60b61":[{"tool":"Trinity","ex90n50":2958},{"tool":"Oases","ex90n50":3115},{"tool":"Trans_ABySS","ex90n50":2178},{"tool":"SOAPdenovo_Trans","ex90n50":2489},{"tool":"Bridger","ex90n50":2841},{"tool":"BinPacker","ex90n50":3965},{"tool":"IDBA_Tran","ex90n50":2048},{"tool":"Shannon","ex90n50":1714},{"tool":"SPAdes_sc","ex90n50":3071},{"tool":"SPAdes_rna","ex90n50":2203}],"cf0c8708-5d50-4ab0-8c27-8c8f66041c1d":[{"tool":"Trinity","ex90n50":3234},{"tool":"Oases","ex90n50":3300},{"tool":"Trans_ABySS","ex90n50":2902},{"tool":"SOAPdenovo_Trans","ex90n50":2836},{"tool":"Bridger","ex90n50":3252},{"tool":"BinPacker","ex90n50":2909},{"tool":"IDBA_Tran","ex90n50":1397},{"tool":"Shannon","ex90n50":2368},{"tool":"SPAdes_sc","ex90n50":3315},{"tool":"SPAdes_rna","ex90n50":2549}]},"extension":[]}
        var id_name = '#vis-699ad978-f6fb-496e-8725-f7dc8a4ab9e6';
        Nyaplot.core.parse(model, id_name);

        require(['downloadable'], function(downloadable){
          var svg = d3.select(id_name).select("svg");
	  if(!svg.empty())
	    svg.call(downloadable().filename('fig'));
	});
    };
    if(window['Nyaplot']==undefined){
        window.addEventListener('load_nyaplot', render, false);
	return;
    } else {
       render();
    }
})();
</script>
<a name="busco">
<h1>S8: BUSCO</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p><a target="_blank" href="http://busco.ezlab.org/">BUSCO</a>: Assessing genome assembly and annotation completeness with Benchmarking Universal Single-Copy Orthologs (<a href="https://www.ncbi.nlm.nih.gov/pubmed/26059717">26059717</a>)</p><table id="buscoTab11" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="100%">
<thead><tr><th>Escherichia coli</th><th>Candida albicans</th><th>Arabidopsis thaliana</th></tr></thead><tbody><tr><td><a href="data/busco/eco/plot/busco_figure.pdf"><img src="data/busco/eco/plot/busco_figure.png" width="500px"></a></br></td><td><a href="data/busco/cal3/plot/busco_figure.pdf"><img src="data/busco/cal3/plot/busco_figure.png" width="500px"></a></br></td><td><a href="data/busco/ath/plot/busco_figure.pdf"><img src="data/busco/ath/plot/busco_figure.png" width="500px"></a></br></td></tr></tbody><thead><tr><th>Homo sapiens + EBOV 3h</th><th>Homo sapiens + EBOV 7h</th><th>Homo sapiens + EBOV 23h</th></tr></thead><tbody><tr><td><a href="data/busco/ebov_hsa_3h/plot/busco_figure.pdf"><img src="data/busco/ebov_hsa_3h/plot/busco_figure.png" width="500px"></a></br></td><td><a href="data/busco/ebov_hsa_7h/plot/busco_figure.pdf"><img src="data/busco/ebov_hsa_7h/plot/busco_figure.png" width="500px"></a></br></td><td><a href="data/busco/ebov_hsa_23h/plot/busco_figure.pdf"><img src="data/busco/ebov_hsa_23h/plot/busco_figure.png" width="500px"></a></br></td></tr></tbody><thead><tr><th>Mus musculus</th><th>Homo sapiens</th><th>Homo sapiens simulated</th></tr></thead><tbody><tr><td><a href="data/busco/mmu/plot/busco_figure.pdf"><img src="data/busco/mmu/plot/busco_figure.png" width="500px"></a></br></td><td><a href="data/busco/hsa/plot/busco_figure.pdf"><img src="data/busco/hsa/plot/busco_figure.png" width="500px"></a></br></td><td><a href="data/busco/hsa_flux/plot/busco_figure.pdf"><img src="data/busco/hsa_flux/plot/busco_figure.png" width="500px"></a></br></td></tr></tbody></table><br><br><br>

<a name="detonate">
<h1>S9: DETONATE</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p><a target="_blank" href="http://deweylab.biostat.wisc.edu/detonate/">DETONATE</a>: DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25608678">25608678</a>).</p><h3><i>Escherichia coli</i> DETONATE results</h3>
<table id="detonateTab1" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-210712852.43</td><td>0.817076</td><td>0.0320276</td><td>0.639604</td></tr><tr><td>oases</td>
<td>-245161321.58</td><td>0.79302</td><td>0.0264751</td><td>0.487533</td></tr><tr><td>trans-abyss</td>
<td>-151118336.50</td><td>0.822888</td><td>0.0328714</td><td>0.611844</td></tr><tr><td>soap-trans-default</td>
<td>-340812817.07</td><td>0.861768</td><td>0.062212</td><td>0.735111</td></tr><tr><td>bridger</td>
<td>-174824953.19</td><td>0.811427</td><td>0.0303294</td><td>0.635494</td></tr><tr><td>binpacker</td>
<td>-262761906.88</td><td>0.162242</td><td>0.000152509</td><td>0.0646008</td></tr><tr><td>idba-tran</td>
<td>-437779699.97</td><td>0.828836</td><td>0.028401</td><td>0.661802</td></tr><tr><td>shannon</td>
<td>-197944222.32</td><td>0.807803</td><td>0.026662</td><td>0.62028</td></tr><tr><td>spades</td>
<td>-194007062.42</td><td>0.874921</td><td>0.0582104</td><td>0.711878</td></tr><tr><td>rna-spades</td>
<td>-200089085.55</td><td>0.843184</td><td>0.0697163</td><td>0.716887</td></tr></tbody></table></br></br>

<h3><i>Candida albicans</i> DETONATE results</h3>
<table id="detonateTab2" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-334561156.85</td><td>0.679298</td><td>0.0766464</td><td>0.726441</td></tr><tr><td>oases</td>
<td>-419101803.95</td><td>0.617874</td><td>0.0756063</td><td>0.505806</td></tr><tr><td>trans-abyss</td>
<td>-341314448.16</td><td>0.734363</td><td>0.0836716</td><td>0.594869</td></tr><tr><td>soap-trans-default</td>
<td>-448426551.63</td><td>0.603828</td><td>0.0627947</td><td>0.730318</td></tr><tr><td>bridger</td>
<td>-354594754.45</td><td>0.681131</td><td>0.0647035</td><td>0.742303</td></tr><tr><td>binpacker</td>
<td>-355541613.29</td><td>0.681339</td><td>0.0650714</td><td>0.725447</td></tr><tr><td>idba-tran</td>
<td>-611471453.61</td><td>0.552593</td><td>0.0463257</td><td>0.727045</td></tr><tr><td>shannon</td>
<td>-356256806.38</td><td>0.718923</td><td>0.0686033</td><td>0.635121</td></tr><tr><td>spades</td>
<td>-369270117.24</td><td>0.707372</td><td>0.0618753</td><td>0.744748</td></tr><tr><td>rna-spades</td>
<td>-352586681.38</td><td>0.691902</td><td>0.0555932</td><td>0.738039</td></tr></tbody></table></br></br>

<h3><i>Arabidopsis thaliana</i> DETONATE results</h3>
<table id="detonateTab3" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-458951076.16</td><td>0.706259</td><td>0.0503788</td><td>0.713888</td></tr><tr><td>oases</td>
<td>-601376841.14</td><td>0.657365</td><td>0.0315914</td><td>0.421727</td></tr><tr><td>trans-abyss</td>
<td>-452320279.03</td><td>0.742054</td><td>0.0417782</td><td>0.586087</td></tr><tr><td>soap-trans-default</td>
<td>-733935298.80</td><td>0.616916</td><td>0.0417285</td><td>0.766783</td></tr><tr><td>bridger</td>
<td>-540615846.28</td><td>0.664801</td><td>0.0413928</td><td>0.717523</td></tr><tr><td>binpacker</td>
<td>-963432760.48</td><td>0.479321</td><td>0.00483293</td><td>0.255829</td></tr><tr><td>idba-tran</td>
<td>-671741034.20</td><td>0.656141</td><td>0.0324962</td><td>0.762677</td></tr><tr><td>shannon</td>
<td>-1343654128.47</td><td>0.373612</td><td>0.0404999</td><td>0.631825</td></tr><tr><td>spades</td>
<td>-593763835.97</td><td>0.713745</td><td>0.0592855</td><td>0.792133</td></tr><tr><td>rna-spades</td>
<td>-573548801.65</td><td>0.720746</td><td>0.0674779</td><td>0.786267</td></tr></tbody></table></br></br>

<h3><i>Mus musculus</i> DETONATE results</h3>
<table id="detonateTab4" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-2226879012.19</td><td>0.663224</td><td>0.00924793</td><td>0.424159</td></tr><tr><td>oases</td>
<td>-3338075293.16</td><td>0.466246</td><td>0.00191568</td><td>0.185852</td></tr><tr><td>trans-abyss</td>
<td>-2152573788.62</td><td>0.671318</td><td>0.0245507</td><td>0.399779</td></tr><tr><td>soap-trans-default</td>
<td>-2638014207.45</td><td>0.55072</td><td>0.0232672</td><td>0.509524</td></tr><tr><td>bridger</td>
<td>-2376224869.25</td><td>0.601723</td><td>0.00662728</td><td>0.450365</td></tr><tr><td>binpacker</td>
<td>-4888140588.38</td><td>0.284626</td><td>8.78969e-05</td><td>0.0681366</td></tr><tr><td>idba-tran</td>
<td>-5029841952.79</td><td>0.531343</td><td>0.0107357</td><td>0.495647</td></tr><tr><td>shannon</td>
<td>-2938129166.97</td><td>0.559955</td><td>0.00746879</td><td>0.38078</td></tr><tr><td>spades</td>
<td>-3008040595.31</td><td>0.686892</td><td>0.0051975</td><td>0.522144</td></tr><tr><td>rna-spades</td>
<td>-2447339857.69</td><td>0.714442</td><td>0.0143569</td><td>0.498195</td></tr></tbody></table></br></br>

<h3><i>Homo sapiens</i> DETONATE results</h3>
<table id="detonateTab5" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-6510590491.36</td><td>0.511762</td><td>0.0161655</td><td>0.42599</td></tr><tr><td>oases</td>
<td>-11817309171.03</td><td>0.243335</td><td>0.0189894</td><td>0.181064</td></tr><tr><td>trans-abyss</td>
<td>-6255709753.29</td><td>0.550435</td><td>0.203218</td><td>0.509015</td></tr><tr><td>soap-trans-default</td>
<td>-9033548866.99</td><td>0.373036</td><td>0.205052</td><td>0.566304</td></tr><tr><td>bridger</td>
<td>-7718007001.60</td><td>0.398725</td><td>0.010466</td><td>0.480306</td></tr><tr><td>binpacker</td>
<td>-10026565572.34</td><td>0.366908</td><td>0.000374367</td><td>0.1451</td></tr><tr><td>idba-tran</td>
<td>-16304023086.75</td><td>0.286957</td><td>0.0168006</td><td>0.552302</td></tr><tr><td>shannon</td>
<td>-8959903268.75</td><td>0.420737</td><td>0.0229751</td><td>0.346262</td></tr><tr><td>spades</td>
<td>-12123853339.02</td><td>0.389103</td><td>0.0140352</td><td>0.606564</td></tr><tr><td>rna-spades</td>
<td>-7163901001.16</td><td>0.427956</td><td>0.0128943</td><td>0.619367</td></tr></tbody></table></br></br>

<h3><i>Homo sapiens + EBOV 3h</i> DETONATE results</h3>
<table id="detonateTab6" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-1235651692.60</td><td>0.518515</td><td>0.0149362</td><td>0.457458</td></tr><tr><td>oases</td>
<td>-1726915020.53</td><td>0.400049</td><td>0.0149624</td><td>0.213295</td></tr><tr><td>trans-abyss</td>
<td>-1196653162.51</td><td>0.541571</td><td>0.0469628</td><td>0.458425</td></tr><tr><td>soap-trans-default</td>
<td>-1348871374.03</td><td>0.452867</td><td>0.0387229</td><td>0.558308</td></tr><tr><td>bridger</td>
<td>-1245885780.74</td><td>0.507537</td><td>0.0134374</td><td>0.491961</td></tr><tr><td>binpacker</td>
<td>-2956358196.35</td><td>0.216398</td><td>6.61916e-05</td><td>0.0537161</td></tr><tr><td>idba-tran</td>
<td>-1840189936.95</td><td>0.435537</td><td>0.0135966</td><td>0.552636</td></tr><tr><td>shannon</td>
<td>-1264628787.04</td><td>0.516653</td><td>0.0279902</td><td>0.388568</td></tr><tr><td>spades</td>
<td>-1316600593.27</td><td>0.493276</td><td>0.0130479</td><td>0.571439</td></tr><tr><td>rna-spades</td>
<td>-1199874337.85</td><td>0.542847</td><td>0.022366</td><td>0.610776</td></tr></tbody></table></br></br>

<h3><i>Homo sapiens + EBOV 7h</i> DETONATE results</h3>
<table id="detonateTab7" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-1621797552.26</td><td>0.548502</td><td>0.0133497</td><td>0.461589</td></tr><tr><td>oases</td>
<td>-2354768025.10</td><td>0.446502</td><td>0.01377</td><td>0.204378</td></tr><tr><td>trans-abyss</td>
<td>-1644765267.08</td><td>0.565015</td><td>0.0474134</td><td>0.456358</td></tr><tr><td>soap-trans-default</td>
<td>-1853762729.85</td><td>0.489643</td><td>0.0382348</td><td>0.552421</td></tr><tr><td>bridger</td>
<td>-1803791019.01</td><td>0.509876</td><td>0.0115023</td><td>0.489854</td></tr><tr><td>binpacker</td>
<td>-1948162277.75</td><td>0.498755</td><td>0.000981932</td><td>0.303781</td></tr><tr><td>idba-tran</td>
<td>-2796730511.36</td><td>0.433572</td><td>0.0122321</td><td>0.555899</td></tr><tr><td>shannon</td>
<td>-1739716273.79</td><td>0.551822</td><td>0.0303068</td><td>0.373245</td></tr><tr><td>spades</td>
<td>-2194102523.92</td><td>0.481227</td><td>0.0139344</td><td>0.570306</td></tr><tr><td>rna-spades</td>
<td>-1712618020.99</td><td>0.555596</td><td>0.0208343</td><td>0.6074</td></tr></tbody></table></br></br>

<h3><i>Homo sapiens + EBOV 23h</i> DETONATE results</h3>
<table id="detonateTab8" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-1434423706.95</td><td>0.767084</td><td>0.0124217</td><td>0.44378</td></tr><tr><td>oases</td>
<td>-3869713310.16</td><td>0.20915</td><td>0.0121553</td><td>0.218292</td></tr><tr><td>trans-abyss</td>
<td>-1445566961.21</td><td>0.779996</td><td>0.044702</td><td>0.448004</td></tr><tr><td>soap-trans-default</td>
<td>-1599735160.94</td><td>0.694346</td><td>0.0354917</td><td>0.547701</td></tr><tr><td>bridger</td>
<td>-1543477872.71</td><td>0.739796</td><td>0.0109893</td><td>0.47975</td></tr><tr><td>binpacker</td>
<td>-2016262591.54</td><td>0.706858</td><td>0.00035399</td><td>0.203836</td></tr><tr><td>idba-tran</td>
<td>-4311460879.81</td><td>0.161691</td><td>0.0113647</td><td>0.551236</td></tr><tr><td>shannon</td>
<td>-4947278470.01</td><td>0.0955414</td><td>0.0282574</td><td>0.379485</td></tr><tr><td>spades</td>
<td>-3603623299.10</td><td>0.247863</td><td>0.0105064</td><td>0.564203</td></tr><tr><td>rna-spades</td>
<td>-1673532042.88</td><td>0.589338</td><td>0.0196731</td><td>0.601376</td></tr></tbody></table></br></br>

<h3><i>Homo sapiens simulated</i> DETONATE results</h3>
<table id="detonateTab9" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>RSEM-EVAL score</th><th>KC score</th><th>Contig F1 score</th><th>Nucleotide F1 score</th></tr></thead>
<tbody>
<tr><td>trinity</td>
<td>-2794698982.84</td><td>0.883827</td><td>0.0633606</td><td>0.566043</td></tr><tr><td>oases</td>
<td>-4615211068.41</td><td>0.738406</td><td>0.0488742</td><td>0.223824</td></tr><tr><td>trans-abyss</td>
<td>-2379646547.27</td><td>0.923633</td><td>0.0987237</td><td>0.679067</td></tr><tr><td>soap-trans-default</td>
<td>-5270177770.68</td><td>0.595837</td><td>0.0752428</td><td>0.733277</td></tr><tr><td>bridger</td>
<td>-3663922534.65</td><td>0.81708</td><td>0.04805</td><td>0.710854</td></tr><tr><td>binpacker</td>
<td>-3649658509.59</td><td>0.821483</td><td>0.0475004</td><td>0.601458</td></tr><tr><td>idba-tran</td>
<td>-7528195849.86</td><td>0.59977</td><td>0.0504635</td><td>0.783647</td></tr><tr><td>shannon</td>
<td>-11513711720.10</td><td>0.261483</td><td>0.0695831</td><td>0.453332</td></tr><tr><td>spades</td>
<td>-4779916710.32</td><td>0.773079</td><td>0.0528581</td><td>0.789225</td></tr><tr><td>rna-spades</td>
<td>-3911011816.48</td><td>0.785485</td><td>0.0541619</td><td>0.695881</td></tr></tbody></table></br></br>

<a name="metrics">
<h1>S10: Selected main metrics</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>Based on our experiences and in comparison to other studies (see manuscript for details) we selected 20 metrics to get an initial idea of the performance 
    of each assembler on each data set. For each metric and assembly we calculated (0,1)-normalized scores displayed in subscript next to the statistical value.</p><h3><i>Escherichia coli</i> selected metrics</h3>
<table id="metricsTab1" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>77.01<sub>0.7</sub></td><td>347<sub>0.42</sub></td><td>13<sub>0.87</sub></td><td>0.368<sub>0.88</sub></td><td>467.035<sub>0.2</sub></td><td>396<sub>0.95</sub></td><td>1.012<sub>0.98</sub></td><td>369<sub>0.46</sub></td><td>407<sub>0.92</sub></td><td>0.31635<sub>1.0</sub></td><td>75.0361<sub>1.0</sub></td><td>NA</td><td>NA</td><td>2143<sub>0.43</sub></td><td>0.639604<sub>0.86</sub></td><td>0.0320276<sub>0.46</sub></td><td>0.817076<sub>0.92</sub></td><td>-0.21<sub>0.79</sub></td><td>284<sub>0.83</sub></td><td>190<sub>0.96</sub></td></tr><td>Oases</td><td>49.16<sub>0.24</sub></td><td>743<sub>1.0</sub></td><td>103<sub>0.0</sub></td><td>0.706<sub>0.7</sub></td><td>457.717<sub>0.19</sub></td><td>414<sub>1.0</sub></td><td>1.443<sub>0.0</sub></td><td>229<sub>0.21</sub></td><td>418<sub>0.94</sub></td><td>0.30558<sub>0.97</sub></td><td>65.71726<sub>0.69</sub></td><td>NA</td><td>NA</td><td>3602<sub>0.0</sub></td><td>0.487533<sub>0.63</sub></td><td>0.0264751<sub>0.38</sub></td><td>0.79302<sub>0.89</sub></td><td>-0.25<sub>0.66</sub></td><td>299<sub>0.88</sub></td><td>172<sub>1.0</sub></td></tr><td>Trans_ABySS</td><td>95.67<sub>1.0</sub></td><td>446<sub>0.56</sub></td><td>18<sub>0.83</sub></td><td>0.322<sub>0.9</sub></td><td>310.442<sub>0.0</sub></td><td>32<sub>0.0</sub></td><td>1.044<sub>0.9</sub></td><td>177<sub>0.12</sub></td><td>436<sub>0.99</sub></td><td>0.02844<sub>0.07</sub></td><td>72.73044<sub>0.92</sub></td><td>NA</td><td>NA</td><td>3081<sub>0.15</sub></td><td>0.611844<sub>0.82</sub></td><td>0.0328714<sub>0.47</sub></td><td>0.822888<sub>0.93</sub></td><td>-0.15<sub>1.0</sub></td><td>297<sub>0.88</sub></td><td>170<sub>1.0</sub></td></tr><td>SOAPdenovo_Trans</td><td>56.62<sub>0.36</sub></td><td>439<sub>0.55</sub></td><td>1<sub>0.99</sub></td><td>0.134<sub>1.0</sub></td><td>364.881<sub>0.07</sub></td><td>261<sub>0.6</sub></td><td>1.002<sub>1.0</sub></td><td>173<sub>0.11</sub></td><td>423<sub>0.96</sub></td><td>0.19492<sub>0.61</sub></td><td>71.98807<sub>0.9</sub></td><td>NA</td><td>NA</td><td>2579<sub>0.3</sub></td><td>0.735111<sub>1.0</sub></td><td>0.062212<sub>0.89</sub></td><td>0.861768<sub>0.98</sub></td><td>-0.34<sub>0.34</sub></td><td>316<sub>0.94</sub></td><td>178<sub>0.99</sub></td></tr><td>Bridger</td><td>87.35<sub>0.86</sub></td><td>363<sub>0.44</sub></td><td>52<sub>0.5</sub></td><td>0.265<sub>0.93</sub></td><td>472.806<sub>0.21</sub></td><td>382<sub>0.92</sub></td><td>1.014<sub>0.97</sub></td><td>393<sub>0.5</sub></td><td>402<sub>0.9</sub></td><td>0.31082<sub>0.98</sub></td><td>74.59062<sub>0.99</sub></td><td>NA</td><td>NA</td><td>2117<sub>0.44</sub></td><td>0.635494<sub>0.85</sub></td><td>0.0303294<sub>0.43</sub></td><td>0.811427<sub>0.91</sub></td><td>-0.17<sub>0.93</sub></td><td>285<sub>0.83</sub></td><td>190<sub>0.96</sub></td></tr><td>BinPacker</td><td>71.09<sub>0.6</sub></td><td>66<sub>0.0</sub></td><td>8<sub>0.92</sub></td><td>2.051<sub>0.0</sub></td><td>1100.774<sub>1.0</sub></td><td>34<sub>0.01</sub></td><td>1.267<sub>0.4</sub></td><td>675<sub>1.0</sub></td><td>37<sub>0.0</sub></td><td>0.00619<sub>0.0</sub></td><td>45.13338<sub>0.0</sub></td><td>NA</td><td>NA</td><td>191<sub>1.0</sub></td><td>0.0646008<sub>0.0</sub></td><td>0.000152509<sub>0.0</sub></td><td>0.162242<sub>0.0</sub></td><td>-0.26<sub>0.62</sub></td><td>50<sub>0.0</sub></td><td>711<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>34.31<sub>0.0</sub></td><td>414<sub>0.51</sub></td><td>0<sub>1.0</sub></td><td>0.167<sub>0.98</sub></td><td>540.828<sub>0.29</sub></td><td>250<sub>0.57</sub></td><td>1.001<sub>1.0</sub></td><td>304<sub>0.34</sub></td><td>396<sub>0.89</sub></td><td>0.16029<sub>0.5</sub></td><td>71.99301<sub>0.9</sub></td><td>NA</td><td>NA</td><td>2069<sub>0.45</sub></td><td>0.661802<sub>0.89</sub></td><td>0.028401<sub>0.41</sub></td><td>0.828836<sub>0.94</sub></td><td>-0.44<sub>0.0</sub></td><td>296<sub>0.87</sub></td><td>196<sub>0.95</sub></td></tr><td>Shannon</td><td>76.69<sub>0.69</sub></td><td>372<sub>0.45</sub></td><td>54<sub>0.48</sub></td><td>0.185<sub>0.97</sub></td><td>482.659<sub>0.22</sub></td><td>393<sub>0.95</sub></td><td>1.027<sub>0.94</sub></td><td>363<sub>0.45</sub></td><td>399<sub>0.9</sub></td><td>0.3049<sub>0.96</sub></td><td>73.3447<sub>0.94</sub></td><td>NA</td><td>NA</td><td>2145<sub>0.43</sub></td><td>0.62028<sub>0.83</sub></td><td>0.026662<sub>0.38</sub></td><td>0.807803<sub>0.91</sub></td><td>-0.2<sub>0.83</sub></td><td>280<sub>0.82</sub></td><td>198<sub>0.95</sub></td></tr><td>SPAdes_sc</td><td>88.04<sub>0.88</sub></td><td>489<sub>0.62</sub></td><td>12<sub>0.88</sub></td><td>0.248<sub>0.94</sub></td><td>558.904<sub>0.31</sub></td><td>210<sub>0.47</sub></td><td>1.006<sub>0.99</sub></td><td>111<sub>0.0</sub></td><td>441<sub>1.0</sub></td><td>0.14391<sub>0.44</sub></td><td>69.46914<sub>0.81</sub></td><td>NA</td><td>NA</td><td>2370<sub>0.36</sub></td><td>0.711878<sub>0.97</sub></td><td>0.0582104<sub>0.83</sub></td><td>0.874921<sub>1.0</sub></td><td>-0.19<sub>0.86</sub></td><td>332<sub>1.0</sub></td><td>172<sub>1.0</sub></td></tr><td>SPAdes_rna</td><td>88.9<sub>0.89</sub></td><td>340<sub>0.4</sub></td><td>10<sub>0.9</sub></td><td>0.212<sub>0.96</sub></td><td>359.195<sub>0.06</sub></td><td>187<sub>0.41</sub></td><td>1.009<sub>0.98</sub></td><td>245<sub>0.24</sub></td><td>338<sub>0.75</sub></td><td>0.17587<sub>0.55</sub></td><td>73.77677<sub>0.96</sub></td><td>NA</td><td>NA</td><td>2556<sub>0.31</sub></td><td>0.716887<sub>0.97</sub></td><td>0.0697163<sub>1.0</sub></td><td>0.843184<sub>0.96</sub></td><td>-0.2<sub>0.83</sub></td><td>255<sub>0.73</sub></td><td>189<sub>0.96</sub></td></tr></tbody></table><h3><i>Escherichia coli</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 13.61/18</li><li>Oases: 10.36/18</li><li>Trans_ABySS: 11.53/18</li><li>SOAPdenovo_Trans: 12.59/18</li><li>Bridger: 13.55/18</li><li>BinPacker: 5.55/18</li><li>IDBA_Tran: 11.49/18</li><li>Shannon: 13.08/18</li><li>SPAdes_sc: 13.37/18</li><li>SPAdes_rna: 12.85/18</li></ul></br>Download <a href="data/main_metrics/csv/eco.xlsx">Excel</a> | <a href="data/main_metrics/csv/eco.csv">csv</a> | <a href="data/main_metrics/tex/eco.pdf">PDF</a> | <a href="data/main_metrics/tex/eco.tex">LaTeX</a> </br></br><h3><i>Candida albicans</i> selected metrics</h3>
<table id="metricsTab2" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>97.32<sub>0.91</sub></td><td>3417<sub>0.09</sub></td><td>72<sub>0.89</sub></td><td>1.819<sub>0.27</sub></td><td>969.327<sub>0.56</sub></td><td>4146<sub>0.46</sub></td><td>1.039<sub>0.97</sub></td><td>1862<sub>0.73</sub></td><td>1732<sub>0.76</sub></td><td>0.18357<sub>0.06</sub></td><td>81.69261<sub>0.51</sub></td><td>0.45572<sub>0.83</sub></td><td>0.13015<sub>0.85</sub></td><td>9391<sub>0.96</sub></td><td>0.726441<sub>0.92</sub></td><td>0.0766464<sub>0.81</sub></td><td>0.679298<sub>0.7</sub></td><td>-0.33<sub>1.0</sub></td><td>1355<sub>0.65</sub></td><td>164<sub>0.7</sub></td></tr><td>Oases</td><td>93.61<sub>0.6</sub></td><td>9813<sub>1.0</sub></td><td>613<sub>0.0</sub></td><td>2.121<sub>0.09</sub></td><td>941.6<sub>0.53</sub></td><td>4808<sub>0.79</sub></td><td>2.25<sub>0.0</sub></td><td>1588<sub>0.4</sub></td><td>1836<sub>0.91</sub></td><td>0.23593<sub>0.31</sub></td><td>79.537<sub>0.33</sub></td><td>0.05827<sub>0.05</sub></td><td>0.86792<sub>0.0</sub></td><td>26010<sub>0.0</sub></td><td>0.505806<sub>0.0</sub></td><td>0.0756063<sub>0.78</sub></td><td>0.617874<sub>0.36</sub></td><td>-0.42<sub>0.68</sub></td><td>1382<sub>0.71</sub></td><td>140<sub>0.79</sub></td></tr><td>Trans_ABySS</td><td>98.45<sub>1.0</sub></td><td>7979<sub>0.74</sub></td><td>449<sub>0.27</sub></td><td>1.288<sub>0.58</sub></td><td>773.584<sub>0.29</sub></td><td>5214<sub>1.0</sub></td><td>1.916<sub>0.27</sub></td><td>1664<sub>0.49</sub></td><td>1895<sub>1.0</sub></td><td>0.3824<sub>1.0</sub></td><td>81.46816<sub>0.49</sub></td><td>0.03363<sub>0.0</sub></td><td>0.82194<sub>0.05</sub></td><td>22581<sub>0.2</sub></td><td>0.594869<sub>0.37</sub></td><td>0.0836716<sub>1.0</sub></td><td>0.734363<sub>1.0</sub></td><td>-0.34<sub>0.96</sub></td><td>1461<sub>0.87</sub></td><td>109<sub>0.9</sub></td></tr><td>SOAPdenovo_Trans</td><td>95.12<sub>0.73</sub></td><td>2769<sub>0.0</sub></td><td>9<sub>1.0</sub></td><td>0.569<sub>1.0</sub></td><td>567.169<sub>0.0</sub></td><td>3290<sub>0.03</sub></td><td>1.007<sub>1.0</sub></td><td>1374<sub>0.14</sub></td><td>1274<sub>0.08</sub></td><td>0.19889<sub>0.13</sub></td><td>84.14638<sub>0.71</sub></td><td>0.48033<sub>0.88</sub></td><td>0.01897<sub>0.98</sub></td><td>8947<sub>0.98</sub></td><td>0.730318<sub>0.94</sub></td><td>0.0627947<sub>0.44</sub></td><td>0.603828<sub>0.28</sub></td><td>-0.45<sub>0.57</sub></td><td>1042<sub>0.0</sub></td><td>248<sub>0.4</sub></td></tr><td>Bridger</td><td>96.72<sub>0.86</sub></td><td>4038<sub>0.18</sub></td><td>236<sub>0.62</sub></td><td>1.996<sub>0.17</sub></td><td>1131.326<sub>0.79</sub></td><td>4005<sub>0.39</sub></td><td>1.105<sub>0.92</sub></td><td>1923<sub>0.8</sub></td><td>1698<sub>0.71</sub></td><td>0.17196<sub>0.0</sub></td><td>79.39954<sub>0.32</sub></td><td>0.36308<sub>0.65</sub></td><td>0.31231<sub>0.64</sub></td><td>10651<sub>0.88</sub></td><td>0.742303<sub>0.99</sub></td><td>0.0647035<sub>0.49</sub></td><td>0.681131<sub>0.71</sub></td><td>-0.35<sub>0.93</sub></td><td>1416<sub>0.78</sub></td><td>133<sub>0.81</sub></td></tr><td>BinPacker</td><td>96.66<sub>0.85</sub></td><td>4267<sub>0.21</sub></td><td>259<sub>0.59</sub></td><td>2.283<sub>0.0</sub></td><td>1189.552<sub>0.87</sub></td><td>4033<sub>0.4</sub></td><td>1.136<sub>0.89</sub></td><td>1876<sub>0.74</sub></td><td>1701<sub>0.71</sub></td><td>0.17137<sub>0.0</sub></td><td>78.64989<sub>0.26</sub></td><td>0.36772<sub>0.66</sub></td><td>0.3605<sub>0.59</sub></td><td>11072<sub>0.86</sub></td><td>0.725447<sub>0.92</sub></td><td>0.0650714<sub>0.5</sub></td><td>0.681339<sub>0.71</sub></td><td>-0.36<sub>0.89</sub></td><td>1416<sub>0.78</sub></td><td>135<sub>0.81</sub></td></tr><td>IDBA_Tran</td><td>86.34<sub>0.0</sub></td><td>2949<sub>0.03</sub></td><td>8<sub>1.0</sub></td><td>0.807<sub>0.86</sub></td><td>922.096<sub>0.5</sub></td><td>3238<sub>0.0</sub></td><td>1.003<sub>1.0</sub></td><td>1257<sub>0.0</sub></td><td>1223<sub>0.0</sub></td><td>0.1961<sub>0.12</sub></td><td>83.37413<sub>0.65</sub></td><td>0.41421<sub>0.75</sub></td><td>0.00289<sub>1.0</sub></td><td>8652<sub>1.0</sub></td><td>0.727045<sub>0.93</sub></td><td>0.0463257<sub>0.0</sub></td><td>0.552593<sub>0.0</sub></td><td>-0.61<sub>0.0</sub></td><td>1070<sub>0.06</sub></td><td>240<sub>0.43</sub></td></tr><td>Shannon</td><td>96.51<sub>0.84</sub></td><td>4077<sub>0.19</sub></td><td>362<sub>0.41</sub></td><td>0.999<sub>0.75</sub></td><td>678.947<sub>0.16</sub></td><td>3736<sub>0.25</sub></td><td>1.412<sub>0.67</sub></td><td>1321<sub>0.08</sub></td><td>1480<sub>0.38</sub></td><td>0.30264<sub>0.62</sub></td><td>87.68973<sub>1.0</sub></td><td>0.05188<sub>0.04</sub></td><td>0.48455<sub>0.44</sub></td><td>12793<sub>0.76</sub></td><td>0.635121<sub>0.54</sub></td><td>0.0686033<sub>0.6</sub></td><td>0.718923<sub>0.92</sub></td><td>-0.36<sub>0.89</sub></td><td>1088<sub>0.1</sub></td><td>359<sub>0.0</sub></td></tr><td>SPAdes_sc</td><td>97.51<sub>0.92</sub></td><td>3504<sub>0.1</sub></td><td>60<sub>0.91</sub></td><td>1.799<sub>0.28</sub></td><td>1279.411<sub>1.0</sub></td><td>4437<sub>0.61</sub></td><td>1.001<sub>1.0</sub></td><td>2090<sub>1.0</sub></td><td>1847<sub>0.93</sub></td><td>0.17457<sub>0.02</sub></td><td>77.02173<sub>0.13</sub></td><td>0.54236<sub>1.0</sub></td><td>0.00294<sub>1.0</sub></td><td>9173<sub>0.97</sub></td><td>0.744748<sub>1.0</sub></td><td>0.0618753<sub>0.42</sub></td><td>0.707372<sub>0.85</sub></td><td>-0.37<sub>0.86</sub></td><td>1523<sub>1.0</sub></td><td>81<sub>1.0</sub></td></tr><td>SPAdes_rna</td><td>97.43<sub>0.92</sub></td><td>3939<sub>0.17</sub></td><td>82<sub>0.88</sub></td><td>1.959<sub>0.19</sub></td><td>1171.714<sub>0.85</sub></td><td>4533<sub>0.66</sub></td><td>1.059<sub>0.95</sub></td><td>2034<sub>0.93</sub></td><td>1837<sub>0.91</sub></td><td>0.19819<sub>0.13</sub></td><td>75.46275<sub>0.0</sub></td><td>0.40623<sub>0.73</sub></td><td>0.16062<sub>0.82</sub></td><td>10567<sub>0.89</sub></td><td>0.738039<sub>0.97</sub></td><td>0.0555932<sub>0.25</sub></td><td>0.691902<sub>0.77</sub></td><td>-0.35<sub>0.93</sub></td><td>1499<sub>0.95</sub></td><td>88<sub>0.97</sub></td></tr></tbody></table><h3><i>Candida albicans</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 13.633/20</li><li>Oases: 8.329/20</li><li>Trans_ABySS: 12.486/20</li><li>SOAPdenovo_Trans: 10.277/20</li><li>Bridger: 12.639/20</li><li>BinPacker: 12.243/20</li><li>IDBA_Tran: 8.308/20</li><li>Shannon: 9.632/20</li><li>SPAdes_sc: 14.996/20</li><li>SPAdes_rna: 13.86/20</li></ul></br>Download <a href="data/main_metrics/csv/cal3.xlsx">Excel</a> | <a href="data/main_metrics/csv/cal3.csv">csv</a> | <a href="data/main_metrics/tex/cal3.pdf">PDF</a> | <a href="data/main_metrics/tex/cal3.tex">LaTeX</a> </br></br><h3><i>Arabidopsis thaliana</i> selected metrics</h3>
<table id="metricsTab3" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>95.71<sub>0.96</sub></td><td>13357<sub>0.33</sub></td><td>1212<sub>0.84</sub></td><td>1.315<sub>0.79</sub></td><td>978.213<sub>0.55</sub></td><td>2302<sub>0.8</sub></td><td>1.128<sub>0.88</sub></td><td>1451<sub>0.83</sub></td><td>7219<sub>0.81</sub></td><td>0.15512<sub>0.69</sub></td><td>74.79009<sub>0.57</sub></td><td>NA</td><td>NA</td><td>35964<sub>0.61</sub></td><td>0.713888<sub>0.85</sub></td><td>0.0503788<sub>0.73</sub></td><td>0.706259<sub>0.9</sub></td><td>-0.46<sub>0.99</sub></td><td>1117<sub>1.0</sub></td><td>219<sub>1.0</sub></td></tr><td>Oases</td><td>91.93<sub>0.88</sub></td><td>32647<sub>1.0</sub></td><td>6804<sub>0.06</sub></td><td>3.471<sub>0.42</sub></td><td>1232.688<sub>0.89</sub></td><td>2753<sub>1.0</sub></td><td>1.976<sub>0.0</sub></td><td>1305<sub>0.71</sub></td><td>8361<sub>1.0</sub></td><td>0.19207<sub>0.9</sub></td><td>72.07497<sub>0.38</sub></td><td>NA</td><td>NA</td><td>78762<sub>0.0</sub></td><td>0.421727<sub>0.31</sub></td><td>0.0315914<sub>0.43</sub></td><td>0.657365<sub>0.77</sub></td><td>-0.6<sub>0.83</sub></td><td>1108<sub>0.99</sub></td><td>248<sub>0.97</sub></td></tr><td>Trans_ABySS</td><td>97.55<sub>1.0</sub></td><td>19714<sub>0.55</sub></td><td>7227<sub>0.0</sub></td><td>0.612<sub>0.91</sub></td><td>745.088<sub>0.24</sub></td><td>1960<sub>0.65</sub></td><td>1.341<sub>0.66</sub></td><td>940<sub>0.41</sub></td><td>8133<sub>0.96</sub></td><td>0.20992<sub>1.0</sub></td><td>72.50447<sub>0.41</sub></td><td>NA</td><td>NA</td><td>53180<sub>0.37</sub></td><td>0.586087<sub>0.62</sub></td><td>0.0417782<sub>0.59</sub></td><td>0.742054<sub>1.0</sub></td><td>-0.45<sub>1.0</sub></td><td>1119<sub>1.0</sub></td><td>224<sub>0.99</sub></td></tr><td>SOAPdenovo_Trans</td><td>85.31<sub>0.73</sub></td><td>8709<sub>0.17</sub></td><td>51<sub>1.0</sub></td><td>0.099<sub>1.0</sub></td><td>561.746<sub>0.0</sub></td><td>1213<sub>0.32</sub></td><td>1.008<sub>1.0</sub></td><td>449<sub>0.0</sub></td><td>5411<sub>0.52</sub></td><td>0.1479<sub>0.65</sub></td><td>80.77778<sub>1.0</sub></td><td>NA</td><td>NA</td><td>28632<sub>0.72</sub></td><td>0.766783<sub>0.95</sub></td><td>0.0417285<sub>0.59</sub></td><td>0.616916<sub>0.66</sub></td><td>-0.73<sub>0.69</sub></td><td>1058<sub>0.93</sub></td><td>248<sub>0.97</sub></td></tr><td>Bridger</td><td>90.64<sub>0.85</sub></td><td>12882<sub>0.32</sub></td><td>1995<sub>0.73</sub></td><td>2.427<sub>0.6</sub></td><td>990.349<sub>0.57</sub></td><td>1909<sub>0.63</sub></td><td>1.067<sub>0.94</sub></td><td>1636<sub>0.99</sub></td><td>7163<sub>0.8</sub></td><td>0.132<sub>0.57</sub></td><td>74.66313<sub>0.56</sub></td><td>NA</td><td>NA</td><td>33008<sub>0.66</sub></td><td>0.717523<sub>0.86</sub></td><td>0.0413928<sub>0.58</sub></td><td>0.664801<sub>0.79</sub></td><td>-0.54<sub>0.9</sub></td><td>1103<sub>0.98</sub></td><td>229<sub>0.99</sub></td></tr><td>BinPacker</td><td>67.15<sub>0.34</sub></td><td>3775<sub>0.0</sub></td><td>1233<sub>0.84</sub></td><td>5.902<sub>0.0</sub></td><td>1312.171<sub>1.0</sub></td><td>489<sub>0.0</sub></td><td>1.149<sub>0.85</sub></td><td>1452<sub>0.83</sub></td><td>2227<sub>0.0</sub></td><td>0.03029<sub>0.0</sub></td><td>66.83681<sub>0.0</sub></td><td>NA</td><td>NA</td><td>9081<sub>1.0</sub></td><td>0.255829<sub>0.0</sub></td><td>0.00483293<sub>0.0</sub></td><td>0.479321<sub>0.29</sub></td><td>-0.96<sub>0.43</sub></td><td>262<sub>0.0</sub></td><td>1162<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>89.04<sub>0.82</sub></td><td>9577<sub>0.2</sub></td><td>201<sub>0.98</sub></td><td>0.136<sub>0.99</sub></td><td>848.356<sub>0.38</sub></td><td>1175<sub>0.3</sub></td><td>1.01<sub>1.0</sub></td><td>805<sub>0.3</sub></td><td>6087<sub>0.63</sub></td><td>0.14613<sub>0.64</sub></td><td>80.70797<sub>0.99</sub></td><td>NA</td><td>NA</td><td>27666<sub>0.73</sub></td><td>0.762677<sub>0.95</sub></td><td>0.0324962<sub>0.44</sub></td><td>0.656141<sub>0.77</sub></td><td>-0.67<sub>0.75</sub></td><td>930<sub>0.78</sub></td><td>269<sub>0.95</sub></td></tr><td>Shannon</td><td>51.53<sub>0.0</sub></td><td>12018<sub>0.29</sub></td><td>769<sub>0.9</sub></td><td>0.724<sub>0.89</sub></td><td>1044.904<sub>0.64</sub></td><td>2004<sub>0.67</sub></td><td>1.185<sub>0.82</sub></td><td>1654<sub>1.0</sub></td><td>6541<sub>0.7</sub></td><td>0.14777<sub>0.65</sub></td><td>76.08368<sub>0.66</sub></td><td>NA</td><td>NA</td><td>30178<sub>0.7</sub></td><td>0.631825<sub>0.7</sub></td><td>0.0404999<sub>0.57</sub></td><td>0.373612<sub>0.0</sub></td><td>-1.34<sub>0.0</sub></td><td>1049<sub>0.92</sub></td><td>296<sub>0.92</sub></td></tr><td>SPAdes_sc</td><td>94.29<sub>0.93</sub></td><td>11171<sub>0.26</sub></td><td>1189<sub>0.84</sub></td><td>0.43<sub>0.94</sub></td><td>944.388<sub>0.51</sub></td><td>2318<sub>0.81</sub></td><td>1.007<sub>1.0</sub></td><td>1030<sub>0.48</sub></td><td>7800<sub>0.91</sub></td><td>0.151<sub>0.67</sub></td><td>74.47182<sub>0.55</sub></td><td>NA</td><td>NA</td><td>29569<sub>0.71</sub></td><td>0.792133<sub>1.0</sub></td><td>0.0592855<sub>0.87</sub></td><td>0.713745<sub>0.92</sub></td><td>-0.59<sub>0.84</sub></td><td>1074<sub>0.95</sub></td><td>225<sub>0.99</sub></td></tr><td>SPAdes_rna</td><td>93.67<sub>0.92</sub></td><td>11160<sub>0.26</sub></td><td>505<sub>0.94</sub></td><td>0.341<sub>0.96</sub></td><td>807.422<sub>0.33</sub></td><td>2237<sub>0.77</sub></td><td>1.027<sub>0.98</sub></td><td>1239<sub>0.66</sub></td><td>7501<sub>0.86</sub></td><td>0.15415<sub>0.69</sub></td><td>75.67217<sub>0.63</sub></td><td>NA</td><td>NA</td><td>30621<sub>0.69</sub></td><td>0.786267<sub>0.99</sub></td><td>0.0674779<sub>1.0</sub></td><td>0.720746<sub>0.94</sub></td><td>-0.57<sub>0.87</sub></td><td>1012<sub>0.88</sub></td><td>264<sub>0.95</sub></td></tr></tbody></table><h3><i>Arabidopsis thaliana</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 14.15/18</li><li>Oases: 11.53/18</li><li>Trans_ABySS: 12.36/18</li><li>SOAPdenovo_Trans: 11.9/18</li><li>Bridger: 13.31/18</li><li>BinPacker: 5.57/18</li><li>IDBA_Tran: 12.6/18</li><li>Shannon: 11.03/18</li><li>SPAdes_sc: 14.18/18</li><li>SPAdes_rna: 14.3/18</li></ul></br>Download <a href="data/main_metrics/csv/ath.xlsx">Excel</a> | <a href="data/main_metrics/csv/ath.csv">csv</a> | <a href="data/main_metrics/tex/ath.pdf">PDF</a> | <a href="data/main_metrics/tex/ath.tex">LaTeX</a> </br></br><h3><i>Mus musculus</i> selected metrics</h3>
<table id="metricsTab4" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>92.14<sub>0.89</sub></td><td>21804<sub>0.4</sub></td><td>564<sub>0.99</sub></td><td>0.84<sub>0.85</sub></td><td>1216.863<sub>0.52</sub></td><td>6032<sub>0.96</sub></td><td>1.547<sub>0.39</sub></td><td>420<sub>0.0</sub></td><td>8210<sub>0.97</sub></td><td>0.21834<sub>0.88</sub></td><td>52.20767<sub>0.82</sub></td><td>0.22136<sub>0.53</sub></td><td>0.51973<sub>0.43</sub></td><td>78726<sub>0.6</sub></td><td>0.424159<sub>0.78</sub></td><td>0.00924793<sub>0.37</sub></td><td>0.663224<sub>0.88</sub></td><td>-2.23<sub>0.97</sub></td><td>3895<sub>0.97</sub></td><td>1959<sub>0.98</sub></td></tr><td>Oases</td><td>89.29<sub>0.82</sub></td><td>51832<sub>1.0</sub></td><td>52668<sub>0.0</sub></td><td>0.625<sub>0.9</sub></td><td>406.084<sub>0.0</sub></td><td>183<sub>0.02</sub></td><td>1.9<sub>0.0</sub></td><td>2044<sub>0.48</sub></td><td>6293<sub>0.71</sub></td><td>0.02211<sub>0.09</sub></td><td>45.35795<sub>0.4</sub></td><td>0.01921<sub>0.0</sub></td><td>0.90903<sub>0.0</sub></td><td>188947<sub>0.0</sub></td><td>0.185852<sub>0.26</sub></td><td>0.00191568<sub>0.07</sub></td><td>0.466246<sub>0.42</sub></td><td>-3.34<sub>0.59</sub></td><td>3066<sub>0.75</sub></td><td>2476<sub>0.85</sub></td></tr><td>Trans_ABySS</td><td>93.96<sub>0.93</sub></td><td>29719<sub>0.56</sub></td><td>612<sub>0.99</sub></td><td>0.316<sub>0.97</sub></td><td>752.811<sub>0.22</sub></td><td>6281<sub>1.0</sub></td><td>1.75<sub>0.17</sub></td><td>530<sub>0.03</sub></td><td>8401<sub>1.0</sub></td><td>0.24728<sub>1.0</sub></td><td>53.7886<sub>0.92</sub></td><td>0.11552<sub>0.25</sub></td><td>0.62836<sub>0.31</sub></td><td>105153<sub>0.46</sub></td><td>0.399779<sub>0.73</sub></td><td>0.0245507<sub>1.0</sub></td><td>0.671318<sub>0.9</sub></td><td>-2.15<sub>1.0</sub></td><td>3992<sub>1.0</sub></td><td>1926<sub>0.99</sub></td></tr><td>SOAPdenovo_Trans</td><td>90.98<sub>0.86</sub></td><td>12603<sub>0.21</sub></td><td>61<sub>1.0</sub></td><td>0.163<sub>1.0</sub></td><td>519.962<sub>0.07</sub></td><td>2244<sub>0.35</sub></td><td>1.031<sub>0.97</sub></td><td>2796<sub>0.7</sub></td><td>7375<sub>0.86</sub></td><td>0.09539<sub>0.38</sub></td><td>51.45431<sub>0.78</sub></td><td>0.40138<sub>1.0</sub></td><td>0.11922<sub>0.88</sub></td><td>53611<sub>0.74</sub></td><td>0.509524<sub>0.97</sub></td><td>0.0232672<sub>0.95</sub></td><td>0.55072<sub>0.62</sub></td><td>-2.64<sub>0.83</sub></td><td>3616<sub>0.9</sub></td><td>1987<sub>0.97</sub></td></tr><td>Bridger</td><td>91.66<sub>0.88</sub></td><td>18266<sub>0.33</sub></td><td>2643<sub>0.95</sub></td><td>0.913<sub>0.83</sub></td><td>1073.395<sub>0.43</sub></td><td>2236<sub>0.35</sub></td><td>1.348<sub>0.62</sub></td><td>3100<sub>0.79</sub></td><td>7639<sub>0.9</sub></td><td>0.09933<sub>0.4</sub></td><td>49.56206<sub>0.66</sub></td><td>0.18076<sub>0.42</sub></td><td>0.41364<sub>0.55</sub></td><td>66954<sub>0.67</sub></td><td>0.450365<sub>0.84</sub></td><td>0.00662728<sub>0.27</sub></td><td>0.601723<sub>0.74</sub></td><td>-2.38<sub>0.92</sub></td><td>3914<sub>0.98</sub></td><td>1926<sub>0.99</sub></td></tr><td>BinPacker</td><td>54.31<sub>0.0</sub></td><td>2037<sub>0.0</sub></td><td>628<sub>0.99</sub></td><td>4.609<sub>0.0</sub></td><td>1972.809<sub>1.0</sub></td><td>554<sub>0.08</sub></td><td>1.346<sub>0.62</sub></td><td>2715<sub>0.68</sub></td><td>1014<sub>0.0</sub></td><td>0.0154<sub>0.06</sub></td><td>38.85579<sub>0.0</sub></td><td>0.13036<sub>0.29</sub></td><td>0.60065<sub>0.34</sub></td><td>6514<sub>1.0</sub></td><td>0.0681366<sub>0.0</sub></td><td>8.78969e-05<sub>0.0</sub></td><td>0.284626<sub>0.0</sub></td><td>-4.89<sub>0.05</sub></td><td>310<sub>0.0</sub></td><td>5871<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>70.64<sub>0.38</sub></td><td>12294<sub>0.21</sub></td><td>41<sub>1.0</sub></td><td>0.187<sub>0.99</sub></td><td>811.99<sub>0.26</sub></td><td>967<sub>0.14</sub></td><td>1.005<sub>1.0</sub></td><td>1672<sub>0.37</sub></td><td>4208<sub>0.43</sub></td><td>0.09159<sub>0.37</sub></td><td>54.28147<sub>0.95</sub></td><td>0.29426<sub>0.72</sub></td><td>0.01711<sub>1.0</sub></td><td>45526<sub>0.79</sub></td><td>0.495647<sub>0.94</sub></td><td>0.0107357<sub>0.44</sub></td><td>0.531343<sub>0.57</sub></td><td>-5.03<sub>0.0</sub></td><td>2679<sub>0.64</sub></td><td>2188<sub>0.92</sub></td></tr><td>Shannon</td><td>86.6<sub>0.76</sub></td><td>16915<sub>0.3</sub></td><td>1927<sub>0.96</sub></td><td>0.802<sub>0.86</sub></td><td>877.706<sub>0.3</sub></td><td>71<sub>0.0</sub></td><td>1.23<sub>0.75</sub></td><td>2067<sub>0.49</sub></td><td>5248<sub>0.57</sub></td><td>0.00107<sub>0.0</sub></td><td>55.04317<sub>1.0</sub></td><td>0.18394<sub>0.43</sub></td><td>0.42689<sub>0.54</sub></td><td>52948<sub>0.75</sub></td><td>0.38078<sub>0.69</sub></td><td>0.00746879<sub>0.3</sub></td><td>0.559955<sub>0.64</sub></td><td>-2.94<sub>0.73</sub></td><td>2829<sub>0.68</sub></td><td>2534<sub>0.84</sub></td></tr><td>SPAdes_sc</td><td>93.83<sub>0.93</sub></td><td>13099<sub>0.22</sub></td><td>570<sub>0.99</sub></td><td>0.706<sub>0.88</sub></td><td>1089.264<sub>0.44</sub></td><td>2633<sub>0.41</sub></td><td>1.003<sub>1.0</sub></td><td>3814<sub>1.0</sub></td><td>7736<sub>0.91</sub></td><td>0.08457<sub>0.34</sub></td><td>44.85287<sub>0.37</sub></td><td>0.37953<sub>0.94</sub></td><td>0.0134<sub>1.0</sub></td><td>51575<sub>0.75</sub></td><td>0.522144<sub>1.0</sub></td><td>0.0051975<sub>0.21</sub></td><td>0.686892<sub>0.94</sub></td><td>-3.01<sub>0.7</sub></td><td>3823<sub>0.95</sub></td><td>1877<sub>1.0</sub></td></tr><td>SPAdes_rna</td><td>96.86<sub>1.0</sub></td><td>19367<sub>0.35</sub></td><td>1080<sub>0.98</sub></td><td>0.802<sub>0.86</sub></td><td>910.466<sub>0.32</sub></td><td>3269<sub>0.51</sub></td><td>1.135<sub>0.85</sub></td><td>3418<sub>0.88</sub></td><td>7859<sub>0.93</sub></td><td>0.10624<sub>0.43</sub></td><td>43.57814<sub>0.29</sub></td><td>0.17718<sub>0.41</sub></td><td>0.33204<sub>0.64</sub></td><td>73461<sub>0.63</sub></td><td>0.498195<sub>0.95</sub></td><td>0.0143569<sub>0.58</sub></td><td>0.714442<sub>1.0</sub></td><td>-2.45<sub>0.9</sub></td><td>3773<sub>0.94</sub></td><td>1880<sub>1.0</sub></td></tr></tbody></table><h3><i>Mus musculus</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 14.209/20</li><li>Oases: 7.358/20</li><li>Trans_ABySS: 14.429/20</li><li>SOAPdenovo_Trans: 15.05/20</li><li>Bridger: 13.501/20</li><li>BinPacker: 5.102/20</li><li>IDBA_Tran: 12.126/20</li><li>Shannon: 11.576/20</li><li>SPAdes_sc: 14.983/20</li><li>SPAdes_rna: 14.46/20</li></ul></br>Download <a href="data/main_metrics/csv/mmu.xlsx">Excel</a> | <a href="data/main_metrics/csv/mmu.csv">csv</a> | <a href="data/main_metrics/tex/mmu.pdf">PDF</a> | <a href="data/main_metrics/tex/mmu.tex">LaTeX</a> </br></br><h3><i>Homo sapiens</i> selected metrics</h3>
<table id="metricsTab5" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>91.9<sub>0.81</sub></td><td>64061<sub>0.22</sub></td><td>3378<sub>0.99</sub></td><td>1.382<sub>0.74</sub></td><td>795.225<sub>0.27</sub></td><td>6788<sub>0.99</sub></td><td>2.396<sub>0.0</sub></td><td>326<sub>0.0</sub></td><td>8930<sub>0.97</sub></td><td>0.22744<sub>0.87</sub></td><td>50.81538<sub>0.64</sub></td><td>0.1318<sub>0.3</sub></td><td>0.59035<sub>0.38</sub></td><td>286479<sub>0.72</sub></td><td>0.42599<sub>0.59</sub></td><td>0.0161655<sub>0.08</sub></td><td>0.511762<sub>0.87</sub></td><td>-6.51<sub>0.98</sub></td><td>4004<sub>0.96</sub></td><td>1804<sub>0.99</sub></td></tr><td>Oases</td><td>88.04<sub>0.69</sub></td><td>207474<sub>1.0</sub></td><td>216127<sub>0.0</sub></td><td>1.253<sub>0.77</sub></td><td>343.479<sub>0.06</sub></td><td>868<sub>0.1</sub></td><td>2.355<sub>0.03</sub></td><td>666<sub>0.17</sub></td><td>8024<sub>0.83</sub></td><td>0.0881<sub>0.33</sub></td><td>42.08886<sub>0.0</sub></td><td>0.01793<sub>0.0</sub></td><td>0.94001<sub>0.0</sub></td><td>843235<sub>0.0</sub></td><td>0.181064<sub>0.08</sub></td><td>0.0189894<sub>0.09</sub></td><td>0.243335<sub>0.0</sub></td><td>-11.82<sub>0.45</sub></td><td>3588<sub>0.79</sub></td><td>1922<sub>0.93</sub></td></tr><td>Trans_ABySS</td><td>98.34<sub>1.0</sub></td><td>59779<sub>0.2</sub></td><td>2743<sub>0.99</sub></td><td>0.573<sub>0.93</sub></td><td>246.846<sub>0.01</sub></td><td>6824<sub>1.0</sub></td><td>1.743<sub>0.47</sub></td><td>441<sub>0.06</sub></td><td>9110<sub>1.0</sub></td><td>0.26219<sub>1.0</sub></td><td>51.92248<sub>0.72</sub></td><td>0.10588<sub>0.23</sub></td><td>0.63297<sub>0.33</sub></td><td>437845<sub>0.53</sub></td><td>0.509015<sub>0.77</sub></td><td>0.203218<sub>0.99</sub></td><td>0.550435<sub>1.0</sub></td><td>-6.26<sub>1.0</sub></td><td>4106<sub>1.0</sub></td><td>1770<sub>1.0</sub></td></tr><td>SOAPdenovo_Trans</td><td>89.93<sub>0.75</sub></td><td>27529<sub>0.03</sub></td><td>279<sub>1.0</sub></td><td>0.269<sub>1.0</sub></td><td>218.0<sub>0.0</sub></td><td>2264<sub>0.31</sub></td><td>1.187<sub>0.87</sub></td><td>711<sub>0.19</sub></td><td>6806<sub>0.64</sub></td><td>0.09183<sub>0.34</sub></td><td>48.01834<sub>0.44</sub></td><td>0.26752<sub>0.66</sub></td><td>0.32675<sub>0.67</sub></td><td>241236<sub>0.78</sub></td><td>0.566304<sub>0.89</sub></td><td>0.205052<sub>1.0</sub></td><td>0.373036<sub>0.42</sub></td><td>-9.03<sub>0.72</sub></td><td>2625<sub>0.39</sub></td><td>2164<sub>0.83</sub></td></tr><td>Bridger</td><td>86.83<sub>0.66</sub></td><td>43201<sub>0.11</sub></td><td>7329<sub>0.97</sub></td><td>1.439<sub>0.73</sub></td><td>654.413<sub>0.21</sub></td><td>2105<sub>0.28</sub></td><td>1.708<sub>0.5</sub></td><td>1370<sub>0.51</sub></td><td>8440<sub>0.89</sub></td><td>0.085<sub>0.31</sub></td><td>45.10419<sub>0.22</sub></td><td>0.14093<sub>0.32</sub></td><td>0.42073<sub>0.57</sub></td><td>206635<sub>0.83</sub></td><td>0.480306<sub>0.71</sub></td><td>0.010466<sub>0.05</sub></td><td>0.398725<sub>0.51</sub></td><td>-7.72<sub>0.85</sub></td><td>3909<sub>0.92</sub></td><td>1812<sub>0.98</sub></td></tr><td>BinPacker</td><td>72.6<sub>0.24</sub></td><td>22611<sub>0.0</sub></td><td>5603<sub>0.98</sub></td><td>4.629<sub>0.0</sub></td><td>2335.729<sub>1.0</sub></td><td>2824<sub>0.39</sub></td><td>2.389<sub>0.01</sub></td><td>2381<sub>1.0</sub></td><td>4456<sub>0.26</sub></td><td>0.07303<sub>0.27</sub></td><td>42.56574<sub>0.04</sub></td><td>0.06956<sub>0.14</sub></td><td>0.83754<sub>0.11</sub></td><td>72918<sub>1.0</sub></td><td>0.1451<sub>0.0</sub></td><td>0.000374367<sub>0.0</sub></td><td>0.366908<sub>0.4</sub></td><td>-10.03<sub>0.62</sub></td><td>2009<sub>0.13</sub></td><td>4078<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>64.61<sub>0.0</sub></td><td>23516<sub>0.0</sub></td><td>302<sub>1.0</sub></td><td>0.666<sub>0.91</sub></td><td>487.106<sub>0.13</sub></td><td>709<sub>0.07</sub></td><td>1.012<sub>1.0</sub></td><td>708<sub>0.19</sub></td><td>2783<sub>0.0</sub></td><td>0.08326<sub>0.31</sub></td><td>52.45984<sub>0.76</sub></td><td>0.25039<sub>0.61</sub></td><td>0.02256<sub>1.0</sub></td><td>138699<sub>0.91</sub></td><td>0.552302<sub>0.86</sub></td><td>0.0168006<sub>0.08</sub></td><td>0.286957<sub>0.14</sub></td><td>-16.3<sub>0.0</sub></td><td>1682<sub>0.0</sub></td><td>2615<sub>0.63</sub></td></tr><td>Shannon</td><td>84.27<sub>0.58</sub></td><td>31328<sub>0.05</sub></td><td>2837<sub>0.99</sub></td><td>1.262<sub>0.77</sub></td><td>711.831<sub>0.23</sub></td><td>242<sub>0.0</sub></td><td>1.53<sub>0.63</sub></td><td>1324<sub>0.49</sub></td><td>6758<sub>0.63</sub></td><td>0.00373<sub>0.0</sub></td><td>55.69782<sub>1.0</sub></td><td>0.06634<sub>0.13</sub></td><td>0.503<sub>0.48</sub></td><td>117068<sub>0.94</sub></td><td>0.346262<sub>0.42</sub></td><td>0.0229751<sub>0.11</sub></td><td>0.420737<sub>0.58</sub></td><td>-8.96<sub>0.73</sub></td><td>3385<sub>0.7</sub></td><td>2133<sub>0.84</sub></td></tr><td>SPAdes_sc</td><td>92.04<sub>0.81</sub></td><td>31039<sub>0.05</sub></td><td>2022<sub>0.99</sub></td><td>0.803<sub>0.88</sub></td><td>410.224<sub>0.09</sub></td><td>1755<sub>0.23</sub></td><td>1.015<sub>1.0</sub></td><td>1186<sub>0.42</sub></td><td>5676<sub>0.46</sub></td><td>0.08042<sub>0.3</sub></td><td>46.12757<sub>0.3</sub></td><td>0.39665<sub>1.0</sub></td><td>0.03119<sub>0.99</sub></td><td>177477<sub>0.86</sub></td><td>0.606564<sub>0.97</sub></td><td>0.0140352<sub>0.07</sub></td><td>0.389103<sub>0.47</sub></td><td>-12.12<sub>0.42</sub></td><td>2625<sub>0.39</sub></td><td>2268<sub>0.78</sub></td></tr><td>SPAdes_rna</td><td>95.95<sub>0.93</sub></td><td>49860<sub>0.15</sub></td><td>5126<sub>0.98</sub></td><td>1.249<sub>0.78</sub></td><td>412.238<sub>0.09</sub></td><td>3253<sub>0.46</sub></td><td>1.192<sub>0.87</sub></td><td>782<sub>0.22</sub></td><td>7155<sub>0.69</sub></td><td>0.11119<sub>0.42</sub></td><td>46.25425<sub>0.31</sub></td><td>0.23201<sub>0.57</sub></td><td>0.21449<sub>0.79</sub></td><td>294083<sub>0.71</sub></td><td>0.619367<sub>1.0</sub></td><td>0.0128943<sub>0.06</sub></td><td>0.427956<sub>0.6</sub></td><td>-7.16<sub>0.91</sub></td><td>3089<sub>0.58</sub></td><td>1949<sub>0.92</sub></td></tr></tbody></table><h3><i>Homo sapiens</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 12.375/20</li><li>Oases: 6.307/20</li><li>Trans_ABySS: 14.235/20</li><li>SOAPdenovo_Trans: 11.92/20</li><li>Bridger: 11.128/20</li><li>BinPacker: 6.587/20</li><li>IDBA_Tran: 8.611/20</li><li>Shannon: 10.299/20</li><li>SPAdes_sc: 11.475/20</li><li>SPAdes_rna: 12.028/20</li></ul></br>Download <a href="data/main_metrics/csv/hsa.xlsx">Excel</a> | <a href="data/main_metrics/csv/hsa.csv">csv</a> | <a href="data/main_metrics/tex/hsa.pdf">PDF</a> | <a href="data/main_metrics/tex/hsa.tex">LaTeX</a> </br></br><h3><i>Homo sapiens + EBOV 3h</i> selected metrics</h3>
<table id="metricsTab6" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>90.73<sub>0.94</sub></td><td>27154<sub>0.26</sub></td><td>1580<sub>0.94</sub></td><td>1.728<sub>0.77</sub></td><td>1083.555<sub>0.25</sub></td><td>3826<sub>0.97</sub></td><td>1.697<sub>0.76</sub></td><td>2753<sub>0.6</sub></td><td>8101<sub>0.98</sub></td><td>0.08434<sub>0.67</sub></td><td>54.50638<sub>0.74</sub></td><td>0.18869<sub>0.3</sub></td><td>0.5966<sub>0.37</sub></td><td>109496<sub>0.67</sub></td><td>0.457458<sub>0.72</sub></td><td>0.0149362<sub>0.32</sub></td><td>0.518515<sub>0.93</sub></td><td>-1.24<sub>0.98</sub></td><td>3979<sub>1.0</sub></td><td>1829<sub>1.0</sub></td></tr><td>Oases</td><td>86.92<sub>0.88</sub></td><td>98196<sub>1.0</sub></td><td>24845<sub>0.0</sub></td><td>2.383<sub>0.66</sub></td><td>1308.216<sub>0.35</sub></td><td>3948<sub>1.0</sub></td><td>3.91<sub>0.0</sub></td><td>2461<sub>0.45</sub></td><td>8288<sub>1.0</sub></td><td>0.12328<sub>1.0</sub></td><td>48.70184<sub>0.32</sub></td><td>0.02109<sub>0.0</sub></td><td>0.93724<sub>0.0</sub></td><td>318936<sub>0.0</sub></td><td>0.213295<sub>0.29</sub></td><td>0.0149624<sub>0.32</sub></td><td>0.400049<sub>0.56</sub></td><td>-1.73<sub>0.7</sub></td><td>3803<sub>0.95</sub></td><td>1892<sub>0.98</sub></td></tr><td>Trans_ABySS</td><td>93.9<sub>1.0</sub></td><td>30767<sub>0.3</sub></td><td>7956<sub>0.68</sub></td><td>0.704<sub>0.95</sub></td><td>517.916<sub>0.02</sub></td><td>3153<sub>0.78</sub></td><td>1.554<sub>0.81</sub></td><td>1947<sub>0.18</sub></td><td>8072<sub>0.97</sub></td><td>0.11645<sub>0.94</sub></td><td>55.63372<sub>0.82</sub></td><td>0.1149<sub>0.17</sub></td><td>0.58088<sub>0.38</sub></td><td>127448<sub>0.61</sub></td><td>0.458425<sub>0.73</sub></td><td>0.0469628<sub>1.0</sub></td><td>0.541571<sub>1.0</sub></td><td>-1.2<sub>1.0</sub></td><td>3982<sub>1.0</sub></td><td>1838<sub>0.99</sub></td></tr><td>SOAPdenovo_Trans</td><td>89.8<sub>0.93</sub></td><td>13110<sub>0.12</sub></td><td>148<sub>1.0</sub></td><td>0.429<sub>1.0</sub></td><td>464.368<sub>0.0</sub></td><td>2553<sub>0.62</sub></td><td>1.042<sub>0.99</sub></td><td>2372<sub>0.4</sub></td><td>7219<sub>0.86</sub></td><td>0.07133<sub>0.57</sub></td><td>52.66849<sub>0.6</sub></td><td>0.35872<sub>0.61</sub></td><td>0.11018<sub>0.89</sub></td><td>62753<sub>0.82</sub></td><td>0.558308<sub>0.91</sub></td><td>0.0387229<sub>0.82</sub></td><td>0.452867<sub>0.72</sub></td><td>-1.35<sub>0.91</sub></td><td>3528<sub>0.88</sub></td><td>1934<sub>0.97</sub></td></tr><td>Bridger</td><td>89.95<sub>0.93</sub></td><td>20434<sub>0.19</sub></td><td>1853<sub>0.93</sub></td><td>1.43<sub>0.83</sub></td><td>925.619<sub>0.19</sub></td><td>2781<sub>0.68</sub></td><td>1.434<sub>0.85</sub></td><td>2676<sub>0.56</sub></td><td>7519<sub>0.9</sub></td><td>0.07058<sub>0.56</sub></td><td>51.84573<sub>0.54</sub></td><td>0.19928<sub>0.32</sub></td><td>0.47679<sub>0.5</sub></td><td>83187<sub>0.76</sub></td><td>0.491961<sub>0.79</sub></td><td>0.0134374<sub>0.29</sub></td><td>0.507537<sub>0.89</sub></td><td>-1.25<sub>0.97</sub></td><td>3879<sub>0.97</sub></td><td>1862<sub>0.99</sub></td></tr><td>BinPacker</td><td>36.6<sub>0.0</sub></td><td>1646<sub>0.0</sub></td><td>279<sub>0.99</sub></td><td>6.184<sub>0.0</sub></td><td>2899.392<sub>1.0</sub></td><td>255<sub>0.0</sub></td><td>2.014<sub>0.65</sub></td><td>3530<sub>1.0</sub></td><td>496<sub>0.0</sub></td><td>0.00361<sub>0.0</sub></td><td>44.28206<sub>0.0</sub></td><td>0.04723<sub>0.05</sub></td><td>0.83111<sub>0.11</sub></td><td>6752<sub>1.0</sub></td><td>0.0537161<sub>0.0</sub></td><td>6.61916e-05<sub>0.0</sub></td><td>0.216398<sub>0.0</sub></td><td>-2.96<sub>0.0</sub></td><td>205<sub>0.0</sub></td><td>5976<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>78.41<sub>0.73</sub></td><td>13508<sub>0.12</sub></td><td>178<sub>1.0</sub></td><td>0.799<sub>0.94</sub></td><td>718.402<sub>0.11</sub></td><td>569<sub>0.09</sub></td><td>1.005<sub>1.0</sub></td><td>1924<sub>0.17</sub></td><td>3121<sub>0.34</sub></td><td>0.0692<sub>0.55</sub></td><td>54.46886<sub>0.73</sub></td><td>0.42677<sub>0.73</sub></td><td>0.00906<sub>1.0</sub></td><td>54494<sub>0.85</sub></td><td>0.552636<sub>0.9</sub></td><td>0.0135966<sub>0.29</sub></td><td>0.435537<sub>0.67</sub></td><td>-1.84<sub>0.64</sub></td><td>2162<sub>0.52</sub></td><td>2216<sub>0.9</sub></td></tr><td>Shannon</td><td>90.23<sub>0.94</sub></td><td>28248<sub>0.28</sub></td><td>2161<sub>0.92</sub></td><td>1.313<sub>0.85</sub></td><td>921.046<sub>0.19</sub></td><td>2303<sub>0.55</sub></td><td>1.808<sub>0.72</sub></td><td>2236<sub>0.33</sub></td><td>7040<sub>0.84</sub></td><td>0.11089<sub>0.9</sub></td><td>58.16531<sub>1.0</sub></td><td>0.05062<sub>0.05</sub></td><td>0.69315<sub>0.26</sub></td><td>104126<sub>0.69</sub></td><td>0.388568<sub>0.6</sub></td><td>0.0279902<sub>0.6</sub></td><td>0.516653<sub>0.92</sub></td><td>-1.26<sub>0.97</sub></td><td>3575<sub>0.89</sub></td><td>2096<sub>0.93</sub></td></tr><td>SPAdes_sc</td><td>91.41<sub>0.96</sub></td><td>12775<sub>0.12</sub></td><td>1262<sub>0.95</sub></td><td>0.944<sub>0.91</sub></td><td>661.322<sub>0.08</sub></td><td>2653<sub>0.65</sub></td><td>1.003<sub>1.0</sub></td><td>2735<sub>0.59</sub></td><td>7537<sub>0.9</sub></td><td>0.07729<sub>0.62</sub></td><td>51.50764<sub>0.52</sub></td><td>0.57475<sub>1.0</sub></td><td>0.011<sub>1.0</sub></td><td>60900<sub>0.83</sub></td><td>0.571439<sub>0.93</sub></td><td>0.0130479<sub>0.28</sub></td><td>0.493276<sub>0.85</sub></td><td>-1.32<sub>0.93</sub></td><td>3890<sub>0.98</sub></td><td>1813<sub>1.0</sub></td></tr><td>SPAdes_rna</td><td>93.83<sub>1.0</sub></td><td>17195<sub>0.16</sub></td><td>943<sub>0.97</sub></td><td>1.001<sub>0.9</sub></td><td>462.087<sub>0.0</sub></td><td>3335<sub>0.83</sub></td><td>1.117<sub>0.96</sub></td><td>1603<sub>0.0</sub></td><td>7410<sub>0.89</sub></td><td>0.08445<sub>0.68</sub></td><td>50.27042<sub>0.43</sub></td><td>0.35252<sub>0.6</sub></td><td>0.16138<sub>0.84</sub></td><td>94274<sub>0.72</sub></td><td>0.610776<sub>1.0</sub></td><td>0.022366<sub>0.48</sub></td><td>0.542847<sub>1.0</sub></td><td>-1.2<sub>1.0</sub></td><td>3655<sub>0.91</sub></td><td>1851<sub>0.99</sub></td></tr></tbody></table><h3><i>Homo sapiens + EBOV 3h</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 14.173/20</li><li>Oases: 10.449/20</li><li>Trans_ABySS: 14.35/20</li><li>SOAPdenovo_Trans: 14.63/20</li><li>Bridger: 13.64/20</li><li>BinPacker: 4.808/20</li><li>IDBA_Tran: 12.255/20</li><li>Shannon: 13.418/20</li><li>SPAdes_sc: 15.082/20</li><li>SPAdes_rna: 14.351/20</li></ul></br>Download <a href="data/main_metrics/csv/ebov_hsa_3h.xlsx">Excel</a> | <a href="data/main_metrics/csv/ebov_hsa_3h.csv">csv</a> | <a href="data/main_metrics/tex/ebov_hsa_3h.pdf">PDF</a> | <a href="data/main_metrics/tex/ebov_hsa_3h.tex">LaTeX</a> </br></br><h3><i>Homo sapiens + EBOV 7h</i> selected metrics</h3>
<table id="metricsTab7" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>92.58<sub>0.91</sub></td><td>31319<sub>0.16</sub></td><td>1294<sub>0.94</sub></td><td>1.746<sub>0.6</sub></td><td>1040.875<sub>0.29</sub></td><td>4696<sub>0.9</sub></td><td>1.757<sub>0.78</sub></td><td>2839<sub>0.56</sub></td><td>8458<sub>0.93</sub></td><td>0.08296<sub>0.37</sub></td><td>52.15256<sub>0.77</sub></td><td>0.19023<sub>0.35</sub></td><td>0.5822<sub>0.39</sub></td><td>131284<sub>0.82</sub></td><td>0.461589<sub>0.64</sub></td><td>0.0133497<sub>0.27</sub></td><td>0.548502<sub>0.87</sub></td><td>-1.62<sub>1.0</sub></td><td>4119<sub>1.0</sub></td><td>1759<sub>1.0</sub></td></tr><td>Oases</td><td>85.78<sub>0.53</sub></td><td>115596<sub>1.0</sub></td><td>18893<sub>0.0</sub></td><td>2.515<sub>0.36</sub></td><td>1364.377<sub>0.44</sub></td><td>5115<sub>1.0</sub></td><td>4.347<sub>0.0</sub></td><td>3003<sub>0.64</sub></td><td>8861<sub>1.0</sub></td><td>0.12149<sub>0.9</sub></td><td>43.81502<sub>0.0</sub></td><td>0.01572<sub>0.0</sub></td><td>0.94707<sub>0.0</sub></td><td>413418<sub>0.0</sub></td><td>0.204378<sub>0.0</sub></td><td>0.01377<sub>0.28</sub></td><td>0.446502<sub>0.1</sub></td><td>-2.35<sub>0.38</sub></td><td>4055<sub>0.97</sub></td><td>1817<sub>0.91</sub></td></tr><td>Trans_ABySS</td><td>94.22<sub>1.0</sub></td><td>37410<sub>0.22</sub></td><td>4769<sub>0.75</sub></td><td>0.753<sub>0.9</sub></td><td>534.402<sub>0.04</sub></td><td>4004<sub>0.74</sub></td><td>1.663<sub>0.8</sub></td><td>2039<sub>0.19</sub></td><td>8405<sub>0.92</sub></td><td>0.12851<sub>1.0</sub></td><td>51.96626<sub>0.75</sub></td><td>0.07246<sub>0.12</sub></td><td>0.61566<sub>0.35</sub></td><td>159092<sub>0.74</sub></td><td>0.456358<sub>0.63</sub></td><td>0.0474134<sub>1.0</sub></td><td>0.565015<sub>1.0</sub></td><td>-1.64<sub>0.98</sub></td><td>4121<sub>1.0</sub></td><td>1771<sub>0.98</sub></td></tr><td>SOAPdenovo_Trans</td><td>90.52<sub>0.79</sub></td><td>15251<sub>0.0</sub></td><td>125<sub>1.0</sub></td><td>0.443<sub>1.0</sub></td><td>455.959<sub>0.0</sub></td><td>3225<sub>0.56</sub></td><td>1.06<sub>0.98</sub></td><td>2385<sub>0.35</sub></td><td>7569<sub>0.77</sub></td><td>0.07571<sub>0.27</sub></td><td>49.37891<sub>0.51</sub></td><td>0.3314<sub>0.64</sub></td><td>0.13479<sub>0.87</sub></td><td>77171<sub>0.97</sub></td><td>0.552421<sub>0.86</sub></td><td>0.0382348<sub>0.8</sub></td><td>0.489643<sub>0.43</sub></td><td>-1.85<sub>0.81</sub></td><td>3667<sub>0.77</sub></td><td>1855<sub>0.85</sub></td></tr><td>Bridger</td><td>89.14<sub>0.72</sub></td><td>23487<sub>0.08</sub></td><td>2509<sub>0.87</sub></td><td>1.487<sub>0.68</sub></td><td>866.608<sub>0.2</sub></td><td>2965<sub>0.5</sub></td><td>1.451<sub>0.87</sub></td><td>2745<sub>0.52</sub></td><td>7540<sub>0.77</sub></td><td>0.07521<sub>0.26</sub></td><td>47.63959<sub>0.35</sub></td><td>0.17706<sub>0.33</sub></td><td>0.46934<sub>0.51</sub></td><td>101788<sub>0.9</sub></td><td>0.489854<sub>0.71</sub></td><td>0.0115023<sub>0.23</sub></td><td>0.509876<sub>0.58</sub></td><td>-1.8<sub>0.85</sub></td><td>3979<sub>0.93</sub></td><td>1785<sub>0.96</sub></td></tr><td>BinPacker</td><td>84.63<sub>0.46</sub></td><td>22928<sub>0.08</sub></td><td>2649<sub>0.87</sub></td><td>3.674<sub>0.0</sub></td><td>2505.426<sub>1.0</sub></td><td>2851<sub>0.48</sub></td><td>1.884<sub>0.74</sub></td><td>3764<sub>1.0</sub></td><td>6844<sub>0.64</sub></td><td>0.05603<sub>0.0</sub></td><td>47.6602<sub>0.35</sub></td><td>0.11378<sub>0.2</sub></td><td>0.75862<sub>0.2</sub></td><td>82239<sub>0.96</sub></td><td>0.303781<sub>0.25</sub></td><td>0.000981932<sub>0.0</sub></td><td>0.498755<sub>0.5</sub></td><td>-1.95<sub>0.72</sub></td><td>3590<sub>0.73</sub></td><td>2392<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>76.39<sub>0.0</sub></td><td>16181<sub>0.01</sub></td><td>203<sub>1.0</sub></td><td>0.834<sub>0.88</sub></td><td>697.748<sub>0.12</sub></td><td>780<sub>0.0</sub></td><td>1.007<sub>1.0</sub></td><td>1967<sub>0.15</sub></td><td>3197<sub>0.0</sub></td><td>0.07533<sub>0.27</sub></td><td>50.41416<sub>0.61</sub></td><td>0.38971<sub>0.76</sub></td><td>0.01116<sub>1.0</sub></td><td>67564<sub>1.0</sub></td><td>0.555899<sub>0.87</sub></td><td>0.0122321<sub>0.24</sub></td><td>0.433572<sub>0.0</sub></td><td>-2.8<sub>0.0</sub></td><td>2134<sub>0.0</sub></td><td>2217<sub>0.28</sub></td></tr><td>Shannon</td><td>90.99<sub>0.82</sub></td><td>35785<sub>0.2</sub></td><td>2605<sub>0.87</sub></td><td>1.426<sub>0.7</sub></td><td>918.854<sub>0.23</sub></td><td>2964<sub>0.5</sub></td><td>1.954<sub>0.72</sub></td><td>2328<sub>0.32</sub></td><td>7524<sub>0.76</sub></td><td>0.1225<sub>0.92</sub></td><td>54.6887<sub>1.0</sub></td><td>0.03624<sub>0.04</sub></td><td>0.70965<sub>0.25</sub></td><td>134108<sub>0.81</sub></td><td>0.373245<sub>0.42</sub></td><td>0.0303068<sub>0.63</sub></td><td>0.551822<sub>0.9</sub></td><td>-1.74<sub>0.9</sub></td><td>3777<sub>0.83</sub></td><td>1955<sub>0.69</sub></td></tr><td>SPAdes_sc</td><td>91.94<sub>0.87</sub></td><td>15598<sub>0.0</sub></td><td>2042<sub>0.9</sub></td><td>0.72<sub>0.91</sub></td><td>489.2<sub>0.02</sub></td><td>2795<sub>0.46</sub></td><td>1.013<sub>1.0</sub></td><td>2766<sub>0.53</sub></td><td>7004<sub>0.67</sub></td><td>0.07618<sub>0.28</sub></td><td>48.01043<sub>0.39</sub></td><td>0.50802<sub>1.0</sub></td><td>0.02529<sub>0.98</sub></td><td>76646<sub>0.97</sub></td><td>0.570306<sub>0.91</sub></td><td>0.0139344<sub>0.28</sub></td><td>0.481227<sub>0.36</sub></td><td>-2.19<sub>0.52</sub></td><td>3644<sub>0.76</sub></td><td>1835<sub>0.88</sub></td></tr><td>SPAdes_rna</td><td>93.4<sub>0.95</sub></td><td>21867<sub>0.07</sub></td><td>1219<sub>0.94</sub></td><td>1.067<sub>0.81</sub></td><td>470.161<sub>0.01</sub></td><td>3850<sub>0.71</sub></td><td>1.161<sub>0.95</sub></td><td>1647<sub>0.0</sub></td><td>7226<sub>0.71</sub></td><td>0.09082<sub>0.48</sub></td><td>45.99342<sub>0.2</sub></td><td>0.29863<sub>0.57</sub></td><td>0.20012<sub>0.8</sub></td><td>120597<sub>0.85</sub></td><td>0.6074<sub>1.0</sub></td><td>0.0208343<sub>0.43</sub></td><td>0.555596<sub>0.93</sub></td><td>-1.71<sub>0.92</sub></td><td>3542<sub>0.71</sub></td><td>1780<sub>0.97</sub></td></tr></tbody></table><h3><i>Homo sapiens + EBOV 7h</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 13.535/20</li><li>Oases: 8.503/20</li><li>Trans_ABySS: 14.111/20</li><li>SOAPdenovo_Trans: 13.243/20</li><li>Bridger: 11.81/20</li><li>BinPacker: 9.171/20</li><li>IDBA_Tran: 8.177/20</li><li>Shannon: 12.505/20</li><li>SPAdes_sc: 12.696/20</li><li>SPAdes_rna: 13.004/20</li></ul></br>Download <a href="data/main_metrics/csv/ebov_hsa_7h.xlsx">Excel</a> | <a href="data/main_metrics/csv/ebov_hsa_7h.csv">csv</a> | <a href="data/main_metrics/tex/ebov_hsa_7h.pdf">PDF</a> | <a href="data/main_metrics/tex/ebov_hsa_7h.tex">LaTeX</a> </br></br><h3><i>Homo sapiens + EBOV 23h</i> selected metrics</h3>
<table id="metricsTab8" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>94.39<sub>0.98</sub></td><td>27785<sub>0.23</sub></td><td>1066<sub>0.91</sub></td><td>1.684<sub>0.63</sub></td><td>1143.492<sub>0.26</sub></td><td>4065<sub>0.93</sub></td><td>1.806<sub>0.73</sub></td><td>2958<sub>0.55</sub></td><td>7935<sub>1.0</sub></td><td>0.07324<sub>0.51</sub></td><td>53.73373<sub>0.46</sub></td><td>0.14025<sub>0.3</sub></td><td>0.60669<sub>0.36</sub></td><td>115578<sub>0.72</sub></td><td>0.44378<sub>0.6</sub></td><td>0.0124217<sub>0.27</sub></td><td>0.767084<sub>0.98</sub></td><td>-1.43<sub>1.0</sub></td><td>3885<sub>0.99</sub></td><td>1944<sub>1.0</sub></td></tr><td>Oases</td><td>70.05<sub>0.6</sub></td><td>83530<sub>1.0</sub></td><td>11194<sub>0.0</sub></td><td>2.311<sub>0.44</sub></td><td>1500.776<sub>0.39</sub></td><td>4330<sub>1.0</sub></td><td>3.97<sub>0.0</sub></td><td>3115<sub>0.62</sub></td><td>2868<sub>0.0</sub></td><td>0.10789<sub>0.89</sub></td><td>47.1938<sub>0.02</sub></td><td>0.03191<sub>0.0</sub></td><td>0.93811<sub>0.0</sub></td><td>301780<sub>0.0</sub></td><td>0.218292<sub>0.04</sub></td><td>0.0121553<sub>0.27</sub></td><td>0.20915<sub>0.17</sub></td><td>-3.87<sub>0.31</sub></td><td>3832<sub>0.97</sub></td><td>2014<sub>0.97</sub></td></tr><td>Trans_ABySS</td><td>95.49<sub>1.0</sub></td><td>32866<sub>0.3</sub></td><td>3253<sub>0.72</sub></td><td>0.769<sub>0.91</sub></td><td>578.881<sub>0.04</sub></td><td>3575<sub>0.79</sub></td><td>1.644<sub>0.78</sub></td><td>2178<sub>0.21</sub></td><td>7872<sub>0.99</sub></td><td>0.11747<sub>1.0</sub></td><td>53.73122<sub>0.46</sub></td><td>0.05168<sub>0.05</sub></td><td>0.61498<sub>0.35</sub></td><td>131712<sub>0.66</sub></td><td>0.448004<sub>0.61</sub></td><td>0.044702<sub>1.0</sub></td><td>0.779996<sub>1.0</sub></td><td>-1.45<sub>0.99</sub></td><td>3907<sub>1.0</sub></td><td>1954<sub>1.0</sub></td></tr><td>SOAPdenovo_Trans</td><td>93.1<sub>0.96</sub></td><td>13418<sub>0.04</sub></td><td>94<sub>1.0</sub></td><td>0.468<sub>1.0</sub></td><td>482.257<sub>0.0</sub></td><td>2749<sub>0.56</sub></td><td>1.049<sub>0.98</sub></td><td>2489<sub>0.34</sub></td><td>7122<sub>0.84</sub></td><td>0.06869<sub>0.46</sub></td><td>51.41692<sub>0.3</sub></td><td>0.39598<sub>1.0</sub></td><td>0.12649<sub>0.88</sub></td><td>64547<sub>0.92</sub></td><td>0.547701<sub>0.86</sub></td><td>0.0354917<sub>0.79</sub></td><td>0.694346<sub>0.87</sub></td><td>-1.6<sub>0.95</sub></td><td>3435<sub>0.79</sub></td><td>2042<sub>0.96</sub></td></tr><td>Bridger</td><td>92.43<sub>0.95</sub></td><td>20560<sub>0.13</sub></td><td>1844<sub>0.84</sub></td><td>1.43<sub>0.71</sub></td><td>929.857<sub>0.17</sub></td><td>2736<sub>0.56</sub></td><td>1.459<sub>0.85</sub></td><td>2841<sub>0.5</sub></td><td>7222<sub>0.86</sub></td><td>0.06884<sub>0.47</sub></td><td>48.88316<sub>0.13</sub></td><td>0.19635<sub>0.45</sub></td><td>0.48201<sub>0.49</sub></td><td>87621<sub>0.83</sub></td><td>0.47975<sub>0.69</sub></td><td>0.0109893<sub>0.24</sub></td><td>0.739796<sub>0.94</sub></td><td>-1.54<sub>0.97</sub></td><td>3784<sub>0.94</sub></td><td>1978<sub>0.99</sub></td></tr><td>BinPacker</td><td>81.57<sub>0.78</sub></td><td>10862<sub>0.0</sub></td><td>1436<sub>0.88</sub></td><td>3.755<sub>0.0</sub></td><td>3072.898<sub>1.0</sub></td><td>1523<sub>0.22</sub></td><td>2.09<sub>0.63</sub></td><td>3965<sub>1.0</sub></td><td>3262<sub>0.08</sub></td><td>0.02631<sub>0.0</sub></td><td>46.96748<sub>0.0</sub></td><td>0.0675<sub>0.1</sub></td><td>0.81643<sub>0.13</sub></td><td>43913<sub>1.0</sub></td><td>0.203836<sub>0.0</sub></td><td>0.00035399<sub>0.0</sub></td><td>0.706858<sub>0.89</sub></td><td>-2.02<sub>0.83</sub></td><td>1695<sub>0.0</sub></td><td>4392<sub>0.0</sub></td></tr><td>IDBA_Tran</td><td>48.37<sub>0.27</sub></td><td>14196<sub>0.05</sub></td><td>134<sub>1.0</sub></td><td>0.866<sub>0.88</sub></td><td>725.58<sub>0.1</sub></td><td>714<sub>0.0</sub></td><td>1.006<sub>1.0</sub></td><td>2048<sub>0.15</sub></td><td>3126<sub>0.05</sub></td><td>0.06954<sub>0.47</sub></td><td>52.33453<sub>0.37</sub></td><td>0.24431<sub>0.58</sub></td><td>0.01096<sub>1.0</sub></td><td>57271<sub>0.95</sub></td><td>0.551236<sub>0.87</sub></td><td>0.0113647<sub>0.25</sub></td><td>0.161691<sub>0.1</sub></td><td>-4.31<sub>0.18</sub></td><td>2042<sub>0.16</sub></td><td>2419<sub>0.81</sub></td></tr><td>Shannon</td><td>31.29<sub>0.0</sub></td><td>15007<sub>0.06</sub></td><td>381<sub>0.97</sub></td><td>0.583<sub>0.97</sub></td><td>703.748<sub>0.09</sub></td><td>1631<sub>0.25</sub></td><td>1.435<sub>0.85</sub></td><td>1714<sub>0.0</sub></td><td>5349<sub>0.49</sub></td><td>0.07719<sub>0.56</sub></td><td>61.57702<sub>1.0</sub></td><td>0.04847<sub>0.05</sub></td><td>0.46502<sub>0.51</sub></td><td>60817<sub>0.93</sub></td><td>0.379485<sub>0.44</sub></td><td>0.0282574<sub>0.63</sub></td><td>0.0955414<sub>0.0</sub></td><td>-4.95<sub>0.0</sub></td><td>2852<sub>0.52</sub></td><td>2764<sub>0.67</sub></td></tr><td>SPAdes_sc</td><td>91.15<sub>0.93</sub></td><td>13245<sub>0.03</sub></td><td>1206<sub>0.9</sub></td><td>0.951<sub>0.85</sub></td><td>649.419<sub>0.07</sub></td><td>2654<sub>0.54</sub></td><td>1.004<sub>1.0</sub></td><td>3071<sub>0.6</sub></td><td>7101<sub>0.84</sub></td><td>0.06837<sub>0.46</sub></td><td>49.53084<sub>0.18</sub></td><td>0.38949<sub>0.98</sub></td><td>0.01877<sub>0.99</sub></td><td>63872<sub>0.92</sub></td><td>0.564203<sub>0.91</sub></td><td>0.0105064<sub>0.23</sub></td><td>0.247863<sub>0.22</sub></td><td>-3.6<sub>0.38</sub></td><td>3710<sub>0.91</sub></td><td>1952<sub>1.0</sub></td></tr><td>SPAdes_rna</td><td>93.12<sub>0.96</sub></td><td>18629<sub>0.11</sub></td><td>908<sub>0.93</sub></td><td>1.15<sub>0.79</sub></td><td>476.176<sub>0.0</sub></td><td>3535<sub>0.78</sub></td><td>1.147<sub>0.95</sub></td><td>2203<sub>0.22</sub></td><td>6960<sub>0.81</sub></td><td>0.08193<sub>0.61</sub></td><td>47.53883<sub>0.04</sub></td><td>0.21737<sub>0.51</sub></td><td>0.19225<sub>0.8</sub></td><td>102095<sub>0.77</sub></td><td>0.601376<sub>1.0</sub></td><td>0.0196731<sub>0.44</sub></td><td>0.589338<sub>0.72</sub></td><td>-1.67<sub>0.93</sub></td><td>3405<sub>0.77</sub></td><td>1992<sub>0.98</sub></td></tr></tbody></table><h3><i>Homo sapiens + EBOV 23h</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 13.426/20</li><li>Oases: 7.683/20</li><li>Trans_ABySS: 13.865/20</li><li>SOAPdenovo_Trans: 14.527/20</li><li>Bridger: 12.722/20</li><li>BinPacker: 7.552/20</li><li>IDBA_Tran: 9.219/20</li><li>Shannon: 8.989/20</li><li>SPAdes_sc: 12.942/20</li><li>SPAdes_rna: 13.126/20</li></ul></br>Download <a href="data/main_metrics/csv/ebov_hsa_23h.xlsx">Excel</a> | <a href="data/main_metrics/csv/ebov_hsa_23h.csv">csv</a> | <a href="data/main_metrics/tex/ebov_hsa_23h.pdf">PDF</a> | <a href="data/main_metrics/tex/ebov_hsa_23h.tex">LaTeX</a> </br></br><h3><i>Homo sapiens simulated</i> selected metrics</h3>
<table id="metricsTab9" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="50%">
<thead><tr><th></th><th>Overall mapping rate</th><th>Transcripts >=1000nt</th><th>Misassemblies</th><th>Mismatches per transcript</th><th>Average alignment length</th><th>95%-assembled isoforms</th><th>Duplication ratio</th><th>Ex90N50</th><th># full-length transcripts</th><th>Reference coverage</th><th>Mean ORF percentage</th><th>Optimal score</th><th>Percentage bases uncovered</th><th>Number of ambiguous bases</th><th>Nucleotide F1</th><th>Contig F1</th><th>KC score</th><th>RSEM EVAL</th><th>Complete BUSCOs</th><th>Missing BUSCOs</th></tr></thead><tbody><tr><td>Trinity</td><td>96.39<sub>0.95</sub></td><td>8451<sub>0.24</sub></td><td>139<sub>0.97</sub></td><td>1.231<sub>0.44</sub></td><td>2261.332<sub>1.0</sub></td><td>2898<sub>0.96</sub></td><td>2.154<sub>0.7</sub></td><td>3234<sub>0.96</sub></td><td>1492<sub>0.85</sub></td><td>0.24396<sub>0.34</sub></td><td>43.37841<sub>0.63</sub></td><td>0.11168<sub>0.22</sub></td><td>0.50236<sub>0.4</sub></td><td>30990<sub>0.77</sub></td><td>0.566043<sub>0.61</sub></td><td>0.0633606<sub>0.31</sub></td><td>0.883827<sub>0.94</sub></td><td>-2.79<sub>0.96</sub></td><td>588<sub>0.94</sub></td><td>26<sub>0.99</sub></td></tr><td>Oases</td><td>73.26<sub>0.62</sub></td><td>28143<sub>1.0</sub></td><td>4094<sub>0.0</sub></td><td>2.105<sub>0.0</sub></td><td>2090.55<sub>0.88</sub></td><td>2995<sub>1.0</sub></td><td>4.854<sub>0.0</sub></td><td>3300<sub>0.99</sub></td><td>1672<sub>1.0</sub></td><td>0.37548<sub>0.74</sub></td><td>37.42736<sub>0.1</sub></td><td>0.0107<sub>0.0</sub></td><td>0.82981<sub>0.0</sub></td><td>110723<sub>0.0</sub></td><td>0.223824<sub>0.0</sub></td><td>0.0488742<sub>0.03</sub></td><td>0.738406<sub>0.72</sub></td><td>-4.62<sub>0.75</sub></td><td>613<sub>1.0</sub></td><td>22<sub>1.0</sub></td></tr><td>Trans_ABySS</td><td>99.56<sub>1.0</sub></td><td>6703<sub>0.17</sub></td><td>117<sub>0.97</sub></td><td>0.459<sub>0.82</sub></td><td>939.312<sub>0.06</sub></td><td>2877<sub>0.95</sub></td><td>1.572<sub>0.85</sub></td><td>2902<sub>0.78</sub></td><td>1323<sub>0.71</sub></td><td>0.45944<sub>1.0</sub></td><td>45.49152<sub>0.81</sub></td><td>0.22431<sub>0.47</sub></td><td>0.31668<sub>0.63</sub></td><td>24282<sub>0.84</sub></td><td>0.679067<sub>0.81</sub></td><td>0.0987237<sub>1.0</sub></td><td>0.923633<sub>1.0</sub></td><td>-2.38<sub>1.0</sub></td><td>537<sub>0.8</sub></td><td>21<sub>1.0</sub></td></tr><td>SOAPdenovo_Trans</td><td>91.68<sub>0.89</sub></td><td>4263<sub>0.07</sub></td><td>49<sub>0.99</sub></td><td>0.308<sub>0.89</sub></td><td>1009.08<sub>0.11</sub></td><td>1679<sub>0.46</sub></td><td>1.274<sub>0.93</sub></td><td>2836<sub>0.75</sub></td><td>1006<sub>0.45</sub></td><td>0.18247<sub>0.16</sub></td><td>36.27748<sub>0.0</sub></td><td>0.20541<sub>0.43</sub></td><td>0.16758<sub>0.81</sub></td><td>15259<sub>0.93</sub></td><td>0.733277<sub>0.9</sub></td><td>0.0752428<sub>0.54</sub></td><td>0.595837<sub>0.5</sub></td><td>-5.27<sub>0.68</sub></td><td>289<sub>0.16</sub></td><td>109<sub>0.74</sub></td></tr><td>Bridger</td><td>94.02<sub>0.92</sub></td><td>5167<sub>0.11</sub></td><td>533<sub>0.87</sub></td><td>1.652<sub>0.23</sub></td><td>1581.299<sub>0.51</sub></td><td>1570<sub>0.41</sub></td><td>1.507<sub>0.87</sub></td><td>3252<sub>0.97</sub></td><td>1210<sub>0.62</sub></td><td>0.1877<sub>0.17</sub></td><td>42.21987<sub>0.52</sub></td><td>0.14218<sub>0.29</sub></td><td>0.22125<sub>0.74</sub></td><td>18025<sub>0.9</sub></td><td>0.710854<sub>0.86</sub></td><td>0.04805<sub>0.01</sub></td><td>0.81708<sub>0.84</sub></td><td>-3.66<sub>0.86</sub></td><td>525<sub>0.77</sub></td><td>28<sub>0.98</sub></td></tr><td>BinPacker</td><td>93.03<sub>0.91</sub></td><td>7424<sub>0.2</sub></td><td>785<sub>0.81</sub></td><td>1.849<sub>0.13</sub></td><td>1755.155<sub>0.64</sub></td><td>1660<sub>0.45</sub></td><td>1.949<sub>0.76</sub></td><td>2909<sub>0.79</sub></td><td>1246<sub>0.65</sub></td><td>0.20281<sub>0.22</sub></td><td>43.57058<sub>0.64</sub></td><td>0.12943<sub>0.26</sub></td><td>0.43188<sub>0.49</sub></td><td>25851<sub>0.82</sub></td><td>0.601458<sub>0.67</sub></td><td>0.0475004<sub>0.0</sub></td><td>0.821483<sub>0.85</sub></td><td>-3.65<sub>0.86</sub></td><td>527<sub>0.78</sub></td><td>29<sub>0.98</sub></td></tr><td>IDBA_Tran</td><td>85.34<sub>0.79</sub></td><td>2740<sub>0.02</sub></td><td>8<sub>1.0</sub></td><td>0.096<sub>1.0</sub></td><td>859.966<sub>0.0</sub></td><td>577<sub>0.0</sub></td><td>1.013<sub>1.0</sub></td><td>1397<sub>0.0</sub></td><td>450<sub>0.0</sub></td><td>0.17764<sub>0.14</sub></td><td>47.6202<sub>1.0</sub></td><td>0.34794<sub>0.74</sub></td><td>0.01292<sub>1.0</sub></td><td>10489<sub>0.97</sub></td><td>0.783647<sub>0.99</sub></td><td>0.0504635<sub>0.06</sub></td><td>0.59977<sub>0.51</sub></td><td>-7.53<sub>0.44</sub></td><td>226<sub>0.0</sub></td><td>142<sub>0.65</sub></td></tr><td>Shannon</td><td>30.77<sub>0.0</sub></td><td>2341<sub>0.0</sub></td><td>66<sub>0.99</sub></td><td>0.394<sub>0.85</sub></td><td>1061.013<sub>0.14</sub></td><td>1056<sub>0.2</sub></td><td>1.437<sub>0.89</sub></td><td>2368<sub>0.51</sub></td><td>666<sub>0.18</sub></td><td>0.13061<sub>0.0</sub></td><td>47.36729<sub>0.98</sub></td><td>0.05617<sub>0.1</sub></td><td>0.17449<sub>0.8</sub></td><td>7801<sub>1.0</sub></td><td>0.453332<sub>0.41</sub></td><td>0.0695831<sub>0.43</sub></td><td>0.261483<sub>0.0</sub></td><td>-11.51<sub>0.0</sub></td><td>242<sub>0.04</sub></td><td>364<sub>0.0</sub></td></tr><td>SPAdes_sc</td><td>97.02<sub>0.96</sub></td><td>2623<sub>0.01</sub></td><td>50<sub>0.99</sub></td><td>0.215<sub>0.94</sub></td><td>979.981<sub>0.09</sub></td><td>1625<sub>0.43</sub></td><td>1.012<sub>1.0</sub></td><td>3315<sub>1.0</sub></td><td>1007<sub>0.46</sub></td><td>0.16027<sub>0.09</sub></td><td>39.61807<sub>0.29</sub></td><td>0.46692<sub>1.0</sub></td><td>0.0122<sub>1.0</sub></td><td>10674<sub>0.97</sub></td><td>0.789225<sub>1.0</sub></td><td>0.0528581<sub>0.1</sub></td><td>0.773079<sub>0.77</sub></td><td>-4.78<sub>0.74</sub></td><td>407<sub>0.47</sub></td><td>55<sub>0.9</sub></td></tr><td>SPAdes_rna</td><td>96.72<sub>0.96</sub></td><td>6053<sub>0.14</sub></td><td>351<sub>0.92</sub></td><td>1.82<sub>0.14</sub></td><td>1649.147<sub>0.56</sub></td><td>2220<sub>0.68</sub></td><td>1.535<sub>0.86</sub></td><td>2549<sub>0.6</sub></td><td>1141<sub>0.57</sub></td><td>0.2459<sub>0.35</sub></td><td>40.38284<sub>0.36</sub></td><td>0.19116<sub>0.4</sub></td><td>0.24767<sub>0.71</sub></td><td>20302<sub>0.88</sub></td><td>0.695881<sub>0.83</sub></td><td>0.0541619<sub>0.13</sub></td><td>0.785485<sub>0.79</sub></td><td>-3.91<sub>0.83</sub></td><td>462<sub>0.61</sub></td><td>26<sub>0.99</sub></td></tr></tbody></table><h3><i>Homo sapiens simulated</i> summary of (0,1)-normalized scores</h3><ul>
<li>Trinity: 14.165/20</li><li>Oases: 9.833/20</li><li>Trans_ABySS: 15.68/20</li><li>SOAPdenovo_Trans: 11.403/20</li><li>Bridger: 12.466/20</li><li>BinPacker: 11.885/20</li><li>IDBA_Tran: 10.306/20</li><li>Shannon: 7.509/20</li><li>SPAdes_sc: 13.219/20</li><li>SPAdes_rna: 12.316/20</li></ul></br>Download <a href="data/main_metrics/csv/hsa_flux.xlsx">Excel</a> | <a href="data/main_metrics/csv/hsa_flux.csv">csv</a> | <a href="data/main_metrics/tex/hsa_flux.pdf">PDF</a> | <a href="data/main_metrics/tex/hsa_flux.tex">LaTeX</a> </br></br><a name="runtime">
<h1>S11: Runtime and memory consumption</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>Shown are the runtime and max memory peak for each assembler and data set, run with 48 threads. For Trinity, we observed high memory peaks in the first few minutes of assembly. Here, we removed this high maximum memory peaks from the plots to show a better comparison of the memory usage over time.</p><div id='vis-414b2471-ea89-4fcd-8417-c10fefa70d6a'></div>
<script>
(function(){
    var render = function(){
        var model = {"panes":[{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"b294803d-fad7-4f21-ac30-81275bcceb60"}],"options":{"x_label":"Escherichia coli","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,28]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"dc010487-fff7-44da-bf1f-1f18b202829a"}],"options":{"x_label":"Escherichia coli","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,7.2]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"8b156fd9-424f-4e30-96d2-58c349f8bd12"}],"options":{"x_label":"Candida albicans","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,109]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"55605d34-ea69-4573-a1b5-242e2632baca"}],"options":{"x_label":"Candida albicans","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,19.2]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"e77963e0-3772-47c0-9a18-2798e990fc92"}],"options":{"x_label":"Arabidopsis thaliana","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,175]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"5e9495f5-83ce-4258-a3ed-b3c55456293e"}],"options":{"x_label":"Arabidopsis thaliana","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,83.6]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"36f5e4ce-c9cb-40a1-87c0-d5f7fe079a0d"}],"options":{"x_label":"Mus musculus","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,671]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"ac718d0d-ed76-4ccd-b55e-862d75587e67"}],"options":{"x_label":"Mus musculus","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,40.3]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"253a3048-5b87-465e-bd6e-081699070bfa"}],"options":{"x_label":"H. sapiens","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,12465]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"e296a0b1-1538-474f-be16-dc9af0c1c304"}],"options":{"x_label":"H. sapiens","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,110.1]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"624dc0c4-c5e9-4129-a0f6-53558d47242f"}],"options":{"x_label":"H. sapiens+EBOV 3h","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,662]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"1cd62563-4158-44fb-9b34-cb9e28d516b6"}],"options":{"x_label":"H. sapiens+EBOV 3h","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,111.4]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"96f67eb9-80d2-450d-9f5c-b4a9f8224a02"}],"options":{"x_label":"H. sapiens+EBOV 7h","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,895]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"337655bc-7095-4452-896d-594be90b15df"}],"options":{"x_label":"H. sapiens+EBOV 7h","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,121.4]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"f3c89e4b-dee7-402a-af50-ee744edabc3c"}],"options":{"x_label":"H. sapiens+EBOV 23h","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,784]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"f1e916d2-bb88-4381-88d2-b64a05286c6d"}],"options":{"x_label":"H. sapiens+EBOV 23h","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,77.1]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"runtime_in_sec","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"d84b5f9d-35fe-4d94-b471-aece9342ea41"}],"options":{"x_label":"H. sapiens simulated","y_label":"runtime [minutes]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,1284]}},{"diagrams":[{"type":"bar","options":{"x":"tool","y":"max_memory_in_mb","color":["rgb(141,211,199)","rgb(255,255,179)","rgb(190,186,218)","rgb(251,128,114)","rgb(128,177,211)","rgb(253,180,98)","rgb(179,222,105)","rgb(252,205,229)","rgb(217,217,217)"]},"data":"e8e47dbd-3cd1-42c5-85de-2a38a45b348f"}],"options":{"x_label":"H. sapiens simulated","y_label":"memory [GB]","legend":true,"rotate_x_label":300,"margin":{"left":80,"right":40,"top":40,"bottom":160},"width":800,"xrange":["trinity","oases","transabyss","soap-trans","bridger","binpacker","idba-tran","shannon","spades","rnaspades"],"yrange":[0,119.2]}}],"data":{"b294803d-fad7-4f21-ac30-81275bcceb60":[{"tool":"trinity","runtime_in_sec":28},{"tool":"oases","runtime_in_sec":25},{"tool":"transabyss","runtime_in_sec":16},{"tool":"soap-trans","runtime_in_sec":1},{"tool":"bridger","runtime_in_sec":11},{"tool":"binpacker","runtime_in_sec":5},{"tool":"idba-tran","runtime_in_sec":7},{"tool":"shannon","runtime_in_sec":9},{"tool":"spades","runtime_in_sec":16},{"tool":"rnaspades","runtime_in_sec":11}],"dc010487-fff7-44da-bf1f-1f18b202829a":[{"tool":"trinity","max_memory_in_mb":7.2},{"tool":"oases","max_memory_in_mb":3.1},{"tool":"transabyss","max_memory_in_mb":0.6},{"tool":"soap-trans","max_memory_in_mb":2.1},{"tool":"bridger","max_memory_in_mb":1.6},{"tool":"binpacker","max_memory_in_mb":1.5},{"tool":"idba-tran","max_memory_in_mb":0.6},{"tool":"shannon","max_memory_in_mb":3.8},{"tool":"spades","max_memory_in_mb":5.0},{"tool":"rnaspades","max_memory_in_mb":5.0}],"8b156fd9-424f-4e30-96d2-58c349f8bd12":[{"tool":"trinity","runtime_in_sec":82},{"tool":"oases","runtime_in_sec":55},{"tool":"transabyss","runtime_in_sec":109},{"tool":"soap-trans","runtime_in_sec":5},{"tool":"bridger","runtime_in_sec":24},{"tool":"binpacker","runtime_in_sec":29},{"tool":"idba-tran","runtime_in_sec":19},{"tool":"shannon","runtime_in_sec":41},{"tool":"spades","runtime_in_sec":26},{"tool":"rnaspades","runtime_in_sec":23}],"55605d34-ea69-4573-a1b5-242e2632baca":[{"tool":"trinity","max_memory_in_mb":11.6},{"tool":"oases","max_memory_in_mb":5.6},{"tool":"transabyss","max_memory_in_mb":1.9},{"tool":"soap-trans","max_memory_in_mb":10.3},{"tool":"bridger","max_memory_in_mb":4.9},{"tool":"binpacker","max_memory_in_mb":4.7},{"tool":"idba-tran","max_memory_in_mb":2.1},{"tool":"shannon","max_memory_in_mb":19.2},{"tool":"spades","max_memory_in_mb":6.3},{"tool":"rnaspades","max_memory_in_mb":5.9}],"e77963e0-3772-47c0-9a18-2798e990fc92":[{"tool":"trinity","runtime_in_sec":175},{"tool":"oases","runtime_in_sec":107},{"tool":"transabyss","runtime_in_sec":160},{"tool":"soap-trans","runtime_in_sec":6},{"tool":"bridger","runtime_in_sec":62},{"tool":"binpacker","runtime_in_sec":56},{"tool":"idba-tran","runtime_in_sec":62},{"tool":"shannon","runtime_in_sec":141},{"tool":"spades","runtime_in_sec":38},{"tool":"rnaspades","runtime_in_sec":33}],"5e9495f5-83ce-4258-a3ed-b3c55456293e":[{"tool":"trinity","max_memory_in_mb":27.7},{"tool":"oases","max_memory_in_mb":9.1},{"tool":"transabyss","max_memory_in_mb":29.2},{"tool":"soap-trans","max_memory_in_mb":33.8},{"tool":"bridger","max_memory_in_mb":30.4},{"tool":"binpacker","max_memory_in_mb":16.0},{"tool":"idba-tran","max_memory_in_mb":10.4},{"tool":"shannon","max_memory_in_mb":83.6},{"tool":"spades","max_memory_in_mb":21.8},{"tool":"rnaspades","max_memory_in_mb":22.1}],"36f5e4ce-c9cb-40a1-87c0-d5f7fe079a0d":[{"tool":"trinity","runtime_in_sec":400},{"tool":"oases","runtime_in_sec":282},{"tool":"transabyss","runtime_in_sec":671},{"tool":"soap-trans","runtime_in_sec":36},{"tool":"bridger","runtime_in_sec":334},{"tool":"binpacker","runtime_in_sec":183},{"tool":"idba-tran","runtime_in_sec":134},{"tool":"shannon","runtime_in_sec":150},{"tool":"spades","runtime_in_sec":159},{"tool":"rnaspades","runtime_in_sec":177}],"ac718d0d-ed76-4ccd-b55e-862d75587e67":[{"tool":"trinity","max_memory_in_mb":14.1},{"tool":"oases","max_memory_in_mb":32.2},{"tool":"transabyss","max_memory_in_mb":19.7},{"tool":"soap-trans","max_memory_in_mb":26.5},{"tool":"bridger","max_memory_in_mb":37.6},{"tool":"binpacker","max_memory_in_mb":37.6},{"tool":"idba-tran","max_memory_in_mb":10.6},{"tool":"shannon","max_memory_in_mb":40.3},{"tool":"spades","max_memory_in_mb":20.8},{"tool":"rnaspades","max_memory_in_mb":20.7}],"253a3048-5b87-465e-bd6e-081699070bfa":[{"tool":"trinity","runtime_in_sec":2641},{"tool":"oases","runtime_in_sec":12465},{"tool":"transabyss","runtime_in_sec":3263},{"tool":"soap-trans","runtime_in_sec":108},{"tool":"bridger","runtime_in_sec":1271},{"tool":"binpacker","runtime_in_sec":957},{"tool":"idba-tran","runtime_in_sec":529},{"tool":"shannon","runtime_in_sec":645},{"tool":"spades","runtime_in_sec":473},{"tool":"rnaspades","runtime_in_sec":444}],"e296a0b1-1538-474f-be16-dc9af0c1c304":[{"tool":"trinity","max_memory_in_mb":35.7},{"tool":"oases","max_memory_in_mb":94.7},{"tool":"transabyss","max_memory_in_mb":42.5},{"tool":"soap-trans","max_memory_in_mb":45.6},{"tool":"bridger","max_memory_in_mb":109.3},{"tool":"binpacker","max_memory_in_mb":96.2},{"tool":"idba-tran","max_memory_in_mb":29.1},{"tool":"shannon","max_memory_in_mb":110.1},{"tool":"spades","max_memory_in_mb":37.4},{"tool":"rnaspades","max_memory_in_mb":44.2}],"624dc0c4-c5e9-4129-a0f6-53558d47242f":[{"tool":"trinity","runtime_in_sec":599},{"tool":"oases","runtime_in_sec":407},{"tool":"transabyss","runtime_in_sec":662},{"tool":"soap-trans","runtime_in_sec":19},{"tool":"bridger","runtime_in_sec":309},{"tool":"binpacker","runtime_in_sec":128},{"tool":"idba-tran","runtime_in_sec":164},{"tool":"shannon","runtime_in_sec":236},{"tool":"spades","runtime_in_sec":149},{"tool":"rnaspades","runtime_in_sec":146}],"1cd62563-4158-44fb-9b34-cb9e28d516b6":[{"tool":"trinity","max_memory_in_mb":10.9},{"tool":"oases","max_memory_in_mb":26.5},{"tool":"transabyss","max_memory_in_mb":19.7},{"tool":"soap-trans","max_memory_in_mb":26.4},{"tool":"bridger","max_memory_in_mb":22.6},{"tool":"binpacker","max_memory_in_mb":22.9},{"tool":"idba-tran","max_memory_in_mb":7.3},{"tool":"shannon","max_memory_in_mb":111.4},{"tool":"spades","max_memory_in_mb":23.7},{"tool":"rnaspades","max_memory_in_mb":26.8}],"96f67eb9-80d2-450d-9f5c-b4a9f8224a02":[{"tool":"trinity","runtime_in_sec":762},{"tool":"oases","runtime_in_sec":649},{"tool":"transabyss","runtime_in_sec":895},{"tool":"soap-trans","runtime_in_sec":24},{"tool":"bridger","runtime_in_sec":441},{"tool":"binpacker","runtime_in_sec":444},{"tool":"idba-tran","runtime_in_sec":233},{"tool":"shannon","runtime_in_sec":366},{"tool":"spades","runtime_in_sec":133},{"tool":"rnaspades","runtime_in_sec":130}],"337655bc-7095-4452-896d-594be90b15df":[{"tool":"trinity","max_memory_in_mb":15.0},{"tool":"oases","max_memory_in_mb":31.3},{"tool":"transabyss","max_memory_in_mb":20.0},{"tool":"soap-trans","max_memory_in_mb":26.7},{"tool":"bridger","max_memory_in_mb":30.4},{"tool":"binpacker","max_memory_in_mb":31.7},{"tool":"idba-tran","max_memory_in_mb":9.6},{"tool":"shannon","max_memory_in_mb":121.4},{"tool":"spades","max_memory_in_mb":24.4},{"tool":"rnaspades","max_memory_in_mb":24.7}],"f3c89e4b-dee7-402a-af50-ee744edabc3c":[{"tool":"trinity","runtime_in_sec":545},{"tool":"oases","runtime_in_sec":745},{"tool":"transabyss","runtime_in_sec":784},{"tool":"soap-trans","runtime_in_sec":27},{"tool":"bridger","runtime_in_sec":357},{"tool":"binpacker","runtime_in_sec":272},{"tool":"idba-tran","runtime_in_sec":240},{"tool":"shannon","runtime_in_sec":361},{"tool":"spades","runtime_in_sec":146},{"tool":"rnaspades","runtime_in_sec":137}],"f1e916d2-bb88-4381-88d2-b64a05286c6d":[{"tool":"trinity","max_memory_in_mb":11.4},{"tool":"oases","max_memory_in_mb":40.1},{"tool":"transabyss","max_memory_in_mb":49.2},{"tool":"soap-trans","max_memory_in_mb":25.8},{"tool":"bridger","max_memory_in_mb":28.8},{"tool":"binpacker","max_memory_in_mb":27.9},{"tool":"idba-tran","max_memory_in_mb":7.0},{"tool":"shannon","max_memory_in_mb":77.1},{"tool":"spades","max_memory_in_mb":20.6},{"tool":"rnaspades","max_memory_in_mb":28.8}],"d84b5f9d-35fe-4d94-b471-aece9342ea41":[{"tool":"trinity","runtime_in_sec":147},{"tool":"oases","runtime_in_sec":893},{"tool":"transabyss","runtime_in_sec":1284},{"tool":"soap-trans","runtime_in_sec":46},{"tool":"bridger","runtime_in_sec":241},{"tool":"binpacker","runtime_in_sec":386},{"tool":"idba-tran","runtime_in_sec":196},{"tool":"shannon","runtime_in_sec":198},{"tool":"spades","runtime_in_sec":207},{"tool":"rnaspades","runtime_in_sec":199}],"e8e47dbd-3cd1-42c5-85de-2a38a45b348f":[{"tool":"trinity","max_memory_in_mb":21.0},{"tool":"oases","max_memory_in_mb":119.2},{"tool":"transabyss","max_memory_in_mb":31.7},{"tool":"soap-trans","max_memory_in_mb":26.3},{"tool":"bridger","max_memory_in_mb":48.7},{"tool":"binpacker","max_memory_in_mb":50.3},{"tool":"idba-tran","max_memory_in_mb":16.9},{"tool":"shannon","max_memory_in_mb":113.3},{"tool":"spades","max_memory_in_mb":23.5},{"tool":"rnaspades","max_memory_in_mb":22.4}]},"extension":[]}
        var id_name = '#vis-414b2471-ea89-4fcd-8417-c10fefa70d6a';
        Nyaplot.core.parse(model, id_name);

        require(['downloadable'], function(downloadable){
          var svg = d3.select(id_name).select("svg");
	  if(!svg.empty())
	    svg.call(downloadable().filename('fig'));
	});
    };
    if(window['Nyaplot']==undefined){
        window.addEventListener('load_nyaplot', render, false);
	return;
    } else {
       render();
    }
})();
</script>
<a name="normscore">
<h1>S12: (0,1)-normalized scores per data set and metric</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>For each metric and data set, we calculated normalized scores in the range of 0 and 1 (see manuscript methods) to compare the performance of the different assembly tools regarding this metric.
    For example, consider the <i>E. coli</i> data set and the metric <i>Overall mapping rate</i> that results in the following vector:</p>


    <code>(77.01, 49.16, 95.67, 56.62, 87.35, 71.09, 34.31, 76.69, 88.04, 88.9)</code>


    <p>The calculated (0,1)-normalized scores are:</p>


    <code>(0.7, 0.24, 1.0, 0.36, 0.86, 0.6, 0.0, 0.69, 0.88, 0.89)</code>


    <p>With this transformation, we aim to achive a somewhat fair comparison of the different assembly tools and metrics.</p>
    <p>For each data set and assembly tool the scores are summed up to achieve a final score for comparison. Here we show a heat map for each data set representing the different 
    normalized scores calculated for each metric and assembly tool. We used this heat maps to identify potentially highly correlated metrics.</p><ul><li><a href="data/main_metrics/scores/eco_scores.pdf"><i>Escherichia coli</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/cal3_scores.pdf"><i>Candida albicans</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/ath_scores.pdf"><i>Arabidopsis thaliana</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/mmu_scores.pdf"><i>Mus musculus</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/hsa_scores.pdf"><i>Homo sapiens</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/ebov_hsa_3h_scores.pdf"><i>Homo sapiens + EBOV 3h</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/ebov_hsa_7h_scores.pdf"><i>Homo sapiens + EBOV 7h</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/ebov_hsa_23h_scores.pdf"><i>Homo sapiens + EBOV 23h</i> normalized score heat map (PDF)</a></li>
<li><a href="data/main_metrics/scores/hsa_flux_scores.pdf"><i>Homo sapiens simulated</i> normalized score heat map (PDF)</a></li>
</ul><br><br></div></body></html>
