<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>A paper per day</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="pandoc.css" />
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<nav id="TOC">
<ul>
<li><a href="#a-paper-per-day">A paper per day</a><ul>
<li><a href="#metagenomics-web-service">2019-06-07, <em>metagenomics</em> | <em>web service</em></a></li>
<li><a href="#transcriptomics-tool">2019-06-08, <em>transcriptomics</em> | <em>tool</em></a></li>
<li><a href="#transriptomics-bats-mirna-longevity">2019-06-11, <em>transriptomics</em> | <em>bats</em> | <em>miRNA</em> | <em>longevity</em></a></li>
<li><a href="#metagenomics-assembly-tool">2019-06-12, <em>metagenomics</em> | <em>assembly</em> | <em>tool</em></a></li>
<li><a href="#metagenomics-pipeline">2019-06-13, <em>metagenomics</em> | <em>pipeline</em></a></li>
<li><a href="#section">2019-06-14,</a></li>
<li><a href="#section-1">2019-06-17,</a></li>
</ul></li>
<li><a href="#read">Read</a></li>
</ul>
</nav>
<h1 id="a-paper-per-day">A paper per day</h1>
<p>To succeed in science, reading is of paramount importance. However, I always lack the time (or I find other excuses) to read all the interesting sounding manuscripts I stumble across every day on Twitter or bioRxiv. Inspired by a tweet of @<a href="https://twitter.com/BeAScientist/status/1136657979317448705">BeAScientist</a> I will challenge myself to read <em>a scientific paper per (work)day</em>.</p>
<p>I’m trying to update this list to 1) keep track of myself, 2) provide some <a href="https://www.urbandictionary.com/define.php?term=tl%3Bdr">TL;DR</a> notes for the impatient and for me to rember, and 3) perhaps inspire others to do the same and to join the <em>a paper per day</em> challenge.</p>
<p><em>Martin Hölzer</em></p>
<h2 id="metagenomics-web-service">2019-06-07, <em>metagenomics</em> | <em>web service</em></h2>
<p><a href="https://academic.oup.com/nar/article/46/D1/D726/4561650">Mitchell, Alex L., et al. “EBI Metagenomics in 2017: enriching the analysis of microbial communities, from sequence reads to assemblies.” Nucleic acids research 46.D1 (2017): D726-D735.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation" data-cites="article">@article</span>{Mitchell:17,<br /> title={{E}{B}{I} {M}etagenomics in 2017: enriching the analysis of microbial communities, from sequence reads to assemblies},<br /> author={Mitchell, Alex L and Scheremetjew, Maxim and Denise, Hubert and Potter, Simon and Tarkowska, Aleksandra and Qureshi, Matloob and Salazar, Gustavo A and Pesseat, Sebastien and Boland, Miguel A and Hunter, Fiona M I and others},<br /> journal={Nucleic Acids Res},<br /> volume={46},<br /> number={D1},<br /> pages={D726–D735},<br /> year={2017},<br /> }
</blockquote>
</div>
<ul>
<li>EBI Metagenomics: one of the worlds largest metagenomic data repository</li>
<li>the paper presents updates from the last 2 years (~2015-2017) to the service, <a href="https://www.ebi.ac.uk/metagenomics/">MGnify</a></li>
<li>analyis and archive sequencing data from microbial environments</li>
<li>new: taxonomic assignment by 16S and 23S rRNA assignments using Infernal and Rfam</li>
<li>also new: metagenomic assembly (w metaSPAdes, MEGAHIT and Minia) upon request –&gt; 50mio non-redundant protein sequences
<ul>
<li>selection of optimal assembly pipeline using a neural network</li>
</ul></li>
<li>metagenome analysis problems
<ul>
<li>need high computational power</li>
<li>different tools/pipelines –&gt; hard to compare results</li>
</ul></li>
<li>majority of data is 16S rRNA amplicons, followed by WGS</li>
<li>improved search interface</li>
</ul>
<h2 id="transcriptomics-tool">2019-06-08, <em>transcriptomics</em> | <em>tool</em></h2>
<p><a href="https://academic.oup.com/bioinformatics/article/35/11/1960/5150437#.XPnnedggr9c.twitter">Vera Alvarez, Roberto, et al. “TPMCalculator: one-step software to quantify mRNA abundance of genomic features.” Bioinformatics (2018).</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation" data-cites="article">@article</span>{Vera:18,<br /> title={{T}{P}{M}{C}alculator: one-step software to quantify m{R}{N}{A} abundance of genomic features},<br /> author={Vera Alvarez, Roberto and Pongor, Lorinc Sandor and Mari{~n}o-Ram{'}rez, Leonardo and Landsman, David},<br /> journal={Bioinformatics},<br /> year={2018}<br /> }
</blockquote>
</div>
<ul>
<li>input: BAM, output: TPM values, raw counts, feature lengths for genes, transcripts, exons and introns</li>
<li><a href="https://github.com/ncbi/TPMCalculator">https://github.com/ncbi/TPMCalculator</a></li>
<li>RNA-Seq raw count normalization evolved from RPKM, FPKM, to TPM
<ul>
<li>R/F-PKM are biased when comparing multiple samples</li>
<li>but still, many tools use them instead of TPM</li>
</ul></li>
</ul>
<h2 id="transriptomics-bats-mirna-longevity">2019-06-11, <em>transriptomics</em> | <em>bats</em> | <em>miRNA</em> | <em>longevity</em></h2>
<p><a href="https://sci-hub.tw/10.1038/s41559-019-0913-3">Zixia Huang et al. “Longitudinal comparative transcriptomics reveals unique mechanisms underlying extended healthspan in bats.” Nature Ecology &amp; Evolution (2019).</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation" data-cites="Article">@Article</span>{Huang:19,<br /> author = {Huang, Zixia and Whelan, Conor V and Foley, Nicole M and Jebb, David and Touzalin, Fr{'e}d{'e}ric and Petit, Eric J and Puechmaille, S{'e}bastien J and Teeling, Emma C},<br /> title = {Longitudinal comparative transcriptomics reveals unique mechanisms underlying extended healthspan in bats},<br /> journal = {Nat Ecol Evol},<br /> year = {2019},<br /> }
</blockquote>
</div>
<ul>
<li>bats are longest-lived mammals</li>
<li>8-year study of ageing in <em>Myotis myotis</em></li>
<li>150 blood samples, RNA-Seq</li>
<li>also compared with ~300 longitudinal RNA-Seq samples of other mammals (human, mouse, wolf)</li>
<li>bats show a unique age-related gene expression</li>
<li>partial regulated by miRNAs</li>
<li><em>Myotis</em> maintain length of telomeres with age without developing cancer</li>
<li>like humans, 75% of the transcriptional output derived from the top 100 expressed transcripts (enriched DNA repair!)</li>
<li><em>Myotis</em> might have evolved an elaborate DNA repair machinery and a distinct immune system (no increased inflammation in age)</li>
<li>genome maintenance and cell cycle regulation down-regulated in the other mammals, not in bats</li>
<li>GenAge database, 207 human ageing-associated genes</li>
<li>M&amp;Ms
<ul>
<li>rRNA- and smallRNA-Seq Illumina</li>
<li>50mill reads, 2x 125bp</li>
<li>8mill reads, 1x 50bp for small</li>
<li>100 transcriptomes, 50 miRNomes from 150 blood samples</li>
<li>TopHat, Cufflinks, …</li>
<li>Metascape for functional enrichment analysis</li>
<li>miRNA-mRNA interaction analysis: ExUTR, miRanda</li>
<li>raw data: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA503704</li>
</ul></li>
<li>supplement
<ul>
<li>only 30% mapping rate with smallRNA-Seq (why?)</li>
<li></li>
</ul></li>
</ul>
<h2 id="metagenomics-assembly-tool">2019-06-12, <em>metagenomics</em> | <em>assembly</em> | <em>tool</em></h2>
<p><a href="https://academic.oup.com/bioinformatics/article/31/10/1674/177884">Li, Dinghua, et al. “MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.” Bioinformatics 31.10 (2015): 1674-1676.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation" data-cites="Article">@Article</span>{Li:15, title={{MEGAHIT}: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de {B}ruijn graph}, author={Li, Dinghua and Liu, Chi-Man and Luo, Ruibang and Sadakane, Kunihiko and Lam, Tak-Wah}, journal={Bioinformatics}, volume={31}, number={10}, pages={1674–1676}, year={2015} }
</blockquote>
</div>
<ul>
<li><a href="https://github.com/voutcn/megahit">GitHub</a></li>
<li>de novo assembler for large and complex metagenomics data</li>
<li>GPU support</li>
<li>based on <em>succinct de Brujin</em> graphs: compressed representations of <em>de Brujin</em></li>
<li>build SdBGs iteratively from a small to a large <em>k</em>-mer</li>
</ul>
<h2 id="metagenomics-pipeline">2019-06-13, <em>metagenomics</em> | <em>pipeline</em></h2>
<p><a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0541-1">Uritskiy, Gherman V., Jocelyne DiRuggiero, and James Taylor. “MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis.” Microbiome 6.1 (2018): 158.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation" data-cites="Article">@Article</span>{Uritskiy:18, title={Meta{WRAP} - a flexible pipeline for genome-resolved metagenomic data analysis}, author={Uritskiy, Gherman V and DiRuggiero, Jocelyne and Taylor, James}, journal={Microbiome}, volume={6}, number={1}, pages={158}, year={2018}, publisher={BioMed Central} }
</blockquote>
</div>
<ul>
<li>genome-resolved metagenomic data analysis</li>
<li><a href="https://github.com/bxlab/metaWRAP">MetaWRAP</a></li>
<li>extract individual draft genomes (bins)</li>
<li>from raw reads to metagenomic bins</li>
<li>binning: often clustering togehter similar scaffolds e.g. by <em>k</em>-mer, codon usage, and similar coverage</li>
<li>highly modular, each modul is a shell script</li>
<li>custom python2.7 scripts</li>
<li>conda, docker</li>
<li>database download guide</li>
<li>reassemble_bins module (map reads to bins and assemble again with SPAdes instead of metaSPAdes/MegaHit); highly improves bin quality
<ul>
<li>The Reassemble_bins module improved upon 78%, 98%, and 2% of the bins in the water, gut, and soil bin sets, respectively.</li>
<li>soil is so diverse, that the re-assembly step confuses</li>
<li>own idea: integrate long reads in the re-assembly step</li>
</ul></li>
<li>metagenomic binning software: MaxBin2, metaBAT2, CONCOCT</li>
<li>test data: water, gut, soil
<ul>
<li>water samples were dominated by <em>Proteobacteria</em></li>
</ul></li>
<li>Bin_refinement module produced 235, 175, and 134 acceptable quality bins in the water, gut, and soil samples</li>
</ul>
<h2 id="section">2019-06-14,</h2>
<p><a href="https://academic.oup.com/gigascience/article/8/4/giy158/5241890">Johnson, Lisa Kristine, Harriet Alexander, and C. Titus Brown. &quot;Re-assembly, quality evaluation, and annotation of 678 microbial eukaryotic reference transcriptomes.&quot; GigaScience (2018): giy158.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@article</span>{Johnson:18, title={Re-assembly, quality evaluation, and annotation of 678 microbial eukaryotic reference transcriptomes}, author={Johnson, Lisa Kristine and Alexander, Harriet and Brown, C Titus}, journal={GigaScience}, volume={8}, number={4}, pages={giy158}, year={2018} }
</blockquote>
</div>
<p>With the growing volume of sequencing data, the development of reusable software is becoming increasingly important. A great demonstration of how re-assembly of old RNA-Seq data by new tools improves the whole analysis!</p>
<ul>
<li>short-read RNA-Seq pipeline for Marine Microbial Eukaryotic Transcriptomes</li>
<li>evaluation and comparison</li>
<li>newer transcriptome assemblies contained the majority of previous contigs as well as new content, ~8% more genes</li>
<li>no single “best” reference transcriptome</li>
<li>we need pipelines to re-assemble!</li>
<li>Trimmomatic -- khmer -- Trinity v2.2.0</li>
<li>re-analyzed 678 samples of sequenced marine protists, spanning more than 40 eukaryotic phyla (<a href="https://github.com/dib-lab/dib-MMETSP">MMETSP</a>)</li>
<li>raw reads: <a href="https://www.ncbi.nlm.nih.gov/bioproject?term=PRJNA231566">PRJNA231566</a></li>
<li>previously assembled with Trans-ABySS</li>
<li>Transcriptomes were annotated using the <a href="https://github.com/dib-lab/dammit">dammit pipeline</a></li>
<li>eval with Transrate</li>
<li>Trinity-based re-assemblies generally outperform Trans-ABySS-based assemblies</li>
</ul>
<h2 id="section-1">2019-06-17,</h2>
<p><a href=""></a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<p><span class="citation" data-cites="Article">@Article</span>{</p>
}
</blockquote>
</div>
<ul>
<li></li>
</ul>
<h1 id="read">Read</h1>
<ul>
<li>Phushing Limits of de novo assembly
<ul>
<li>https://t.co/ZSsiqJVjil?amp=1</li>
</ul></li>
<li>A metaanalysis of bat phylogenetics and positive selection based on genomes and transcriptomes from 18 species
<ul>
<li>https://www.pnas.org/content/pnas/early/2019/05/20/1814995116.full.pdf</li>
</ul></li>
<li>Identification of rare-disease genes using blood transcriptome sequencing and large control cohorts
<ul>
<li>https://www.nature.com/articles/s41591-019-0457-8</li>
</ul></li>
<li>BiosyntheticSPAdes: Reconstructing Biosynthetic Gene Clusters From Assembly Graphs
<ul>
<li></li>
</ul></li>
<li>Exploiting orthology and de novo transcriptome assembly to refine target sequence information
<ul>
<li>https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-019-0524-5</li>
</ul></li>
<li>MULKSG: MULtiple KSimultaneous Graph Assembly
<ul>
<li>https://link.springer.com/chapter/10.1007/978-3-030-18174-1_9</li>
</ul></li>
<li>Overview of Virus Metagenomic Classification Methods and Their Biological Applications
<ul>
<li>https://www.frontiersin.org/articles/10.3389/fmicb.2018.00749/full</li>
</ul></li>
<li>NanoPlot
<ul>
<li>https://academic.oup.com/bioinformatics/article/34/15/2666/4934939</li>
</ul></li>
<li>metaplasmidspades; Plasmid detection and assembly in genomic and metagenomic data sets
<ul>
<li>https://genome.cshlp.org/content/29/6/961</li>
</ul></li>
<li>Next-generation transcriptome assembly and analysis: impact of ploidy
<ul>
<li>https://reader.elsevier.com/reader/sd/pii/S1046202318303803?token=6B56E6005D125657F75FF7A2651B28BF62C680FC2F24D7C3B2EE0034EAFCB86DCAF3B31892D544BF29986452407FC383</li>
</ul></li>
<li><p>Population level mitogenomics of long-lived bats reveals dynamic heteroplasmy and challenges the free radical theory of ageing.</p></li>
<li>Metascape provides a biologist-oriented resource for the analysis of systems-level datasets
<ul>
<li>https://www.nature.com/articles/s41467-019-09234-6</li>
</ul></li>
<li>Overview of Virus Metagenomic Classification Methods and Their Biological Applications
<ul>
<li>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5924777/pdf/fmicb-09-00749.pdf</li>
</ul></li>
<li>Choice of assembly software has a critical impact on virome characterisation.
<ul>
<li>https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0626-5</li>
</ul></li>
<li>New approaches for metagenome assembly with short reads
<ul>
<li>https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz020/5363831</li>
</ul></li>
<li>Software engineering for scientific big data analysis
<ul>
<li>https://academic.oup.com/gigascience/article/8/5/giz054/5497810</li>
</ul></li>
<li>An improved pig reference genome sequence to enable pig genetics and genomics research
<ul>
<li>https://www.biorxiv.org/content/10.1101/668921v1</li>
</ul></li>
<li>MEGAN-LR: new algorithms allow accurate binning and easy interactive exploration of metagenomic long reads and contigs
<ul>
<li>https://link.springer.com/article/10.1186/s13062-018-0208-7</li>
</ul></li>
<<<<<<< HEAD
<li>A Reference-Free Algorithm for Computational Normalization of Shotgun Sequencing Data
<ul>
<li>https://arxiv.org/abs/1203.4802</li>
=======
<li>Long-Read Sequencing - A Powerful Tool in Viral Transcriptome Research
<ul>
<li>https://www.cell.com/trends/microbiology/fulltext/S0966-842X(19)30036-8</li>
>>>>>>> e9c6cb12839327103fc24336c602904125dee7e3
</ul></li>
</ul>
</body>
</html>
