<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title>A paper per day</title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="pandoc.css" type="text/css" />
</head>
<body>
<div id="TOC">
<ul>
<li><a href="#a-paper-per-day">A paper per day</a><ul>
<li><a href="#metagenomics-web-service">2019-06-07, <em>metagenomics</em> | <em>web service</em></a></li>
<li><a href="#transcriptomics-tool">2019-06-08, <em>transcriptomics</em> | <em>tool</em></a></li>
<li><a href="#transriptomics-bats-mirna-longevity">2019-06-11, <em>transriptomics</em> | <em>bats</em> | <em>miRNA</em> | <em>longevity</em></a></li>
<li><a href="#metagenomics-assembly-tool">2019-06-12, <em>metagenomics</em> | <em>assembly</em> | <em>tool</em></a></li>
<li><a href="#metagenomics-pipeline">2019-06-13, <em>metagenomics</em> | <em>pipeline</em></a></li>
<li><a href="#meta-transcriptomics-pipeline-re-assembly">2019-06-14, <em>meta-transcriptomics</em> | <em>pipeline</em> | <em>re-assembly</em></a></li>
<li><a href="#genome-assembly-prokaryotic-de-novo">2019-06-17, <em>genome assembly</em> | <em>prokaryotic</em> | <em>de novo</em></a></li>
<li><a href="#section">2019-06-18,</a></li>
<li><a href="#section-1">2019-06-19,</a></li>
<li><a href="#section-2">2019-06-20,</a></li>
</ul></li>
<li><a href="#todo-list">ToDo list</a></li>
</ul>
</div>
<h1 id="a-paper-per-day">A paper per day</h1>
<p>To succeed in science, reading is of paramount importance. However, I always lack the time (or I find other excuses) to read all the interesting sounding manuscripts I stumble across every day on Twitter or bioRxiv. Inspired by a tweet of @<a href="https://twitter.com/BeAScientist/status/1136657979317448705">BeAScientist</a> I will challenge myself to read <em>a scientific paper per (work)day</em>.</p>
<p>I'm trying to update this list to 1) keep track of myself, 2) provide some <a href="https://www.urbandictionary.com/define.php?term=tl%3Bdr">TL;DR</a> notes for the impatient and for me to rember, and 3) perhaps inspire others to do the same and to join the <em>a paper per day</em> challenge.</p>
<p><em>Martin Hölzer</em></p>
<h2 id="metagenomics-web-service">2019-06-07, <em>metagenomics</em> | <em>web service</em></h2>
<p><a href="https://academic.oup.com/nar/article/46/D1/D726/4561650">Mitchell, Alex L., et al. &quot;EBI Metagenomics in 2017: enriching the analysis of microbial communities, from sequence reads to assemblies.&quot; Nucleic acids research 46.D1 (2017): D726-D735.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@article</span>{Mitchell:17,<br /> title={{E}{B}{I} {M}etagenomics in 2017: enriching the analysis of microbial communities, from sequence reads to assemblies},<br /> author={Mitchell, Alex L and Scheremetjew, Maxim and Denise, Hubert and Potter, Simon and Tarkowska, Aleksandra and Qureshi, Matloob and Salazar, Gustavo A and Pesseat, Sebastien and Boland, Miguel A and Hunter, Fiona M I and others},<br /> journal={Nucleic Acids Res},<br /> volume={46},<br /> number={D1},<br /> pages={D726--D735},<br /> year={2017},<br /> }
</blockquote>
</div>
<ul>
<li>EBI Metagenomics: one of the worlds largest metagenomic data repository</li>
<li>the paper presents updates from the last 2 years (~2015-2017) to the service, <a href="https://www.ebi.ac.uk/metagenomics/">MGnify</a></li>
<li>analyis and archive sequencing data from microbial environments</li>
<li>new: taxonomic assignment by 16S and 23S rRNA assignments using Infernal and Rfam</li>
<li>also new: metagenomic assembly (w metaSPAdes, MEGAHIT and Minia) upon request --&gt; 50mio non-redundant protein sequences
<ul>
<li>selection of optimal assembly pipeline using a neural network</li>
</ul></li>
<li>metagenome analysis problems
<ul>
<li>need high computational power</li>
<li>different tools/pipelines --&gt; hard to compare results</li>
</ul></li>
<li>majority of data is 16S rRNA amplicons, followed by WGS</li>
<li>improved search interface</li>
</ul>
<h2 id="transcriptomics-tool">2019-06-08, <em>transcriptomics</em> | <em>tool</em></h2>
<p><a href="https://academic.oup.com/bioinformatics/article/35/11/1960/5150437#.XPnnedggr9c.twitter">Vera Alvarez, Roberto, et al. &quot;TPMCalculator: one-step software to quantify mRNA abundance of genomic features.&quot; Bioinformatics (2018).</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@article</span>{Vera:18,<br /> title={{T}{P}{M}{C}alculator: one-step software to quantify m{R}{N}{A} abundance of genomic features},<br /> author={Vera Alvarez, Roberto and Pongor, Lorinc Sandor and Mari{~n}o-Ram{'}rez, Leonardo and Landsman, David},<br /> journal={Bioinformatics},<br /> year={2018}<br /> }
</blockquote>
</div>
<ul>
<li>input: BAM, output: TPM values, raw counts, feature lengths for genes, transcripts, exons and introns</li>
<li><a href="https://github.com/ncbi/TPMCalculator" class="uri">https://github.com/ncbi/TPMCalculator</a></li>
<li>RNA-Seq raw count normalization evolved from RPKM, FPKM, to TPM
<ul>
<li>R/F-PKM are biased when comparing multiple samples</li>
<li>but still, many tools use them instead of TPM</li>
</ul></li>
</ul>
<h2 id="transriptomics-bats-mirna-longevity">2019-06-11, <em>transriptomics</em> | <em>bats</em> | <em>miRNA</em> | <em>longevity</em></h2>
<p><a href="https://sci-hub.tw/10.1038/s41559-019-0913-3">Zixia Huang et al. &quot;Longitudinal comparative transcriptomics reveals unique mechanisms underlying extended healthspan in bats.&quot; Nature Ecology &amp; Evolution (2019).</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@Article</span>{Huang:19,<br /> author = {Huang, Zixia and Whelan, Conor V and Foley, Nicole M and Jebb, David and Touzalin, Fr{'e}d{'e}ric and Petit, Eric J and Puechmaille, S{'e}bastien J and Teeling, Emma C},<br /> title = {Longitudinal comparative transcriptomics reveals unique mechanisms underlying extended healthspan in bats},<br /> journal = {Nat Ecol Evol},<br /> year = {2019},<br /> }
</blockquote>
</div>
<ul>
<li>bats are longest-lived mammals</li>
<li>8-year study of ageing in <em>Myotis myotis</em></li>
<li>150 blood samples, RNA-Seq</li>
<li>also compared with ~300 longitudinal RNA-Seq samples of other mammals (human, mouse, wolf)</li>
<li>bats show a unique age-related gene expression</li>
<li>partial regulated by miRNAs</li>
<li><em>Myotis</em> maintain length of telomeres with age without developing cancer</li>
<li>like humans, 75% of the transcriptional output derived from the top 100 expressed transcripts (enriched DNA repair!)</li>
<li><em>Myotis</em> might have evolved an elaborate DNA repair machinery and a distinct immune system (no increased inflammation in age)</li>
<li>genome maintenance and cell cycle regulation down-regulated in the other mammals, not in bats</li>
<li>GenAge database, 207 human ageing-associated genes</li>
<li>M&amp;Ms
<ul>
<li>rRNA- and smallRNA-Seq Illumina</li>
<li>50mill reads, 2x 125bp</li>
<li>8mill reads, 1x 50bp for small</li>
<li>100 transcriptomes, 50 miRNomes from 150 blood samples</li>
<li>TopHat, Cufflinks, ...</li>
<li>Metascape for functional enrichment analysis</li>
<li>miRNA-mRNA interaction analysis: ExUTR, miRanda</li>
<li>raw data: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA503704</li>
</ul></li>
<li>supplement
<ul>
<li>only 30% mapping rate with smallRNA-Seq (why?)</li>
<li></li>
</ul></li>
</ul>
<h2 id="metagenomics-assembly-tool">2019-06-12, <em>metagenomics</em> | <em>assembly</em> | <em>tool</em></h2>
<p><a href="https://academic.oup.com/bioinformatics/article/31/10/1674/177884">Li, Dinghua, et al. &quot;MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.&quot; Bioinformatics 31.10 (2015): 1674-1676.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@Article</span>{Li:15,<br /> title={{MEGAHIT}: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de {B}ruijn graph},<br /> author={Li, Dinghua and Liu, Chi-Man and Luo, Ruibang and Sadakane, Kunihiko and Lam, Tak-Wah},<br /> journal={Bioinformatics},<br /> volume={31},<br /> number={10},<br /> pages={1674--1676},<br /> year={2015}<br /> }
</blockquote>
</div>
<ul>
<li><a href="https://github.com/voutcn/megahit">GitHub</a></li>
<li>de novo assembler for large and complex metagenomics data</li>
<li>GPU support</li>
<li>based on <em>succinct de Brujin</em> graphs: compressed representations of <em>de Brujin</em></li>
<li>build SdBGs iteratively from a small to a large <em>k</em>-mer</li>
</ul>
<h2 id="metagenomics-pipeline">2019-06-13, <em>metagenomics</em> | <em>pipeline</em></h2>
<p><a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0541-1">Uritskiy, Gherman V., Jocelyne DiRuggiero, and James Taylor. &quot;MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis.&quot; Microbiome 6.1 (2018): 158.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@Article</span>{Uritskiy:18,<br /> title={Meta{WRAP} - a flexible pipeline for genome-resolved metagenomic data analysis},<br /> author={Uritskiy, Gherman V and DiRuggiero, Jocelyne and Taylor, James},<br /> journal={Microbiome},<br /> volume={6},<br /> number={1},<br /> pages={158},<br /> year={2018},<br /> publisher={BioMed Central}<br /> }
</blockquote>
</div>
<ul>
<li>genome-resolved metagenomic data analysis</li>
<li><a href="https://github.com/bxlab/metaWRAP">MetaWRAP</a></li>
<li>extract individual draft genomes (bins)</li>
<li>from raw reads to metagenomic bins</li>
<li>binning: often clustering togehter similar scaffolds e.g. by <em>k</em>-mer, codon usage, and similar coverage</li>
<li>highly modular, each modul is a shell script</li>
<li>custom python2.7 scripts</li>
<li>conda, docker</li>
<li>database download guide</li>
<li>reassemble_bins module (map reads to bins and assemble again with SPAdes instead of metaSPAdes/MegaHit); highly improves bin quality
<ul>
<li>The Reassemble_bins module improved upon 78%, 98%, and 2% of the bins in the water, gut, and soil bin sets, respectively.</li>
<li>soil is so diverse, that the re-assembly step confuses</li>
<li>own idea: integrate long reads in the re-assembly step</li>
</ul></li>
<li>metagenomic binning software: MaxBin2, metaBAT2, CONCOCT</li>
<li>test data: water, gut, soil
<ul>
<li>water samples were dominated by <em>Proteobacteria</em></li>
</ul></li>
<li>Bin_refinement module produced 235, 175, and 134 acceptable quality bins in the water, gut, and soil samples</li>
</ul>
<h2 id="meta-transcriptomics-pipeline-re-assembly">2019-06-14, <em>meta-transcriptomics</em> | <em>pipeline</em> | <em>re-assembly</em></h2>
<p><a href="https://academic.oup.com/gigascience/article/8/4/giy158/5241890">Johnson, Lisa Kristine, Harriet Alexander, and C. Titus Brown. &quot;Re-assembly, quality evaluation, and annotation of 678 microbial eukaryotic reference transcriptomes.&quot; GigaScience (2018): giy158.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@article</span>{Johnson:18,<br /> title={Re-assembly, quality evaluation, and annotation of 678 microbial eukaryotic reference transcriptomes},<br /> author={Johnson, Lisa Kristine and Alexander, Harriet and Brown, C Titus},<br /> journal={GigaScience},<br /> volume={8},<br /> number={4},<br /> pages={giy158},<br /> year={2018}<br /> }
</blockquote>
</div>
<p>With the growing volume of sequencing data, the development of reusable software is becoming increasingly important. A great demonstration of how re-assembly of old RNA-Seq data by new tools improves the whole analysis!</p>
<ul>
<li>short-read RNA-Seq pipeline for Marine Microbial Eukaryotic Transcriptomes</li>
<li>evaluation and comparison</li>
<li>newer transcriptome assemblies contained the majority of previous contigs as well as new content, ~8% more genes</li>
<li>no single “best” reference transcriptome</li>
<li>we need pipelines to re-assemble!</li>
<li>Trimmomatic -- khmer -- Trinity v2.2.0</li>
<li>re-analyzed 678 samples of sequenced marine protists, spanning more than 40 eukaryotic phyla (<a href="https://github.com/dib-lab/dib-MMETSP">MMETSP</a>)</li>
<li>raw reads: <a href="https://www.ncbi.nlm.nih.gov/bioproject?term=PRJNA231566">PRJNA231566</a></li>
<li>previously assembled with Trans-ABySS</li>
<li>Transcriptomes were annotated using the <a href="https://github.com/dib-lab/dammit">dammit pipeline</a></li>
<li>eval with Transrate</li>
<li>Trinity-based re-assemblies generally outperform Trans-ABySS-based assemblies</li>
</ul>
<h2 id="genome-assembly-prokaryotic-de-novo">2019-06-17, <em>genome assembly</em> | <em>prokaryotic</em> | <em>de novo</em></h2>
<p><a href="https://academic.oup.com/nar/article/46/17/8953/5078241">Schmid, Michael, et al. &quot;Pushing the limits of de novo genome assembly for complex prokaryotic genomes harboring very long, near identical repeats.&quot; Nucleic acids research 46.17 (2018): 8953-8965.</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@Article</span>{schmid2018pushing,<br /> title={Pushing the limits of  genome assembly for complex prokaryotic genomes harboring very long, near identical repeats},<br /> author={Schmid, Michael and Frei, Daniel and Patrignani, Andrea and Schlapbach, Ralph and Frey, Juerg E and Remus-Emsermann, Mitja NP and Ahrens, Christian H},<br /> journal={Nucleic Acids Res},<br /> volume={46},<br /> number={17},<br /> pages={8953--8965},<br /> year={2018},<br /> publisher={Oxford University Press}<br /> }
</blockquote>
</div>
<ul>
<li>Pseudomonas koreensis P19E3, containing several near identical 70kbp repeats</li>
<li>also a variable shufflon region</li>
<li>not able to assemble with PacBio, but with Nanopore</li>
<li>very complex genome structures seem a general phenomenon among <em>Pseudomonas</em> (gammaproteobacteria)</li>
<li>further: repeat complexity analysis of 9,331 bacterial genomes revealed: ~10% that are particularly difficult to assemble</li>
<li>genome assembly:
<ul>
<li>PacBio: HGAP and Flye, no further polishing</li>
<li>ONT reads: were filtered (&gt;30kb) and assembled w/ Flye followed by three Racon runs and a last Illumina-based polishing using FreeBayes</li>
<li>Illumina: SPAdes</li>
</ul></li>
<li>QC: quast, sniffles (SVs) --&gt; mapping ONT back to the assembly</li>
<li>single misassmbled repeat (flye) --&gt; <strong>manually corrected</strong> with proovread (ONT+Ill)</li>
<li>rDNA operon typically 7 kb</li>
<li>of 270 <em>Pseudomonas</em> genomes, 245 had &gt;30kb repeats and up to 300 500 bp repeats</li>
<li>list of hard to assemble bacteria (many long repeats), Tab. 5</li>
</ul>
<h2 id="section">2019-06-18,</h2>
<p><a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz072/5306331">Mikheenko, Alla, and Mikhail Kolmogorov. &quot;Assembly Graph Browser: interactive visualization of assembly graphs.&quot; Bioinformatics (2019).</a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<span class="citation">@Article</span>{Mikheenko:19,<br /> title={Assembly {G}raph {B}rowser: interactive visualization of assembly graphs},<br /> author={Mikheenko, Alla and Kolmogorov, Mikhail},<br /> journal={Bioinformatics},<br /> year={2019}<br /> }
</blockquote>
</div>
<ul>
<li><a href="https://github.com/almiheenko/AGB">GitHub</a></li>
<li>most genome assembly projects focus on contigs/scaffolds</li>
<li>however, more information is included in the assembly graph that is difficult to explore</li>
<li>AGB helps to visualize (large) graphs including repeat analysis, construction of contracted assembly graphs, ...</li>
<li>Bandage is also a good tool, but difficult to explore large assembly graphs</li>
<li>AGB splits assembly graph in sub-graphs (connected components of the full graph)</li>
<li>GFA, FASTG, and Graphviz format allowed</li>
</ul>
<h2 id="section-1">2019-06-19,</h2>
<p><a href=""></a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<p><span class="citation">@Article</span>{</p>
}
</blockquote>
</div>
<ul>
<li></li>
</ul>
<h2 id="section-2">2019-06-20,</h2>
<p><a href=""></a></p>
<label for="trigger">Show BibTeX</label> <input id="trigger" type="checkbox">
<div class="box">
<blockquote class cite>
<p><span class="citation">@Article</span>{</p>
}
</blockquote>
</div>
<ul>
<li></li>
</ul>
<h1 id="todo-list">ToDo list</h1>
<ul>
<li>A technology-agnostic long-read analysis pipeline for transcriptome discovery and quantification
<ul>
<li>https://www.biorxiv.org/content/10.1101/672931v1</li>
<li>https://github.com/ENCODE-DCC/long-rna-seq-pipeline</li>
</ul></li>
<li>Circos: an information aesthetic for comparative genomics.
<ul>
<li>https://genome.cshlp.org/content/19/9/1639.long</li>
</ul></li>
<li>Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis
<ul>
<li>https://f1000researchdata.s3.amazonaws.com/manuscripts/12836/ef4d405a-7dd0-4de7-b4f7-7b222886c1f2_10571_-_kin_fai_au_v2.pdf?doi=10.12688/f1000research.10571.2&amp;numberOfBrowsableCollections=17&amp;numberOfBrowsableGateways=23</li>
</ul></li>
<li>The advantages and disadvantages of short- and long-read metagenomics to infer bacterial and eukaryotic community composition
<ul>
<li>https://www.biorxiv.org/content/10.1101/650788v2</li>
</ul></li>
<li>Nanopype: A modular and scalable nanopore data processing pipeline
<ul>
<li>https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz461/5514474#.XQIhqWpUgjA.twitter</li>
</ul></li>
<li>ganon: continuously up-to-date with database growth for precise short read classification in metagenomics
<ul>
<li>https://www.biorxiv.org/content/10.1101/406017v2</li>
</ul></li>
<li>A metaanalysis of bat phylogenetics and positive selection based on genomes and transcriptomes from 18 species
<ul>
<li>https://www.pnas.org/content/pnas/early/2019/05/20/1814995116.full.pdf</li>
</ul></li>
<li>Identification of rare-disease genes using blood transcriptome sequencing and large control cohorts
<ul>
<li>https://www.nature.com/articles/s41591-019-0457-8</li>
</ul></li>
<li>BiosyntheticSPAdes: Reconstructing Biosynthetic Gene Clusters From Assembly Graphs
<ul>
<li></li>
</ul></li>
<li>Exploiting orthology and de novo transcriptome assembly to refine target sequence information
<ul>
<li>https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-019-0524-5</li>
</ul></li>
<li>MULKSG: MULtiple KSimultaneous Graph Assembly
<ul>
<li>https://link.springer.com/chapter/10.1007/978-3-030-18174-1_9</li>
</ul></li>
<li>Overview of Virus Metagenomic Classification Methods and Their Biological Applications
<ul>
<li>https://www.frontiersin.org/articles/10.3389/fmicb.2018.00749/full</li>
</ul></li>
<li>NanoPlot
<ul>
<li>https://academic.oup.com/bioinformatics/article/34/15/2666/4934939</li>
</ul></li>
<li>metaplasmidspades; Plasmid detection and assembly in genomic and metagenomic data sets
<ul>
<li>https://genome.cshlp.org/content/29/6/961</li>
</ul></li>
<li>Next-generation transcriptome assembly and analysis: impact of ploidy
<ul>
<li>https://reader.elsevier.com/reader/sd/pii/S1046202318303803?token=6B56E6005D125657F75FF7A2651B28BF62C680FC2F24D7C3B2EE0034EAFCB86DCAF3B31892D544BF29986452407FC383</li>
</ul></li>
<li><p>Population level mitogenomics of long-lived bats reveals dynamic heteroplasmy and challenges the free radical theory of ageing.</p></li>
<li>Metascape provides a biologist-oriented resource for the analysis of systems-level datasets
<ul>
<li>https://www.nature.com/articles/s41467-019-09234-6</li>
</ul></li>
<li>Overview of Virus Metagenomic Classification Methods and Their Biological Applications
<ul>
<li>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5924777/pdf/fmicb-09-00749.pdf</li>
</ul></li>
<li>Choice of assembly software has a critical impact on virome characterisation.
<ul>
<li>https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0626-5</li>
</ul></li>
<li>New approaches for metagenome assembly with short reads
<ul>
<li>https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz020/5363831</li>
</ul></li>
<li>Software engineering for scientific big data analysis
<ul>
<li>https://academic.oup.com/gigascience/article/8/5/giz054/5497810</li>
</ul></li>
<li>An improved pig reference genome sequence to enable pig genetics and genomics research
<ul>
<li>https://www.biorxiv.org/content/10.1101/668921v1</li>
</ul></li>
<li>MEGAN-LR: new algorithms allow accurate binning and easy interactive exploration of metagenomic long reads and contigs
<ul>
<li>https://link.springer.com/article/10.1186/s13062-018-0208-7</li>
</ul></li>
<li>A Reference-Free Algorithm for Computational Normalization of Shotgun Sequencing Data
<ul>
<li>https://arxiv.org/abs/1203.4802</li>
</ul></li>
<li>Long-Read Sequencing - A Powerful Tool in Viral Transcriptome Research
<ul>
<li>https://www.cell.com/trends/microbiology/fulltext/S0966-842X(19)30036-8</li>
</ul></li>
<li>Effect of de novo transcriptome assembly on transcript quantification
<ul>
<li>https://www.biorxiv.org/content/10.1101/380998v4</li>
</ul></li>
<li>What are the most influencing factors in reconstructing a reliable transcriptome assembly?
<ul>
<li>https://www.biorxiv.org/content/10.1101/220269v1</li>
</ul></li>
</ul>
</body>
</html>
