\documentclass{scrartcl}

      \usepackage{lipsum}
      \usepackage[utf8]{inputenc}
      \usepackage[pdftex, colorlinks=true, linkcolor=MidnightBlue, citecolor=MidnightBlue, urlcolor=MidnightBlue, pdfauthor={Dr.\ Martin Hoelzer}, pdfpagelayout=SinglePage, bookmarks, bookmarksopen]{hyperref}
      
      \usepackage{rotating}
      \usepackage{booktabs}
      
      \usepackage{pdflscape}
      \usepackage[left=2cm,right=2cm,top=1.5cm,bottom=1.5cm]{geometry}
    
      \begin{document}
      
      \begin{landscape}
      \pagestyle{empty}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%TABLE ath
%\begin{sidewaystable}
\begin{table}
  \centering
  \caption[Main statistics and metrics for assembly evaluation]{\begin{scriptsize}
  Selected metrics based on the output of
    \texttt{rnaQUAST}, \texttt{HISAT2}, \texttt{DETONATE},
    \texttt{TransRate} and \texttt{BUSCO} for the transcripts
    assembled by all ten tools on the \emph{Arabidopsis thaliana} RNA-Seq
    data set. Details and much more statistics,
    complementing this evaluation, can be found in the electronic
    supplement, content S4--S10. The RSEM-EVAL score is divided by $10^{9}$.
Number of ambiguous bases is given in thousand. F1 score -- an F1 score of 1 would mean that all
nucleotides/contigs in the estimated true assembly were recovered
with at least 90\,\% identity. KC score -- $k$-mer compression
score reflecting the similarity of each assembly to
\texttt{DETONATE}s estimated ``true'' assembly.
The Ex90N50 statistic is computed as usual N50 but limited to the top most highly expressed 
transcripts that represent 90\,\% of the total normalized expression data. 
Normalized scores in the range between 0 and 1 are displayed. Raw values for each metric and assembly tool are given in subscript next to the normalized values.
In the last row, the summarized \emph{metric score} ($MS$) of (0,1)-normalized scores is given for each assembly tool.  
\end{scriptsize}}
  \label{tab:ath_metrics}
  \begin{scriptsize}
  \begin{tabular}{llllllllllll}
    \toprule
       &                         & \texttt{Trinity}  & \texttt{Oases}  & \texttt{Trans-ABySS} &  \texttt{SOAP-Trans} & \texttt{Bridger}  &  \texttt{BinPacker} &  \texttt{IDBA-Tran} & \texttt{Shannon} & \texttt{SPAdes-sc}  & \texttt{SPAdes-rna} \\
      \midrule
       &  $k$-mer size                & default        & 25,35,45,55,65  & 25,35,45,55,65          & default           & default      & default           & 25,35,45,55,65           & default            & default             & default \\ 
\midrule\multicolumn{11}{l}{\textbf{Evaluation metrics 1--20}}\\ \cmidrule{1-2}

\multicolumn{11}{l}{\texttt{HISAT2}}\\ 
\midrule
1 & Overall mapping rate  & 0.96$_{\,95.71}$ & 0.88$_{\,91.93}$ & 1.00$_{\,97.55}$ & 0.73$_{\,85.31}$ & 0.85$_{\,90.64}$ & 0.34$_{\,67.15}$ & 0.82$_{\,89.04}$ & 0.00$_{\,51.53}$ & 0.93$_{\,94.29}$ & 0.92$_{\,93.67}$ \\\midrule
\multicolumn{11}{l}{\texttt{rnaQUAST}}\\ 
\midrule
2 & Transcripts $\geq$1\,000\,nt  & 0.33$_{\,13357}$ & 1.00$_{\,32647}$ & 0.55$_{\,19714}$ & 0.17$_{\,8709}$ & 0.32$_{\,12882}$ & 0.00$_{\,3775}$ & 0.20$_{\,9577}$ & 0.29$_{\,12018}$ & 0.26$_{\,11171}$ & 0.26$_{\,11160}$ \\3 & Misassemblies  & 0.84$_{\,1212}$ & 0.06$_{\,6804}$ & 0.00$_{\,7227}$ & 1.00$_{\,51}$ & 0.73$_{\,1995}$ & 0.84$_{\,1233}$ & 0.98$_{\,201}$ & 0.90$_{\,769}$ & 0.84$_{\,1189}$ & 0.94$_{\,505}$ \\4 & Mismatches per transcript  & 0.79$_{\,1.32}$ & 0.42$_{\,3.47}$ & 0.91$_{\,0.61}$ & 1.00$_{\,0.1}$ & 0.60$_{\,2.43}$ & 0.00$_{\,5.9}$ & 0.99$_{\,0.14}$ & 0.89$_{\,0.72}$ & 0.94$_{\,0.43}$ & 0.96$_{\,0.34}$ \\5 & Average alignment length  & 0.55$_{\,978.21}$ & 0.89$_{\,1232.69}$ & 0.24$_{\,745.09}$ & 0.00$_{\,561.75}$ & 0.57$_{\,990.35}$ & 1.00$_{\,1312.17}$ & 0.38$_{\,848.36}$ & 0.64$_{\,1044.9}$ & 0.51$_{\,944.39}$ & 0.33$_{\,807.42}$ \\6 & 95\,\%-assembled isoforms  & 0.80$_{\,2302}$ & 1.00$_{\,2753}$ & 0.65$_{\,1960}$ & 0.32$_{\,1213}$ & 0.63$_{\,1909}$ & 0.00$_{\,489}$ & 0.30$_{\,1175}$ & 0.67$_{\,2004}$ & 0.81$_{\,2318}$ & 0.77$_{\,2237}$ \\7 & Duplication ratio  & 0.88$_{\,1.128}$ & 0.00$_{\,1.976}$ & 0.66$_{\,1.341}$ & 1.00$_{\,1.008}$ & 0.94$_{\,1.067}$ & 0.85$_{\,1.149}$ & 1.00$_{\,1.01}$ & 0.82$_{\,1.185}$ & 1.00$_{\,1.007}$ & 0.98$_{\,1.027}$ \\\midrule
8 & Ex90N50  & 0.83$_{\,1451}$ & 0.71$_{\,1305}$ & 0.41$_{\,940}$ & 0.00$_{\,449}$ & 0.99$_{\,1636}$ & 0.83$_{\,1452}$ & 0.30$_{\,805}$ & 1.00$_{\,1654}$ & 0.48$_{\,1030}$ & 0.66$_{\,1239}$ \\9 & \# full-length transcripts  & 0.81$_{\,7219}$ & 1.00$_{\,8361}$ & 0.96$_{\,8133}$ & 0.52$_{\,5411}$ & 0.80$_{\,7163}$ & 0.00$_{\,2227}$ & 0.63$_{\,6087}$ & 0.70$_{\,6541}$ & 0.91$_{\,7800}$ & 0.86$_{\,7501}$ \\\midrule
\multicolumn{11}{l}{\texttt{TransRate}}\\ 
\midrule
10 & Reference coverage  & 0.69$_{\,0.16}$ & 0.90$_{\,0.19}$ & 1.00$_{\,0.21}$ & 0.65$_{\,0.15}$ & 0.57$_{\,0.13}$ & 0.00$_{\,0.03}$ & 0.64$_{\,0.15}$ & 0.65$_{\,0.15}$ & 0.67$_{\,0.15}$ & 0.69$_{\,0.15}$ \\11 & Mean ORF percentage  & 0.57$_{\,74.79}$ & 0.38$_{\,72.07}$ & 0.41$_{\,72.5}$ & 1.00$_{\,80.78}$ & 0.56$_{\,74.66}$ & 0.00$_{\,66.84}$ & 0.99$_{\,80.71}$ & 0.66$_{\,76.08}$ & 0.55$_{\,74.47}$ & 0.63$_{\,75.67}$ \\12 & Optimal score$^{a}$  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA\\ 
13 & Percentage bases uncovered$^{a}$  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA\\ 
14 & Number of ambiguous bases  & 0.61$_{\,35964}$ & 0.00$_{\,78762}$ & 0.37$_{\,53180}$ & 0.72$_{\,28632}$ & 0.66$_{\,33008}$ & 1.00$_{\,9081}$ & 0.73$_{\,27666}$ & 0.70$_{\,30178}$ & 0.71$_{\,29569}$ & 0.69$_{\,30621}$ \\\midrule
\multicolumn{11}{l}{\texttt{DETONATE}}\\ 
\midrule
15 & Nucleotide F1  & 0.85$_{\,0.71}$ & 0.31$_{\,0.42}$ & 0.62$_{\,0.59}$ & 0.95$_{\,0.77}$ & 0.86$_{\,0.72}$ & 0.00$_{\,0.26}$ & 0.95$_{\,0.76}$ & 0.70$_{\,0.63}$ & 1.00$_{\,0.79}$ & 0.99$_{\,0.79}$ \\16 & Contig F1  & 0.73$_{\,0.05}$ & 0.43$_{\,0.03}$ & 0.59$_{\,0.04}$ & 0.59$_{\,0.04}$ & 0.58$_{\,0.04}$ & 0.00$_{\,0}$ & 0.44$_{\,0.03}$ & 0.57$_{\,0.04}$ & 0.87$_{\,0.06}$ & 1.00$_{\,0.07}$ \\17 & KC score  & 0.90$_{\,0.71}$ & 0.77$_{\,0.66}$ & 1.00$_{\,0.74}$ & 0.66$_{\,0.62}$ & 0.79$_{\,0.66}$ & 0.29$_{\,0.48}$ & 0.77$_{\,0.66}$ & 0.00$_{\,0.37}$ & 0.92$_{\,0.71}$ & 0.94$_{\,0.72}$ \\18 & RSEM EVAL  & 0.99$_{\,-0.46}$ & 0.83$_{\,-0.6}$ & 1.00$_{\,-0.45}$ & 0.69$_{\,-0.73}$ & 0.90$_{\,-0.54}$ & 0.43$_{\,-0.96}$ & 0.75$_{\,-0.67}$ & 0.00$_{\,-1.34}$ & 0.84$_{\,-0.59}$ & 0.87$_{\,-0.57}$ \\\midrule
\multicolumn{11}{l}{\texttt{BUSCO}}\\ 
\midrule
19 & Complete BUSCOs  & 1.00$_{\,1117}$ & 0.99$_{\,1108}$ & 1.00$_{\,1119}$ & 0.93$_{\,1058}$ & 0.98$_{\,1103}$ & 0.00$_{\,262}$ & 0.78$_{\,930}$ & 0.92$_{\,1049}$ & 0.95$_{\,1074}$ & 0.88$_{\,1012}$ \\20 & Missing BUSCOs  & 1.00$_{\,219}$ & 0.97$_{\,248}$ & 0.99$_{\,224}$ & 0.97$_{\,248}$ & 0.99$_{\,229}$ & 0.00$_{\,1162}$ & 0.95$_{\,269}$ & 0.92$_{\,296}$ & 0.99$_{\,225}$ & 0.95$_{\,264}$ \\\midrule
\multicolumn{2}{l}{\textbf{Summed up metric (0,1)-score}}&14.15&11.53&12.36&11.9&13.31&5.57&12.6&11.03&14.18&14.3\\\bottomrule
\multicolumn{11}{l}{$^{a}$\begin{scriptsize}Not available for the \emph{E.~coli} and \emph{A.~thaliana} data set because this metric is only calculated by \texttt{TransRate} in the case of paired-end data.\end{scriptsize}}\\\end{tabular}\end{scriptsize}
%\end{sidewaystable}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{landscape}

\end{document}