\documentclass{scrartcl}

      \usepackage{lipsum}
      \usepackage[utf8]{inputenc}
      \usepackage[pdftex, colorlinks=true, linkcolor=MidnightBlue, citecolor=MidnightBlue, urlcolor=MidnightBlue, pdfauthor={Dr.\ Martin Hoelzer}, pdfpagelayout=SinglePage, bookmarks, bookmarksopen]{hyperref}
      
      \usepackage{rotating}
      \usepackage{booktabs}
      
      \usepackage{pdflscape}
      \usepackage[left=2cm,right=2cm,top=1.5cm,bottom=1.5cm]{geometry}
    
      \begin{document}
      
      \begin{landscape}
      \pagestyle{empty}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%TABLE ebov_hsa_7h
%\begin{sidewaystable}
\begin{table}
  \centering
  \caption[Main statistics and metrics for assembly evaluation]{\begin{scriptsize}
  Selected metrics based on the output of
    \texttt{rnaQUAST}, \texttt{HISAT2}, \texttt{DETONATE},
    \texttt{TransRate} and \texttt{BUSCO} for the transcripts
    assembled by all ten tools on the \emph{Homo sapiens + EBOV 7h} RNA-Seq
    data set. Details and much more statistics,
    complementing this evaluation, can be found in the electronic
    supplement, content S4--S10. The RSEM-EVAL score is divided by $10^{9}$.
Number of ambiguous bases is given in thousand. F1 score -- an F1 score of 1 would mean that all
nucleotides/contigs in the estimated true assembly were recovered
with at least 90\,\% identity. KC score -- $k$-mer compression
score reflecting the similarity of each assembly to
\texttt{DETONATE}s estimated ``true'' assembly.
The Ex90N50 statistic is computed as usual N50 but limited to the top most highly expressed 
transcripts that represent 90\,\% of the total normalized expression data. 
Normalized scores in the range between 0 and 1 are displayed. Raw values for each metric and assembly tool are given in subscript next to the normalized values.
In the last row, the summarized \emph{metric score} ($MS$) of (0,1)-normalized scores is given for each assembly tool.  
\end{scriptsize}}
  \label{tab:ebov_hsa_7h_metrics}
  \begin{scriptsize}
  \begin{tabular}{llllllllllll}
    \toprule
       &                         & \texttt{Trinity}  & \texttt{Oases}  & \texttt{Trans-ABySS} &  \texttt{SOAP-Trans} & \texttt{Bridger}  &  \texttt{BinPacker} &  \texttt{IDBA-Tran} & \texttt{Shannon} & \texttt{SPAdes-sc}  & \texttt{SPAdes-rna} \\
      \midrule
       &  $k$-mer size                & default        & 25,35,45,55,65  & 25,35,45,55,65          & default           & default      & default           & 25,35,45,55,65           & default            & default             & default \\ 
\midrule\multicolumn{11}{l}{\textbf{Evaluation metrics 1--20}}\\ \cmidrule{1-2}

\multicolumn{11}{l}{\texttt{HISAT2}}\\ 
\midrule
1 & Overall mapping rate  & 0.91$_{\,92.58}$ & 0.53$_{\,85.78}$ & 1.00$_{\,94.22}$ & 0.79$_{\,90.52}$ & 0.72$_{\,89.14}$ & 0.46$_{\,84.63}$ & 0.00$_{\,76.39}$ & 0.82$_{\,90.99}$ & 0.87$_{\,91.94}$ & 0.95$_{\,93.4}$ \\\midrule
\multicolumn{11}{l}{\texttt{rnaQUAST}}\\ 
\midrule
2 & Transcripts $\geq$1\,000\,nt  & 0.16$_{\,31319}$ & 1.00$_{\,115596}$ & 0.22$_{\,37410}$ & 0.00$_{\,15251}$ & 0.08$_{\,23487}$ & 0.08$_{\,22928}$ & 0.01$_{\,16181}$ & 0.20$_{\,35785}$ & 0.00$_{\,15598}$ & 0.07$_{\,21867}$ \\3 & Misassemblies  & 0.94$_{\,1294}$ & 0.00$_{\,18893}$ & 0.75$_{\,4769}$ & 1.00$_{\,125}$ & 0.87$_{\,2509}$ & 0.87$_{\,2649}$ & 1.00$_{\,203}$ & 0.87$_{\,2605}$ & 0.90$_{\,2042}$ & 0.94$_{\,1219}$ \\4 & Mismatches per transcript  & 0.60$_{\,1.75}$ & 0.36$_{\,2.52}$ & 0.90$_{\,0.75}$ & 1.00$_{\,0.44}$ & 0.68$_{\,1.49}$ & 0.00$_{\,3.67}$ & 0.88$_{\,0.83}$ & 0.70$_{\,1.43}$ & 0.91$_{\,0.72}$ & 0.81$_{\,1.07}$ \\5 & Average alignment length  & 0.29$_{\,1040.88}$ & 0.44$_{\,1364.38}$ & 0.04$_{\,534.4}$ & 0.00$_{\,455.96}$ & 0.20$_{\,866.61}$ & 1.00$_{\,2505.43}$ & 0.12$_{\,697.75}$ & 0.23$_{\,918.85}$ & 0.02$_{\,489.2}$ & 0.01$_{\,470.16}$ \\6 & 95\,\%-assembled isoforms  & 0.90$_{\,4696}$ & 1.00$_{\,5115}$ & 0.74$_{\,4004}$ & 0.56$_{\,3225}$ & 0.50$_{\,2965}$ & 0.48$_{\,2851}$ & 0.00$_{\,780}$ & 0.50$_{\,2964}$ & 0.46$_{\,2795}$ & 0.71$_{\,3850}$ \\7 & Duplication ratio  & 0.78$_{\,1.757}$ & 0.00$_{\,4.347}$ & 0.80$_{\,1.663}$ & 0.98$_{\,1.06}$ & 0.87$_{\,1.451}$ & 0.74$_{\,1.884}$ & 1.00$_{\,1.007}$ & 0.72$_{\,1.954}$ & 1.00$_{\,1.013}$ & 0.95$_{\,1.161}$ \\\midrule
8 & Ex90N50  & 0.56$_{\,2839}$ & 0.64$_{\,3003}$ & 0.19$_{\,2039}$ & 0.35$_{\,2385}$ & 0.52$_{\,2745}$ & 1.00$_{\,3764}$ & 0.15$_{\,1967}$ & 0.32$_{\,2328}$ & 0.53$_{\,2766}$ & 0.00$_{\,1647}$ \\9 & \# full-length transcripts  & 0.93$_{\,8458}$ & 1.00$_{\,8861}$ & 0.92$_{\,8405}$ & 0.77$_{\,7569}$ & 0.77$_{\,7540}$ & 0.64$_{\,6844}$ & 0.00$_{\,3197}$ & 0.76$_{\,7524}$ & 0.67$_{\,7004}$ & 0.71$_{\,7226}$ \\\midrule
\multicolumn{11}{l}{\texttt{TransRate}}\\ 
\midrule
10 & Reference coverage  & 0.37$_{\,0.08}$ & 0.90$_{\,0.12}$ & 1.00$_{\,0.13}$ & 0.27$_{\,0.08}$ & 0.26$_{\,0.08}$ & 0.00$_{\,0.06}$ & 0.27$_{\,0.08}$ & 0.92$_{\,0.12}$ & 0.28$_{\,0.08}$ & 0.48$_{\,0.09}$ \\11 & Mean ORF percentage  & 0.77$_{\,52.15}$ & 0.00$_{\,43.82}$ & 0.75$_{\,51.97}$ & 0.51$_{\,49.38}$ & 0.35$_{\,47.64}$ & 0.35$_{\,47.66}$ & 0.61$_{\,50.41}$ & 1.00$_{\,54.69}$ & 0.39$_{\,48.01}$ & 0.20$_{\,45.99}$ \\12 & Optimal score$^{a}$  & 0.35$_{\,0.19}$ & 0.00$_{\,0.02}$ & 0.12$_{\,0.07}$ & 0.64$_{\,0.33}$ & 0.33$_{\,0.18}$ & 0.20$_{\,0.11}$ & 0.76$_{\,0.39}$ & 0.04$_{\,0.04}$ & 1.00$_{\,0.51}$ & 0.57$_{\,0.3}$ \\13 & Percentage bases uncovered$^{a}$  & 0.39$_{\,0.58}$ & 0.00$_{\,0.95}$ & 0.35$_{\,0.62}$ & 0.87$_{\,0.13}$ & 0.51$_{\,0.47}$ & 0.20$_{\,0.76}$ & 1.00$_{\,0.01}$ & 0.25$_{\,0.71}$ & 0.98$_{\,0.03}$ & 0.80$_{\,0.2}$ \\14 & Number of ambiguous bases  & 0.82$_{\,131284}$ & 0.00$_{\,413418}$ & 0.74$_{\,159092}$ & 0.97$_{\,77171}$ & 0.90$_{\,101788}$ & 0.96$_{\,82239}$ & 1.00$_{\,67564}$ & 0.81$_{\,134108}$ & 0.97$_{\,76646}$ & 0.85$_{\,120597}$ \\\midrule
\multicolumn{11}{l}{\texttt{DETONATE}}\\ 
\midrule
15 & Nucleotide F1  & 0.64$_{\,0.46}$ & 0.00$_{\,0.2}$ & 0.63$_{\,0.46}$ & 0.86$_{\,0.55}$ & 0.71$_{\,0.49}$ & 0.25$_{\,0.3}$ & 0.87$_{\,0.56}$ & 0.42$_{\,0.37}$ & 0.91$_{\,0.57}$ & 1.00$_{\,0.61}$ \\16 & Contig F1  & 0.27$_{\,0.01}$ & 0.28$_{\,0.01}$ & 1.00$_{\,0.05}$ & 0.80$_{\,0.04}$ & 0.23$_{\,0.01}$ & 0.00$_{\,0}$ & 0.24$_{\,0.01}$ & 0.63$_{\,0.03}$ & 0.28$_{\,0.01}$ & 0.43$_{\,0.02}$ \\17 & KC score  & 0.87$_{\,0.55}$ & 0.10$_{\,0.45}$ & 1.00$_{\,0.57}$ & 0.43$_{\,0.49}$ & 0.58$_{\,0.51}$ & 0.50$_{\,0.5}$ & 0.00$_{\,0.43}$ & 0.90$_{\,0.55}$ & 0.36$_{\,0.48}$ & 0.93$_{\,0.56}$ \\18 & RSEM EVAL  & 1.00$_{\,-1.62}$ & 0.38$_{\,-2.35}$ & 0.98$_{\,-1.64}$ & 0.81$_{\,-1.85}$ & 0.85$_{\,-1.8}$ & 0.72$_{\,-1.95}$ & 0.00$_{\,-2.8}$ & 0.90$_{\,-1.74}$ & 0.52$_{\,-2.19}$ & 0.92$_{\,-1.71}$ \\\midrule
\multicolumn{11}{l}{\texttt{BUSCO}}\\ 
\midrule
19 & Complete BUSCOs  & 1.00$_{\,4119}$ & 0.97$_{\,4055}$ & 1.00$_{\,4121}$ & 0.77$_{\,3667}$ & 0.93$_{\,3979}$ & 0.73$_{\,3590}$ & 0.00$_{\,2134}$ & 0.83$_{\,3777}$ & 0.76$_{\,3644}$ & 0.71$_{\,3542}$ \\20 & Missing BUSCOs  & 1.00$_{\,1759}$ & 0.91$_{\,1817}$ & 0.98$_{\,1771}$ & 0.85$_{\,1855}$ & 0.96$_{\,1785}$ & 0.00$_{\,2392}$ & 0.28$_{\,2217}$ & 0.69$_{\,1955}$ & 0.88$_{\,1835}$ & 0.97$_{\,1780}$ \\\midrule
\multicolumn{2}{l}{\textbf{Summed up metric (0,1)-score}}&13.54&8.5&14.11&13.24&11.81&9.17&8.18&12.5&12.7&13.0\\\bottomrule
\multicolumn{11}{l}{$^{a}$\begin{scriptsize}Not available for the \emph{E.~coli} and \emph{A.~thaliana} data set because this metric is only calculated by \texttt{TransRate} in the case of paired-end data.\end{scriptsize}}\\\end{tabular}\end{scriptsize}
%\end{sidewaystable}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{landscape}

\end{document}