\documentclass{scrartcl}

      \usepackage{lipsum}
      \usepackage[utf8]{inputenc}
      \usepackage[pdftex, colorlinks=true, linkcolor=MidnightBlue, citecolor=MidnightBlue, urlcolor=MidnightBlue, pdfauthor={Dr.\ Martin Hoelzer}, pdfpagelayout=SinglePage, bookmarks, bookmarksopen]{hyperref}
      
      \usepackage{rotating}
      \usepackage{booktabs}
      
      \usepackage{pdflscape}
      \usepackage[left=2cm,right=2cm,top=1.5cm,bottom=1.5cm]{geometry}
    
      \begin{document}
      
      \begin{landscape}
      \pagestyle{empty}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%TABLE eco
%\begin{sidewaystable}
\begin{table}
  \centering
  \caption[Main statistics and metrics for assembly evaluation]{\begin{scriptsize}
  Selected metrics based on the output of
    \texttt{rnaQUAST}, \texttt{HISAT2}, \texttt{DETONATE},
    \texttt{TransRate} and \texttt{BUSCO} for the transcripts
    assembled by all ten tools on the \emph{Escherichia coli} RNA-Seq
    data set. Details and much more statistics,
    complementing this evaluation, can be found in the electronic
    supplement, content S4--S10. The RSEM-EVAL score is divided by $10^{9}$.
Number of ambiguous bases is given in thousand. F1 score -- an F1 score of 1 would mean that all
nucleotides/contigs in the estimated true assembly were recovered
with at least 90\,\% identity. KC score -- $k$-mer compression
score reflecting the similarity of each assembly to
\texttt{DETONATE}s estimated ``true'' assembly.
The Ex90N50 statistic is computed as usual N50 but limited to the top most highly expressed 
transcripts that represent 90\,\% of the total normalized expression data. 
Normalized scores in the range between 0 and 1 are displayed. Raw values for each metric and assembly tool are given in subscript next to the normalized values.
In the last row, the summarized \emph{metric score} ($MS$) of (0,1)-normalized scores is given for each assembly tool.  
\end{scriptsize}}
  \label{tab:eco_metrics}
  \begin{scriptsize}
  \begin{tabular}{llllllllllll}
    \toprule
       &                         & \texttt{Trinity}  & \texttt{Oases}  & \texttt{Trans-ABySS} &  \texttt{SOAP-Trans} & \texttt{Bridger}  &  \texttt{BinPacker} &  \texttt{IDBA-Tran} & \texttt{Shannon} & \texttt{SPAdes-sc}  & \texttt{SPAdes-rna} \\
      \midrule
       &  $k$-mer size                & default        & 25,35,45,55,65  & 25,35,45,55,65          & default           & default      & default           & 25,35,45,55,65           & default            & default             & default \\ 
\midrule\multicolumn{11}{l}{\textbf{Evaluation metrics 1--20}}\\ \cmidrule{1-2}

\multicolumn{11}{l}{\texttt{HISAT2}}\\ 
\midrule
1 & Overall mapping rate  & 0.70$_{\,77.01}$ & 0.24$_{\,49.16}$ & 1.00$_{\,95.67}$ & 0.36$_{\,56.62}$ & 0.86$_{\,87.35}$ & 0.60$_{\,71.09}$ & 0.00$_{\,34.31}$ & 0.69$_{\,76.69}$ & 0.88$_{\,88.04}$ & 0.89$_{\,88.9}$ \\\midrule
\multicolumn{11}{l}{\texttt{rnaQUAST}}\\ 
\midrule
2 & Transcripts $\geq$1\,000\,nt  & 0.42$_{\,347}$ & 1.00$_{\,743}$ & 0.56$_{\,446}$ & 0.55$_{\,439}$ & 0.44$_{\,363}$ & 0.00$_{\,66}$ & 0.51$_{\,414}$ & 0.45$_{\,372}$ & 0.62$_{\,489}$ & 0.40$_{\,340}$ \\3 & Misassemblies  & 0.87$_{\,13}$ & 0.00$_{\,103}$ & 0.83$_{\,18}$ & 0.99$_{\,1}$ & 0.50$_{\,52}$ & 0.92$_{\,8}$ & 1.00$_{\,0}$ & 0.48$_{\,54}$ & 0.88$_{\,12}$ & 0.90$_{\,10}$ \\4 & Mismatches per transcript  & 0.88$_{\,0.37}$ & 0.70$_{\,0.71}$ & 0.90$_{\,0.32}$ & 1.00$_{\,0.13}$ & 0.93$_{\,0.27}$ & 0.00$_{\,2.05}$ & 0.98$_{\,0.17}$ & 0.97$_{\,0.19}$ & 0.94$_{\,0.25}$ & 0.96$_{\,0.21}$ \\5 & Average alignment length  & 0.20$_{\,467.04}$ & 0.19$_{\,457.72}$ & 0.00$_{\,310.44}$ & 0.07$_{\,364.88}$ & 0.21$_{\,472.81}$ & 1.00$_{\,1100.77}$ & 0.29$_{\,540.83}$ & 0.22$_{\,482.66}$ & 0.31$_{\,558.9}$ & 0.06$_{\,359.2}$ \\6 & 95\,\%-assembled isoforms  & 0.95$_{\,396}$ & 1.00$_{\,414}$ & 0.00$_{\,32}$ & 0.60$_{\,261}$ & 0.92$_{\,382}$ & 0.01$_{\,34}$ & 0.57$_{\,250}$ & 0.95$_{\,393}$ & 0.47$_{\,210}$ & 0.41$_{\,187}$ \\7 & Duplication ratio  & 0.98$_{\,1.012}$ & 0.00$_{\,1.443}$ & 0.90$_{\,1.044}$ & 1.00$_{\,1.002}$ & 0.97$_{\,1.014}$ & 0.40$_{\,1.267}$ & 1.00$_{\,1.001}$ & 0.94$_{\,1.027}$ & 0.99$_{\,1.006}$ & 0.98$_{\,1.009}$ \\\midrule
8 & Ex90N50  & 0.46$_{\,369}$ & 0.21$_{\,229}$ & 0.12$_{\,177}$ & 0.11$_{\,173}$ & 0.50$_{\,393}$ & 1.00$_{\,675}$ & 0.34$_{\,304}$ & 0.45$_{\,363}$ & 0.00$_{\,111}$ & 0.24$_{\,245}$ \\9 & \# full-length transcripts  & 0.92$_{\,407}$ & 0.94$_{\,418}$ & 0.99$_{\,436}$ & 0.96$_{\,423}$ & 0.90$_{\,402}$ & 0.00$_{\,37}$ & 0.89$_{\,396}$ & 0.90$_{\,399}$ & 1.00$_{\,441}$ & 0.75$_{\,338}$ \\\midrule
\multicolumn{11}{l}{\texttt{TransRate}}\\ 
\midrule
10 & Reference coverage  & 1.00$_{\,0.32}$ & 0.97$_{\,0.31}$ & 0.07$_{\,0.03}$ & 0.61$_{\,0.19}$ & 0.98$_{\,0.31}$ & 0.00$_{\,0.01}$ & 0.50$_{\,0.16}$ & 0.96$_{\,0.3}$ & 0.44$_{\,0.14}$ & 0.55$_{\,0.18}$ \\11 & Mean ORF percentage  & 1.00$_{\,75.04}$ & 0.69$_{\,65.72}$ & 0.92$_{\,72.73}$ & 0.90$_{\,71.99}$ & 0.99$_{\,74.59}$ & 0.00$_{\,45.13}$ & 0.90$_{\,71.99}$ & 0.94$_{\,73.34}$ & 0.81$_{\,69.47}$ & 0.96$_{\,73.78}$ \\12 & Optimal score$^{a}$  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA\\ 
13 & Percentage bases uncovered$^{a}$  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA\\ 
14 & Number of ambiguous bases  & 0.43$_{\,2143}$ & 0.00$_{\,3602}$ & 0.15$_{\,3081}$ & 0.30$_{\,2579}$ & 0.44$_{\,2117}$ & 1.00$_{\,191}$ & 0.45$_{\,2069}$ & 0.43$_{\,2145}$ & 0.36$_{\,2370}$ & 0.31$_{\,2556}$ \\\midrule
\multicolumn{11}{l}{\texttt{DETONATE}}\\ 
\midrule
15 & Nucleotide F1  & 0.86$_{\,0.64}$ & 0.63$_{\,0.49}$ & 0.82$_{\,0.61}$ & 1.00$_{\,0.74}$ & 0.85$_{\,0.64}$ & 0.00$_{\,0.06}$ & 0.89$_{\,0.66}$ & 0.83$_{\,0.62}$ & 0.97$_{\,0.71}$ & 0.97$_{\,0.72}$ \\16 & Contig F1  & 0.46$_{\,0.03}$ & 0.38$_{\,0.03}$ & 0.47$_{\,0.03}$ & 0.89$_{\,0.06}$ & 0.43$_{\,0.03}$ & 0.00$_{\,0}$ & 0.41$_{\,0.03}$ & 0.38$_{\,0.03}$ & 0.83$_{\,0.06}$ & 1.00$_{\,0.07}$ \\17 & KC score  & 0.92$_{\,0.82}$ & 0.89$_{\,0.79}$ & 0.93$_{\,0.82}$ & 0.98$_{\,0.86}$ & 0.91$_{\,0.81}$ & 0.00$_{\,0.16}$ & 0.94$_{\,0.83}$ & 0.91$_{\,0.81}$ & 1.00$_{\,0.87}$ & 0.96$_{\,0.84}$ \\18 & RSEM EVAL  & 0.79$_{\,-0.21}$ & 0.66$_{\,-0.25}$ & 1.00$_{\,-0.15}$ & 0.34$_{\,-0.34}$ & 0.93$_{\,-0.17}$ & 0.62$_{\,-0.26}$ & 0.00$_{\,-0.44}$ & 0.83$_{\,-0.2}$ & 0.86$_{\,-0.19}$ & 0.83$_{\,-0.2}$ \\\midrule
\multicolumn{11}{l}{\texttt{BUSCO}}\\ 
\midrule
19 & Complete BUSCOs  & 0.83$_{\,284}$ & 0.88$_{\,299}$ & 0.88$_{\,297}$ & 0.94$_{\,316}$ & 0.83$_{\,285}$ & 0.00$_{\,50}$ & 0.87$_{\,296}$ & 0.82$_{\,280}$ & 1.00$_{\,332}$ & 0.73$_{\,255}$ \\20 & Missing BUSCOs  & 0.96$_{\,190}$ & 1.00$_{\,172}$ & 1.00$_{\,170}$ & 0.99$_{\,178}$ & 0.96$_{\,190}$ & 0.00$_{\,711}$ & 0.95$_{\,196}$ & 0.95$_{\,198}$ & 1.00$_{\,172}$ & 0.96$_{\,189}$ \\\midrule
\multicolumn{2}{l}{\textbf{Summed up metric (0,1)-score}}&13.61&10.36&11.53&12.59&13.55&5.55&11.49&13.08&13.37&12.85\\\bottomrule
\multicolumn{11}{l}{$^{a}$\begin{scriptsize}Not available for the \emph{E.~coli} and \emph{A.~thaliana} data set because this metric is only calculated by \texttt{TransRate} in the case of paired-end data.\end{scriptsize}}\\\end{tabular}\end{scriptsize}
%\end{sidewaystable}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{landscape}

\end{document}