\documentclass{scrartcl}

      \usepackage{lipsum}
      \usepackage[utf8]{inputenc}
      \usepackage[pdftex, colorlinks=true, linkcolor=MidnightBlue, citecolor=MidnightBlue, urlcolor=MidnightBlue, pdfauthor={Dr.\ Martin Hoelzer}, pdfpagelayout=SinglePage, bookmarks, bookmarksopen]{hyperref}
      
      \usepackage{rotating}
      \usepackage{booktabs}
      
      \usepackage{pdflscape}
      \usepackage[left=2cm,right=2cm,top=1.5cm,bottom=1.5cm]{geometry}
    
      \begin{document}
      
      \begin{landscape}
      \pagestyle{empty}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%TABLE hsa
%\begin{sidewaystable}
\begin{table}
  \centering
  \caption[Main statistics and metrics for assembly evaluation]{\begin{scriptsize}
  Selected metrics based on the output of
    \texttt{rnaQUAST}, \texttt{HISAT2}, \texttt{DETONATE},
    \texttt{TransRate} and \texttt{BUSCO} for the transcripts
    assembled by all ten tools on the \emph{Homo sapiens} RNA-Seq
    data set. Details and much more statistics,
    complementing this evaluation, can be found in the electronic
    supplement, content S4--S10. The RSEM-EVAL score is divided by $10^{9}$.
Number of ambiguous bases is given in thousand. F1 score -- an F1 score of 1 would mean that all
nucleotides/contigs in the estimated true assembly were recovered
with at least 90\,\% identity. KC score -- $k$-mer compression
score reflecting the similarity of each assembly to
\texttt{DETONATE}s estimated ``true'' assembly.
The Ex90N50 statistic is computed as usual N50 but limited to the top most highly expressed 
transcripts that represent 90\,\% of the total normalized expression data. 
Normalized scores in the range between 0 and 1 are displayed. Raw values for each metric and assembly tool are given in subscript next to the normalized values.
In the last row, the summarized \emph{metric score} ($MS$) of (0,1)-normalized scores is given for each assembly tool.  
\end{scriptsize}}
  \label{tab:hsa_metrics}
  \begin{scriptsize}
  \begin{tabular}{llllllllllll}
    \toprule
       &                         & \texttt{Trinity}  & \texttt{Oases}  & \texttt{Trans-ABySS} &  \texttt{SOAP-Trans} & \texttt{Bridger}  &  \texttt{BinPacker} &  \texttt{IDBA-Tran} & \texttt{Shannon} & \texttt{SPAdes-sc}  & \texttt{SPAdes-rna} \\
      \midrule
       &  $k$-mer size                & default        & 25,35,45,55,65  & 25,35,45,55,65          & default           & default      & default           & 25,35,45,55,65           & default            & default             & default \\ 
\midrule\multicolumn{11}{l}{\textbf{Evaluation metrics 1--20}}\\ \cmidrule{1-2}

\multicolumn{11}{l}{\texttt{HISAT2}}\\ 
\midrule
1 & Overall mapping rate  & 0.81$_{\,91.9}$ & 0.69$_{\,88.04}$ & 1.00$_{\,98.34}$ & 0.75$_{\,89.93}$ & 0.66$_{\,86.83}$ & 0.24$_{\,72.6}$ & 0.00$_{\,64.61}$ & 0.58$_{\,84.27}$ & 0.81$_{\,92.04}$ & 0.93$_{\,95.95}$ \\\midrule
\multicolumn{11}{l}{\texttt{rnaQUAST}}\\ 
\midrule
2 & Transcripts $\geq$1\,000\,nt  & 0.22$_{\,64061}$ & 1.00$_{\,207474}$ & 0.20$_{\,59779}$ & 0.03$_{\,27529}$ & 0.11$_{\,43201}$ & 0.00$_{\,22611}$ & 0.00$_{\,23516}$ & 0.05$_{\,31328}$ & 0.05$_{\,31039}$ & 0.15$_{\,49860}$ \\3 & Misassemblies  & 0.99$_{\,3378}$ & 0.00$_{\,216127}$ & 0.99$_{\,2743}$ & 1.00$_{\,279}$ & 0.97$_{\,7329}$ & 0.98$_{\,5603}$ & 1.00$_{\,302}$ & 0.99$_{\,2837}$ & 0.99$_{\,2022}$ & 0.98$_{\,5126}$ \\4 & Mismatches per transcript  & 0.74$_{\,1.38}$ & 0.77$_{\,1.25}$ & 0.93$_{\,0.57}$ & 1.00$_{\,0.27}$ & 0.73$_{\,1.44}$ & 0.00$_{\,4.63}$ & 0.91$_{\,0.67}$ & 0.77$_{\,1.26}$ & 0.88$_{\,0.8}$ & 0.78$_{\,1.25}$ \\5 & Average alignment length  & 0.27$_{\,795.23}$ & 0.06$_{\,343.48}$ & 0.01$_{\,246.85}$ & 0.00$_{\,218}$ & 0.21$_{\,654.41}$ & 1.00$_{\,2335.73}$ & 0.13$_{\,487.11}$ & 0.23$_{\,711.83}$ & 0.09$_{\,410.22}$ & 0.09$_{\,412.24}$ \\6 & 95\,\%-assembled isoforms  & 0.99$_{\,6788}$ & 0.10$_{\,868}$ & 1.00$_{\,6824}$ & 0.31$_{\,2264}$ & 0.28$_{\,2105}$ & 0.39$_{\,2824}$ & 0.07$_{\,709}$ & 0.00$_{\,242}$ & 0.23$_{\,1755}$ & 0.46$_{\,3253}$ \\7 & Duplication ratio  & 0.00$_{\,2.396}$ & 0.03$_{\,2.355}$ & 0.47$_{\,1.743}$ & 0.87$_{\,1.187}$ & 0.50$_{\,1.708}$ & 0.01$_{\,2.389}$ & 1.00$_{\,1.012}$ & 0.63$_{\,1.53}$ & 1.00$_{\,1.015}$ & 0.87$_{\,1.192}$ \\\midrule
8 & Ex90N50  & 0.00$_{\,326}$ & 0.17$_{\,666}$ & 0.06$_{\,441}$ & 0.19$_{\,711}$ & 0.51$_{\,1370}$ & 1.00$_{\,2381}$ & 0.19$_{\,708}$ & 0.49$_{\,1324}$ & 0.42$_{\,1186}$ & 0.22$_{\,782}$ \\9 & \# full-length transcripts  & 0.97$_{\,8930}$ & 0.83$_{\,8024}$ & 1.00$_{\,9110}$ & 0.64$_{\,6806}$ & 0.89$_{\,8440}$ & 0.26$_{\,4456}$ & 0.00$_{\,2783}$ & 0.63$_{\,6758}$ & 0.46$_{\,5676}$ & 0.69$_{\,7155}$ \\\midrule
\multicolumn{11}{l}{\texttt{TransRate}}\\ 
\midrule
10 & Reference coverage  & 0.87$_{\,0.23}$ & 0.33$_{\,0.09}$ & 1.00$_{\,0.26}$ & 0.34$_{\,0.09}$ & 0.31$_{\,0.09}$ & 0.27$_{\,0.07}$ & 0.31$_{\,0.08}$ & 0.00$_{\,0}$ & 0.30$_{\,0.08}$ & 0.42$_{\,0.11}$ \\11 & Mean ORF percentage  & 0.64$_{\,50.82}$ & 0.00$_{\,42.09}$ & 0.72$_{\,51.92}$ & 0.44$_{\,48.02}$ & 0.22$_{\,45.1}$ & 0.04$_{\,42.57}$ & 0.76$_{\,52.46}$ & 1.00$_{\,55.7}$ & 0.30$_{\,46.13}$ & 0.31$_{\,46.25}$ \\12 & Optimal score$^{a}$  & 0.30$_{\,0.13}$ & 0.00$_{\,0.02}$ & 0.23$_{\,0.11}$ & 0.66$_{\,0.27}$ & 0.32$_{\,0.14}$ & 0.14$_{\,0.07}$ & 0.61$_{\,0.25}$ & 0.13$_{\,0.07}$ & 1.00$_{\,0.4}$ & 0.57$_{\,0.23}$ \\13 & Percentage bases uncovered$^{a}$  & 0.38$_{\,0.59}$ & 0.00$_{\,0.94}$ & 0.33$_{\,0.63}$ & 0.67$_{\,0.33}$ & 0.57$_{\,0.42}$ & 0.11$_{\,0.84}$ & 1.00$_{\,0.02}$ & 0.48$_{\,0.5}$ & 0.99$_{\,0.03}$ & 0.79$_{\,0.21}$ \\14 & Number of ambiguous bases  & 0.72$_{\,286479}$ & 0.00$_{\,843235}$ & 0.53$_{\,437845}$ & 0.78$_{\,241236}$ & 0.83$_{\,206635}$ & 1.00$_{\,72918}$ & 0.91$_{\,138699}$ & 0.94$_{\,117068}$ & 0.86$_{\,177477}$ & 0.71$_{\,294083}$ \\\midrule
\multicolumn{11}{l}{\texttt{DETONATE}}\\ 
\midrule
15 & Nucleotide F1  & 0.59$_{\,0.43}$ & 0.08$_{\,0.18}$ & 0.77$_{\,0.51}$ & 0.89$_{\,0.57}$ & 0.71$_{\,0.48}$ & 0.00$_{\,0.15}$ & 0.86$_{\,0.55}$ & 0.42$_{\,0.35}$ & 0.97$_{\,0.61}$ & 1.00$_{\,0.62}$ \\16 & Contig F1  & 0.08$_{\,0.02}$ & 0.09$_{\,0.02}$ & 0.99$_{\,0.2}$ & 1.00$_{\,0.21}$ & 0.05$_{\,0.01}$ & 0.00$_{\,0}$ & 0.08$_{\,0.02}$ & 0.11$_{\,0.02}$ & 0.07$_{\,0.01}$ & 0.06$_{\,0.01}$ \\17 & KC score  & 0.87$_{\,0.51}$ & 0.00$_{\,0.24}$ & 1.00$_{\,0.55}$ & 0.42$_{\,0.37}$ & 0.51$_{\,0.4}$ & 0.40$_{\,0.37}$ & 0.14$_{\,0.29}$ & 0.58$_{\,0.42}$ & 0.47$_{\,0.39}$ & 0.60$_{\,0.43}$ \\18 & RSEM EVAL  & 0.98$_{\,-6.51}$ & 0.45$_{\,-11.82}$ & 1.00$_{\,-6.26}$ & 0.72$_{\,-9.03}$ & 0.85$_{\,-7.72}$ & 0.62$_{\,-10.03}$ & 0.00$_{\,-16.3}$ & 0.73$_{\,-8.96}$ & 0.42$_{\,-12.12}$ & 0.91$_{\,-7.16}$ \\\midrule
\multicolumn{11}{l}{\texttt{BUSCO}}\\ 
\midrule
19 & Complete BUSCOs  & 0.96$_{\,4004}$ & 0.79$_{\,3588}$ & 1.00$_{\,4106}$ & 0.39$_{\,2625}$ & 0.92$_{\,3909}$ & 0.13$_{\,2009}$ & 0.00$_{\,1682}$ & 0.70$_{\,3385}$ & 0.39$_{\,2625}$ & 0.58$_{\,3089}$ \\20 & Missing BUSCOs  & 0.99$_{\,1804}$ & 0.93$_{\,1922}$ & 1.00$_{\,1770}$ & 0.83$_{\,2164}$ & 0.98$_{\,1812}$ & 0.00$_{\,4078}$ & 0.63$_{\,2615}$ & 0.84$_{\,2133}$ & 0.78$_{\,2268}$ & 0.92$_{\,1949}$ \\\midrule
\multicolumn{2}{l}{\textbf{Summed up metric (0,1)-score}}&12.38&6.31&14.24&11.92&11.13&6.59&8.61&10.3&11.47&12.03\\\bottomrule
\multicolumn{11}{l}{$^{a}$\begin{scriptsize}Not available for the \emph{E.~coli} and \emph{A.~thaliana} data set because this metric is only calculated by \texttt{TransRate} in the case of paired-end data.\end{scriptsize}}\\\end{tabular}\end{scriptsize}
%\end{sidewaystable}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{landscape}

\end{document}