\documentclass{scrartcl}

      \usepackage{lipsum}
      \usepackage[utf8]{inputenc}
      \usepackage[pdftex, colorlinks=true, linkcolor=MidnightBlue, citecolor=MidnightBlue, urlcolor=MidnightBlue, pdfauthor={Dr.\ Martin Hoelzer}, pdfpagelayout=SinglePage, bookmarks, bookmarksopen]{hyperref}
      
      \usepackage{rotating}
      \usepackage{booktabs}
      
      \usepackage{pdflscape}
      \usepackage[left=2cm,right=2cm,top=1.5cm,bottom=1.5cm]{geometry}
    
      \begin{document}
      
      \begin{landscape}
      \pagestyle{empty}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%TABLE hsa_flux
%\begin{sidewaystable}
\begin{table}
  \centering
  \caption[Main statistics and metrics for assembly evaluation]{\begin{scriptsize}
  Selected metrics based on the output of
    \texttt{rnaQUAST}, \texttt{HISAT2}, \texttt{DETONATE},
    \texttt{TransRate} and \texttt{BUSCO} for the transcripts
    assembled by all ten tools on the \emph{Homo sapiens simulated} RNA-Seq
    data set. Details and much more statistics,
    complementing this evaluation, can be found in the electronic
    supplement, content S4--S10. The RSEM-EVAL score is divided by $10^{9}$.
Number of ambiguous bases is given in thousand. F1 score -- an F1 score of 1 would mean that all
nucleotides/contigs in the estimated true assembly were recovered
with at least 90\,\% identity. KC score -- $k$-mer compression
score reflecting the similarity of each assembly to
\texttt{DETONATE}s estimated ``true'' assembly.
The Ex90N50 statistic is computed as usual N50 but limited to the top most highly expressed 
transcripts that represent 90\,\% of the total normalized expression data. 
Normalized scores in the range between 0 and 1 are displayed. Raw values for each metric and assembly tool are given in subscript next to the normalized values.
In the last row, the summarized \emph{metric score} ($MS$) of (0,1)-normalized scores is given for each assembly tool.  
\end{scriptsize}}
  \label{tab:hsa_flux_metrics}
  \begin{scriptsize}
  \begin{tabular}{llllllllllll}
    \toprule
       &                         & \texttt{Trinity}  & \texttt{Oases}  & \texttt{Trans-ABySS} &  \texttt{SOAP-Trans} & \texttt{Bridger}  &  \texttt{BinPacker} &  \texttt{IDBA-Tran} & \texttt{Shannon} & \texttt{SPAdes-sc}  & \texttt{SPAdes-rna} \\
      \midrule
       &  $k$-mer size                & default        & 25,35,45,55,65  & 25,35,45,55,65          & default           & default      & default           & 25,35,45,55,65           & default            & default             & default \\ 
\midrule\multicolumn{11}{l}{\textbf{Evaluation metrics 1--20}}\\ \cmidrule{1-2}

\multicolumn{11}{l}{\texttt{HISAT2}}\\ 
\midrule
1 & Overall mapping rate  & 0.95$_{\,96.39}$ & 0.62$_{\,73.26}$ & 1.00$_{\,99.56}$ & 0.89$_{\,91.68}$ & 0.92$_{\,94.02}$ & 0.91$_{\,93.03}$ & 0.79$_{\,85.34}$ & 0.00$_{\,30.77}$ & 0.96$_{\,97.02}$ & 0.96$_{\,96.72}$ \\\midrule
\multicolumn{11}{l}{\texttt{rnaQUAST}}\\ 
\midrule
2 & Transcripts $\geq$1\,000\,nt  & 0.24$_{\,8451}$ & 1.00$_{\,28143}$ & 0.17$_{\,6703}$ & 0.07$_{\,4263}$ & 0.11$_{\,5167}$ & 0.20$_{\,7424}$ & 0.02$_{\,2740}$ & 0.00$_{\,2341}$ & 0.01$_{\,2623}$ & 0.14$_{\,6053}$ \\3 & Misassemblies  & 0.97$_{\,139}$ & 0.00$_{\,4094}$ & 0.97$_{\,117}$ & 0.99$_{\,49}$ & 0.87$_{\,533}$ & 0.81$_{\,785}$ & 1.00$_{\,8}$ & 0.99$_{\,66}$ & 0.99$_{\,50}$ & 0.92$_{\,351}$ \\4 & Mismatches per transcript  & 0.44$_{\,1.23}$ & 0.00$_{\,2.11}$ & 0.82$_{\,0.46}$ & 0.89$_{\,0.31}$ & 0.23$_{\,1.65}$ & 0.13$_{\,1.85}$ & 1.00$_{\,0.1}$ & 0.85$_{\,0.39}$ & 0.94$_{\,0.22}$ & 0.14$_{\,1.82}$ \\5 & Average alignment length  & 1.00$_{\,2261.33}$ & 0.88$_{\,2090.55}$ & 0.06$_{\,939.31}$ & 0.11$_{\,1009.08}$ & 0.51$_{\,1581.3}$ & 0.64$_{\,1755.16}$ & 0.00$_{\,859.97}$ & 0.14$_{\,1061.01}$ & 0.09$_{\,979.98}$ & 0.56$_{\,1649.15}$ \\6 & 95\,\%-assembled isoforms  & 0.96$_{\,2898}$ & 1.00$_{\,2995}$ & 0.95$_{\,2877}$ & 0.46$_{\,1679}$ & 0.41$_{\,1570}$ & 0.45$_{\,1660}$ & 0.00$_{\,577}$ & 0.20$_{\,1056}$ & 0.43$_{\,1625}$ & 0.68$_{\,2220}$ \\7 & Duplication ratio  & 0.70$_{\,2.154}$ & 0.00$_{\,4.854}$ & 0.85$_{\,1.572}$ & 0.93$_{\,1.274}$ & 0.87$_{\,1.507}$ & 0.76$_{\,1.949}$ & 1.00$_{\,1.013}$ & 0.89$_{\,1.437}$ & 1.00$_{\,1.012}$ & 0.86$_{\,1.535}$ \\\midrule
8 & Ex90N50  & 0.96$_{\,3234}$ & 0.99$_{\,3300}$ & 0.78$_{\,2902}$ & 0.75$_{\,2836}$ & 0.97$_{\,3252}$ & 0.79$_{\,2909}$ & 0.00$_{\,1397}$ & 0.51$_{\,2368}$ & 1.00$_{\,3315}$ & 0.60$_{\,2549}$ \\9 & \# full-length transcripts  & 0.85$_{\,1492}$ & 1.00$_{\,1672}$ & 0.71$_{\,1323}$ & 0.45$_{\,1006}$ & 0.62$_{\,1210}$ & 0.65$_{\,1246}$ & 0.00$_{\,450}$ & 0.18$_{\,666}$ & 0.46$_{\,1007}$ & 0.57$_{\,1141}$ \\\midrule
\multicolumn{11}{l}{\texttt{TransRate}}\\ 
\midrule
10 & Reference coverage  & 0.34$_{\,0.24}$ & 0.74$_{\,0.38}$ & 1.00$_{\,0.46}$ & 0.16$_{\,0.18}$ & 0.17$_{\,0.19}$ & 0.22$_{\,0.2}$ & 0.14$_{\,0.18}$ & 0.00$_{\,0.13}$ & 0.09$_{\,0.16}$ & 0.35$_{\,0.25}$ \\11 & Mean ORF percentage  & 0.63$_{\,43.38}$ & 0.10$_{\,37.43}$ & 0.81$_{\,45.49}$ & 0.00$_{\,36.28}$ & 0.52$_{\,42.22}$ & 0.64$_{\,43.57}$ & 1.00$_{\,47.62}$ & 0.98$_{\,47.37}$ & 0.29$_{\,39.62}$ & 0.36$_{\,40.38}$ \\12 & Optimal score$^{a}$  & 0.22$_{\,0.11}$ & 0.00$_{\,0.01}$ & 0.47$_{\,0.22}$ & 0.43$_{\,0.21}$ & 0.29$_{\,0.14}$ & 0.26$_{\,0.13}$ & 0.74$_{\,0.35}$ & 0.10$_{\,0.06}$ & 1.00$_{\,0.47}$ & 0.40$_{\,0.19}$ \\13 & Percentage bases uncovered$^{a}$  & 0.40$_{\,0.5}$ & 0.00$_{\,0.83}$ & 0.63$_{\,0.32}$ & 0.81$_{\,0.17}$ & 0.74$_{\,0.22}$ & 0.49$_{\,0.43}$ & 1.00$_{\,0.01}$ & 0.80$_{\,0.17}$ & 1.00$_{\,0.01}$ & 0.71$_{\,0.25}$ \\14 & Number of ambiguous bases  & 0.77$_{\,30990}$ & 0.00$_{\,110723}$ & 0.84$_{\,24282}$ & 0.93$_{\,15259}$ & 0.90$_{\,18025}$ & 0.82$_{\,25851}$ & 0.97$_{\,10489}$ & 1.00$_{\,7801}$ & 0.97$_{\,10674}$ & 0.88$_{\,20302}$ \\\midrule
\multicolumn{11}{l}{\texttt{DETONATE}}\\ 
\midrule
15 & Nucleotide F1  & 0.61$_{\,0.57}$ & 0.00$_{\,0.22}$ & 0.81$_{\,0.68}$ & 0.90$_{\,0.73}$ & 0.86$_{\,0.71}$ & 0.67$_{\,0.6}$ & 0.99$_{\,0.78}$ & 0.41$_{\,0.45}$ & 1.00$_{\,0.79}$ & 0.83$_{\,0.7}$ \\16 & Contig F1  & 0.31$_{\,0.06}$ & 0.03$_{\,0.05}$ & 1.00$_{\,0.1}$ & 0.54$_{\,0.08}$ & 0.01$_{\,0.05}$ & 0.00$_{\,0.05}$ & 0.06$_{\,0.05}$ & 0.43$_{\,0.07}$ & 0.10$_{\,0.05}$ & 0.13$_{\,0.05}$ \\17 & KC score  & 0.94$_{\,0.88}$ & 0.72$_{\,0.74}$ & 1.00$_{\,0.92}$ & 0.50$_{\,0.6}$ & 0.84$_{\,0.82}$ & 0.85$_{\,0.82}$ & 0.51$_{\,0.6}$ & 0.00$_{\,0.26}$ & 0.77$_{\,0.77}$ & 0.79$_{\,0.79}$ \\18 & RSEM EVAL  & 0.96$_{\,-2.79}$ & 0.75$_{\,-4.62}$ & 1.00$_{\,-2.38}$ & 0.68$_{\,-5.27}$ & 0.86$_{\,-3.66}$ & 0.86$_{\,-3.65}$ & 0.44$_{\,-7.53}$ & 0.00$_{\,-11.51}$ & 0.74$_{\,-4.78}$ & 0.83$_{\,-3.91}$ \\\midrule
\multicolumn{11}{l}{\texttt{BUSCO}}\\ 
\midrule
19 & Complete BUSCOs  & 0.94$_{\,588}$ & 1.00$_{\,613}$ & 0.80$_{\,537}$ & 0.16$_{\,289}$ & 0.77$_{\,525}$ & 0.78$_{\,527}$ & 0.00$_{\,226}$ & 0.04$_{\,242}$ & 0.47$_{\,407}$ & 0.61$_{\,462}$ \\20 & Missing BUSCOs  & 0.99$_{\,26}$ & 1.00$_{\,22}$ & 1.00$_{\,21}$ & 0.74$_{\,109}$ & 0.98$_{\,28}$ & 0.98$_{\,29}$ & 0.65$_{\,142}$ & 0.00$_{\,364}$ & 0.90$_{\,55}$ & 0.99$_{\,26}$ \\\midrule
\multicolumn{2}{l}{\textbf{Summed up metric (0,1)-score}}&14.16&9.83&15.68&11.4&12.47&11.88&10.31&7.51&13.22&12.32\\\bottomrule
\multicolumn{11}{l}{$^{a}$\begin{scriptsize}Not available for the \emph{E.~coli} and \emph{A.~thaliana} data set because this metric is only calculated by \texttt{TransRate} in the case of paired-end data.\end{scriptsize}}\\\end{tabular}\end{scriptsize}
%\end{sidewaystable}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{landscape}

\end{document}