<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html>
  <head>

    <style type='text/css'>
  	details{
		cursor:pointer;
		}
    </style>    
    
    <title>The dark art of de novo transcriptome assembly: a comprehensive comparison</title>
    <link rel="stylesheet" href="../../../../src/css/style.css" type="text/css" media="print, projection, screen" />
    <link rel="stylesheet" href="../../../../src/css/jq.css" type="text/css" media="print, projection, screen" />
    
    <script type="text/javascript" src="../../../../src/jquery/jquery-latest.js"></script> 
    <script type="text/javascript" src="../../../../src/jquery/jquery.tablesorter.js"></script>
    <script type="text/javascript" src="../../../../src/jquery/jquery.tablesorter.staticrow.min.js"></script>

    <script src="http://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.14/require.min.js"></script> <!--NYAPLOT-->

		<script type="text/javascript" src="src/details-shim-master/details-shim.min.js">
		</script><link rel="stylesheet" type="text/css" href="src/details-shim-master/details-shim.min.css">
    
    <script type="text/javascript" language="javascript" class="init">
   if(window['d3'] === undefined ||
      window['Nyaplot'] === undefined){
       var path = {"d3":"http://d3js.org/d3.v3.min","downloadable":"http://cdn.rawgit.com/domitry/d3-downloadable/master/d3-downloadable"};

       var shim = {"d3":{"exports":"d3"},"downloadable":{"exports":"downloadable"}};

       require.config({paths: path, shim:shim});

   require(['d3'], function(d3){window['d3']=d3;console.log('finished loading d3');require(['downloadable'], function(downloadable){window['downloadable']=downloadable;console.log('finished loading downloadable');

	var script = d3.select("head")
	    .append("script")
	    .attr("src", "http://cdn.rawgit.com/domitry/Nyaplotjs/master/release/nyaplot.js")
	    .attr("async", true);

	script[0][0].onload = script[0][0].onreadystatechange = function(){

	    var event = document.createEvent("HTMLEvents");
	    event.initEvent("load_nyaplot",false,false);
	    window.dispatchEvent(event);
	    console.log('Finished loading Nyaplotjs');
	};
});});
}

	$(function() {
	  $('a[href*=#]:not([href=#])').click(function() {
	    if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
	      var target = $(this.hash);
	      target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
	      if (target.length) {
	        $('html,body').animate({
	          scrollTop: target.offset().top
	        }, 500);
	        return false;
	      }
	    }
	  });
	});

$(document).ready(function() {

		var headers = {},
		$table = $('#samplesTab'), i,
		l = $table.find('thead th').length,
		sortcolumns = [];
		for (i = 0; i < l; i++) {
			if ($.inArray(i, sortcolumns) < 0) {
				headers[i] = { sorter: false }
			}
		}
		$("#samplesTab").tablesorter({widgets: ['zebra']});
		$("#shortreportTab1").tablesorter({widgets: ['zebra']});
$("#shortreportTab2").tablesorter({widgets: ['zebra']});
$("#sensitivityTab1").tablesorter({widgets: ['zebra']});
$("#specificityTab1").tablesorter({widgets: ['zebra']});
$("#plotsTab1").tablesorter({widgets: ['zebra']});
<!--PLACEHOLDER-->
});
</script>
</head>
<body>
<div id="banner">
<h1 id="top"><em>De novo transcriptome assembly</em>: A comprehensive cross-species comparison of<br>short-read RNA-Seq assemblers</h1>
<h2>Supplementary Data</b>
<h3>Martin H&ouml;lzer and Manja Marz</h3>
<a href="../../../index.html#quast">Back to main supplement page</a>
</div>


<div id="main"><a name="contents"></a>
<h1>RNA-Quast contents for <i>Homo sapiens simulated</i></h1>
<ul><li>S5.1: <a href="#shortreport">Short report for <i>Homo sapiens simulated</i> assemblies</a></li> <li>S5.2: <a href="#sensitivity">Assembly completeness (sensitivity, true positive rate) for <i>Homo sapiens simulated</i> assemblies</a></li> <li>S5.3: <a href="#specificity">Assembly specificity (true negative rate) for <i>Homo sapiens simulated</i> assemblies</a></li> <li>S5.4: <a href="#plots">Various comparison and single plots for <i>Homo sapiens simulated</i> assemblies</a></li> </ul><br><br><a name="shortreport">
<h1>S5.1: Short report for <i>Homo sapiens simulated</i> assemblies</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><table id="shortreportTab1" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="100%">
<thead>
<tr><th>METRICS/TRANSCRIPTS</th><th>Genes</th><th>Avg. number of exons per isoform</th><th>Transcripts</th><th>Transcripts > 500 bp</th><th>Transcripts > 1000 bp</th><th>Aligned</th><th>Uniquely aligned</th><th>Multiply aligned</th><th>Unaligned</th><th>Avg. aligned fraction</th><th>Avg. alignment length</th><th>Avg. mismatches per transcript</th><th>Misassemblies</th><th>Database coverage</th></tr><tbody>
<tr><td>Trinity</td><td>3119</td><td>6.99</td><td>13484</td><td>10717</td><td>8451</td><td>13454</td><td>13214</td><td>12</td><td>30</td><td>0.989</td><td>2261.332</td><td>1.231</td><td>139</td><td>0.461</td></tr>
<tr><td>Trans-ABySS</td><td>3119</td><td>6.99</td><td>25272</td><td>11460</td><td>6703</td><td>25150</td><td>24896</td><td>39</td><td>122</td><td>0.994</td><td>939.312</td><td>0.459</td><td>117</td><td>0.535</td></tr>
<tr><td>Oases</td><td>3119</td><td>6.99</td><td>46469</td><td>34981</td><td>28143</td><td>46049</td><td>41196</td><td>48</td><td>420</td><td>0.959</td><td>2090.55</td><td>2.105</td><td>4094</td><td>0.592</td></tr>
<tr><td>SOAPdenovo-Trans</td><td>3119</td><td>6.99</td><td>14965</td><td>6270</td><td>4263</td><td>14792</td><td>14692</td><td>18</td><td>173</td><td>0.994</td><td>1009.08</td><td>0.308</td><td>49</td><td>0.406</td></tr>
<tr><td>IDBA-Tran</td><td>3119</td><td>6.99</td><td>12151</td><td>5642</td><td>2740</td><td>12134</td><td>12099</td><td>11</td><td>17</td><td>0.998</td><td>859.966</td><td>0.096</td><td>8</td><td>0.382</td></tr>
<tr><td>Bridger</td><td>3119</td><td>6.99</td><td>10457</td><td>7180</td><td>5167</td><td>10307</td><td>9537</td><td>7</td><td>150</td><td>0.983</td><td>1581.299</td><td>1.652</td><td>533</td><td>0.346</td></tr>
<tr><td>BinPacker</td><td>3119</td><td>6.99</td><td>13415</td><td>9858</td><td>7424</td><td>13252</td><td>12174</td><td>10</td><td>163</td><td>0.981</td><td>1755.155</td><td>1.849</td><td>785</td><td>0.372</td></tr>
<tr><td>Shannon</td><td>3119</td><td>6.99</td><td>7255</td><td>3729</td><td>2341</td><td>7196</td><td>7035</td><td>7</td><td>59</td><td>0.996</td><td>1061.013</td><td>0.394</td><td>66</td><td>0.189</td></tr>
<tr><td>rnaSPAdes</td><td>3119</td><td>6.99</td><td>11766</td><td>9116</td><td>6053</td><td>11744</td><td>11306</td><td>7</td><td>22</td><td>0.968</td><td>1649.147</td><td>1.82</td><td>351</td><td>0.431</td></tr>
<tr><td>SPAdes
</td><td>3119
</td><td>6.99
</td><td>10537
</td><td>4309
</td><td>2623
</td><td>10503
</td><td>10347
</td><td>87
</td><td>34
</td><td>0.995
</td><td>979.981
</td><td>0.215
</td><td>50
</td><td>0.367
</td></tr>
</tbody></table>

<table id="shortreportTab2" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="100%">
<thead>
<tr><th>METRICS/TRANSCRIPTS</th><th>50%-assembled genes</th><th>95%-assembled genes</th><th>50%-covered genes</th><th>95%-covered genes</th><th>50%-assembled isoforms</th><th>95%-assembled isoforms</th><th>50%-covered isoforms</th><th>95%-covered isoforms</th><th>Mean isoform coverage</th><th>Mean isoform assembly</th><th>Predicted genes</th><th>50%-matched</th><th>95%-matched</th><th>Unannotated</th><th>Mean fraction of transcript matched</th></tr><tbody>
<tr><td>Trinity</td><td>2478</td><td>1915</td><td>2644</td><td>2096</td><td>4654</td><td>2898</td><td>5003</td><td>3254</td><td>0.806</td><td>0.759</td><td>6829</td><td>12797</td><td>5462</td><td>96</td><td>0.864</td></tr>
<tr><td>Trans-ABySS</td><td>2680</td><td>2074</td><td>2903</td><td>2485</td><td>5992</td><td>2877</td><td>6966</td><td>3785</td><td>0.674</td><td>0.602</td><td>8600</td><td>24508</td><td>16312</td><td>82</td><td>0.927</td></tr>
<tr><td>Oases</td><td>2623</td><td>1865</td><td>2812</td><td>2052</td><td>6106</td><td>2995</td><td>6780</td><td>3525</td><td>0.737</td><td>0.682</td><td>21274</td><td>38562</td><td>16512</td><td>585</td><td>0.84</td></tr>
<tr><td>SOAPdenovo-Trans</td><td>2365</td><td>1380</td><td>2719</td><td>1755</td><td>3630</td><td>1679</td><td>4399</td><td>2145</td><td>0.606</td><td>0.531</td><td>3567</td><td>14251</td><td>9194</td><td>24</td><td>0.907</td></tr>
<tr><td>IDBA-Tran</td><td>2494</td><td>570</td><td>2733</td><td>1158</td><td>3896</td><td>577</td><td>4760</td><td>1203</td><td>0.591</td><td>0.512</td><td>4599</td><td>12072</td><td>8280</td><td>6</td><td>0.934</td></tr>
<tr><td>Bridger</td><td>2247</td><td>1353</td><td>2427</td><td>1527</td><td>3362</td><td>1570</td><td>3725</td><td>1786</td><td>0.655</td><td>0.61</td><td>4400</td><td>9287</td><td>4556</td><td>34</td><td>0.884</td></tr>
<tr><td>BinPacker</td><td>2300</td><td>1402</td><td>2480</td><td>1579</td><td>3701</td><td>1660</td><td>4086</td><td>1907</td><td>0.659</td><td>0.614</td><td>6344</td><td>11761</td><td>5451</td><td>62</td><td>0.87</td></tr>
<tr><td>Shannon</td><td>1441</td><td>854</td><td>1621</td><td>954</td><td>1920</td><td>1056</td><td>2183</td><td>1181</td><td>0.658</td><td>0.599</td><td>2598</td><td>7005</td><td>4606</td><td>8</td><td>0.931</td></tr>
<tr><td>rnaSPAdes</td><td>2606</td><td>1760</td><td>2686</td><td>2025</td><td>4557</td><td>2220</td><td>4848</td><td>2596</td><td>0.717</td><td>0.678</td><td>6088</td><td>10955</td><td>5031</td><td>37</td><td>0.875</td></tr>
<tr><td>SPAdes
</td><td>2537
</td><td>1566
</td><td>2624
</td><td>1772
</td><td>3668
</td><td>1625
</td><td>4054
</td><td>1865
</td><td>0.576
</td><td>0.538
</td><td>2863
</td><td>10295
</td><td>7030
</td><td>44
</td><td>0.919
</td></tr>
</tbody></table><br><br><br>

<a name="sensitivity">
<h1>S5.2: Assembly completeness (sensitivity, true positive rate) for <i>Homo sapiens simulated</i> assemblies</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p><b>From the <a target="_blank" href="http://spades.bioinf.spbau.ru/rnaquast/release1.4.0/manual.html#sec4.1">RNA-Quast manual page</a>:</b> "Assembly completeness (sensitivity). For the following metrics (calculated with reference genome and gene database) rnaQUAST attempts to select best-matching database isoforms for every transcript. Note that a single transcript can contribute to multiple isoforms in the case of, for example, paralogous genes or genomic repeats. At the same time, an isoform can be covered by multiple transcripts in the case of fragmented assembly or duplicated transcripts in the assembly." For further details see the RNA-Quast manual.</p><table id="sensitivityTab1" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="100%">
<thead>
<tr><th>METRICS/TRANSCRIPTS</th><th>Database coverage</th><th>Duplication ratio</th><th>Avg. number of transcripts mapped to one isoform</th><th>50%-assembled genes</th><th>95%-assembled genes</th><th>50%-covered genes</th><th>95%-covered genes</th><th>50%-assembled isoforms</th><th>95%-assembled isoforms</th><th>50%-covered isoforms</th><th>95%-covered isoforms</th><th>50%-assembled exons</th><th>95%-assembled exons</th><th>Mean isoform assembly</th><th>Mean isoform coverage</th><th>Mean exon coverage</th><th>Avg. percentage of isoform 50%-covered exons</th><th>Avg. percentage of isoform 95%-covered exons</th><th>GeneMarkS-T genes</th></tr><tbody>
<tr><td>Trinity</td><td>0.461</td><td>2.154</td><td>2.175</td><td>2478</td><td>1915</td><td>2644</td><td>2096</td><td>4654</td><td>2898</td><td>5003</td><td>3254</td><td>25771</td><td>24145</td><td>0.759</td><td>0.806</td><td>0.965</td><td>0.798</td><td>0.721</td><td>6829</td></tr>
<tr><td>Trans-ABySS</td><td>0.535</td><td>1.572</td><td>2.387</td><td>2680</td><td>2074</td><td>2903</td><td>2485</td><td>5992</td><td>2877</td><td>6966</td><td>3785</td><td>31540</td><td>27974</td><td>0.602</td><td>0.674</td><td>0.938</td><td>0.666</td><td>0.551</td><td>8600</td></tr>
<tr><td>Oases</td><td>0.592</td><td>4.854</td><td>4.527</td><td>2623</td><td>1865</td><td>2812</td><td>2052</td><td>6106</td><td>2995</td><td>6780</td><td>3525</td><td>29821</td><td>26567</td><td>0.682</td><td>0.737</td><td>0.945</td><td>0.733</td><td>0.618</td><td>21274</td></tr>
<tr><td>SOAPdenovo-Trans</td><td>0.406</td><td>1.274</td><td>1.952</td><td>2365</td><td>1380</td><td>2719</td><td>1755</td><td>3630</td><td>1679</td><td>4399</td><td>2145</td><td>24266</td><td>21384</td><td>0.531</td><td>0.606</td><td>0.914</td><td>0.57</td><td>0.449</td><td>3567</td></tr>
<tr><td>IDBA-Tran</td><td>0.382</td><td>1.013</td><td>1.504</td><td>2494</td><td>570</td><td>2733</td><td>1158</td><td>3896</td><td>577</td><td>4760</td><td>1203</td><td>24449</td><td>20346</td><td>0.512</td><td>0.591</td><td>0.923</td><td>0.523</td><td>0.381</td><td>4599</td></tr>
<tr><td>Bridger</td><td>0.346</td><td>1.507</td><td>1.643</td><td>2247</td><td>1353</td><td>2427</td><td>1527</td><td>3362</td><td>1570</td><td>3725</td><td>1786</td><td>21962</td><td>19881</td><td>0.61</td><td>0.655</td><td>0.933</td><td>0.614</td><td>0.497</td><td>4400</td></tr>
<tr><td>BinPacker</td><td>0.372</td><td>1.949</td><td>1.936</td><td>2300</td><td>1402</td><td>2480</td><td>1579</td><td>3701</td><td>1660</td><td>4086</td><td>1907</td><td>23036</td><td>20812</td><td>0.614</td><td>0.659</td><td>0.935</td><td>0.624</td><td>0.51</td><td>6344</td></tr>
<tr><td>Shannon</td><td>0.189</td><td>1.437</td><td>2.074</td><td>1441</td><td>854</td><td>1621</td><td>954</td><td>1920</td><td>1056</td><td>2183</td><td>1181</td><td>10637</td><td>9225</td><td>0.599</td><td>0.658</td><td>0.904</td><td>0.633</td><td>0.503</td><td>2598</td></tr>
<tr><td>rnaSPAdes</td><td>0.431</td><td>1.535</td><td>1.698</td><td>2606</td><td>1760</td><td>2686</td><td>2025</td><td>4557</td><td>2220</td><td>4848</td><td>2596</td><td>25828</td><td>23901</td><td>0.678</td><td>0.717</td><td>0.953</td><td>0.688</td><td>0.608</td><td>6088</td></tr>
<tr><td>SPAdes</td><td>0.367</td><td>1.012</td><td>1.405</td><td>2537</td><td>1566</td><td>2624</td><td>1772</td><td>3668</td><td>1625</td><td>4054</td><td>1865</td><td>24554</td><td>21627</td><td>0.538</td><td>0.576</td><td>0.918</td><td>0.531</td><td>0.41</td><td>2863</td></tr>
</tbody></table><br><br><br>

<a name="specificity">
<h1>S5.3: Assembly specificity (true negative rate) for <i>Homo sapiens simulated</i> assemblies</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p><b>From the <a target="_blank" href="http://spades.bioinf.spbau.ru/rnaquast/release1.4.0/manual.html#sec4.1">RNA-Quast manual page</a>:</b> "Assembly specificity. To compute the following metrics we use only transcripts that have at least one significant alignment and are not misassembled." For further details see the RNA-Quast manual.</p><table id="specificityTab1" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="100%">
<thead>
<tr><th>METRICS/TRANSCRIPTS</th><th>Unannotated</th><th>50%-matched</th><th>95%-matched</th><th>Mean fraction of transcript matched</th><th>Mean fraction of block matched</th><th>50%-matched blocks</th><th>95%-matched blocks</th><th>Matched length</th><th>Unmatched length</th></tr><tbody>
<tr><td>Trinity</td><td>96</td><td>12797</td><td>5462</td><td>0.864</td><td>0.845</td><td>0.855</td><td>0.785</td><td>25548915</td><td>4852934</td></tr>
<tr><td>Trans-ABySS</td><td>82</td><td>24508</td><td>16312</td><td>0.927</td><td>0.915</td><td>0.93</td><td>0.845</td><td>21649568</td><td>2007979</td></tr>
<tr><td>Oases</td><td>585</td><td>38562</td><td>16512</td><td>0.84</td><td>0.776</td><td>0.783</td><td>0.73</td><td>73983580</td><td>17472494</td></tr>
<tr><td>SOAPdenovo-Trans</td><td>24</td><td>14251</td><td>9194</td><td>0.907</td><td>0.882</td><td>0.894</td><td>0.814</td><td>13320772</td><td>1669839</td></tr>
<tr><td>IDBA-Tran</td><td>6</td><td>12072</td><td>8280</td><td>0.934</td><td>0.949</td><td>0.973</td><td>0.858</td><td>9968961</td><td>477422</td></tr>
<tr><td>Bridger</td><td>34</td><td>9287</td><td>4556</td><td>0.884</td><td>0.86</td><td>0.875</td><td>0.784</td><td>13405399</td><td>2221251</td></tr>
<tr><td>BinPacker</td><td>62</td><td>11761</td><td>5451</td><td>0.87</td><td>0.831</td><td>0.844</td><td>0.767</td><td>18651818</td><td>3621382</td></tr>
<tr><td>Shannon</td><td>8</td><td>7005</td><td>4606</td><td>0.931</td><td>0.912</td><td>0.93</td><td>0.832</td><td>6996209</td><td>526628</td></tr>
<tr><td>rnaSPAdes</td><td>37</td><td>10955</td><td>5031</td><td>0.875</td><td>0.879</td><td>0.892</td><td>0.806</td><td>17015512</td><td>2185138</td></tr>
<tr><td>SPAdes</td><td>44</td><td>10295</td><td>7030</td><td>0.919</td><td>0.916</td><td>0.934</td><td>0.835</td><td>9561201</td><td>920540</td></tr>
</tbody></table><br><br><br>

<a name="plots">
<h1>S5.4: Various comparison and single plots for <i>Homo sapiens simulated</i> assemblies</h1>
<div id="scrolltop"><h3><a href="#top">back to top</a></h3></div><p>Various comparison and single plots for <i>Homo sapiens simulated</i> assemblies. For detailed descriptions of each plot please check the <b><a target="_blank" href="http://spades.bioinf.spbau.ru/rnaquast/release1.5.0/manual.html#sec4.3">RNA-Quast manual page</a>:</b></p>
<table id="plotsTab11" class="stablesorter" border="0" cellpadding="0" cellspacing="1" width="100%">
<thead><tr><th>alignment_multiplicity</th><th>mismatch_rate</th></tr></thead><tbody><tr><td><a href="comparison_output/alignment_multiplicity.png"><img src="comparison_output/alignment_multiplicity.png" width="600px"></a></br>Single plots: <a href="Trinity_output/alignment_multiplicity.png">Trinity</a> | <a href="Oases_output/alignment_multiplicity.png">Oases</a> | <a href="Trans-ABySS_output/alignment_multiplicity.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/alignment_multiplicity.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/alignment_multiplicity.png">Bridger</a> | <a href="BinPacker_output/alignment_multiplicity.png">BinPacker</a> | <a href="IDBA-Tran_output/alignment_multiplicity.png">IDBA-Tran</a> | <a href="Shannon_output/alignment_multiplicity.png">Shannon</a> | <a href="SPAdes_output/alignment_multiplicity.png">SPAdes</a> | <a href="rnaSPAdes_output/alignment_multiplicity.png">rnaSPAdes</a></td><td><a href="comparison_output/mismatch_rate.png"><img src="comparison_output/mismatch_rate.png" width="600px"></a></br>Single plots: <a href="Trinity_output/mismatch_rate.png">Trinity</a> | <a href="Oases_output/mismatch_rate.png">Oases</a> | <a href="Trans-ABySS_output/mismatch_rate.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/mismatch_rate.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/mismatch_rate.png">Bridger</a> | <a href="BinPacker_output/mismatch_rate.png">BinPacker</a> | <a href="IDBA-Tran_output/mismatch_rate.png">IDBA-Tran</a> | <a href="Shannon_output/mismatch_rate.png">Shannon</a> | <a href="SPAdes_output/mismatch_rate.png">SPAdes</a> | <a href="rnaSPAdes_output/mismatch_rate.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>x-covered_exons</th><th>blocks_per_alignment</th></tr></thead><tbody><tr><td><a href="comparison_output/x-covered_exons.png"><img src="comparison_output/x-covered_exons.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-covered_exons.png">Trinity</a> | <a href="Oases_output/x-covered_exons.png">Oases</a> | <a href="Trans-ABySS_output/x-covered_exons.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-covered_exons.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-covered_exons.png">Bridger</a> | <a href="BinPacker_output/x-covered_exons.png">BinPacker</a> | <a href="IDBA-Tran_output/x-covered_exons.png">IDBA-Tran</a> | <a href="Shannon_output/x-covered_exons.png">Shannon</a> | <a href="SPAdes_output/x-covered_exons.png">SPAdes</a> | <a href="rnaSPAdes_output/x-covered_exons.png">rnaSPAdes</a></td><td><a href="comparison_output/blocks_per_alignment.png"><img src="comparison_output/blocks_per_alignment.png" width="600px"></a></br>Single plots: <a href="Trinity_output/blocks_per_alignment.png">Trinity</a> | <a href="Oases_output/blocks_per_alignment.png">Oases</a> | <a href="Trans-ABySS_output/blocks_per_alignment.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/blocks_per_alignment.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/blocks_per_alignment.png">Bridger</a> | <a href="BinPacker_output/blocks_per_alignment.png">BinPacker</a> | <a href="IDBA-Tran_output/blocks_per_alignment.png">IDBA-Tran</a> | <a href="Shannon_output/blocks_per_alignment.png">Shannon</a> | <a href="SPAdes_output/blocks_per_alignment.png">SPAdes</a> | <a href="rnaSPAdes_output/blocks_per_alignment.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>NAx</th><th>x-assembled_exons</th></tr></thead><tbody><tr><td><a href="comparison_output/NAx.png"><img src="comparison_output/NAx.png" width="600px"></a></br>Single plots: <a href="Trinity_output/NAx.png">Trinity</a> | <a href="Oases_output/NAx.png">Oases</a> | <a href="Trans-ABySS_output/NAx.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/NAx.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/NAx.png">Bridger</a> | <a href="BinPacker_output/NAx.png">BinPacker</a> | <a href="IDBA-Tran_output/NAx.png">IDBA-Tran</a> | <a href="Shannon_output/NAx.png">Shannon</a> | <a href="SPAdes_output/NAx.png">SPAdes</a> | <a href="rnaSPAdes_output/NAx.png">rnaSPAdes</a></td><td><a href="comparison_output/x-assembled_exons.png"><img src="comparison_output/x-assembled_exons.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-assembled_exons.png">Trinity</a> | <a href="Oases_output/x-assembled_exons.png">Oases</a> | <a href="Trans-ABySS_output/x-assembled_exons.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-assembled_exons.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-assembled_exons.png">Bridger</a> | <a href="BinPacker_output/x-assembled_exons.png">BinPacker</a> | <a href="IDBA-Tran_output/x-assembled_exons.png">IDBA-Tran</a> | <a href="Shannon_output/x-assembled_exons.png">Shannon</a> | <a href="SPAdes_output/x-assembled_exons.png">SPAdes</a> | <a href="rnaSPAdes_output/x-assembled_exons.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>block_length</th><th>x-covered</th></tr></thead><tbody><tr><td><a href="comparison_output/block_length.png"><img src="comparison_output/block_length.png" width="600px"></a></br>Single plots: <a href="Trinity_output/block_length.png">Trinity</a> | <a href="Oases_output/block_length.png">Oases</a> | <a href="Trans-ABySS_output/block_length.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/block_length.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/block_length.png">Bridger</a> | <a href="BinPacker_output/block_length.png">BinPacker</a> | <a href="IDBA-Tran_output/block_length.png">IDBA-Tran</a> | <a href="Shannon_output/block_length.png">Shannon</a> | <a href="SPAdes_output/block_length.png">SPAdes</a> | <a href="rnaSPAdes_output/block_length.png">rnaSPAdes</a></td><td><a href="comparison_output/x-covered.png"><img src="comparison_output/x-covered.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-covered.png">Trinity</a> | <a href="Oases_output/x-covered.png">Oases</a> | <a href="Trans-ABySS_output/x-covered.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-covered.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-covered.png">Bridger</a> | <a href="BinPacker_output/x-covered.png">BinPacker</a> | <a href="IDBA-Tran_output/x-covered.png">IDBA-Tran</a> | <a href="Shannon_output/x-covered.png">Shannon</a> | <a href="SPAdes_output/x-covered.png">SPAdes</a> | <a href="rnaSPAdes_output/x-covered.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>x-matched</th><th>transcript_length</th></tr></thead><tbody><tr><td><a href="comparison_output/x-matched.png"><img src="comparison_output/x-matched.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-matched.png">Trinity</a> | <a href="Oases_output/x-matched.png">Oases</a> | <a href="Trans-ABySS_output/x-matched.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-matched.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-matched.png">Bridger</a> | <a href="BinPacker_output/x-matched.png">BinPacker</a> | <a href="IDBA-Tran_output/x-matched.png">IDBA-Tran</a> | <a href="Shannon_output/x-matched.png">Shannon</a> | <a href="SPAdes_output/x-matched.png">SPAdes</a> | <a href="rnaSPAdes_output/x-matched.png">rnaSPAdes</a></td><td><a href="comparison_output/transcript_length.png"><img src="comparison_output/transcript_length.png" width="600px"></a></br>Single plots: <a href="Trinity_output/transcript_length.png">Trinity</a> | <a href="Oases_output/transcript_length.png">Oases</a> | <a href="Trans-ABySS_output/transcript_length.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/transcript_length.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/transcript_length.png">Bridger</a> | <a href="BinPacker_output/transcript_length.png">BinPacker</a> | <a href="IDBA-Tran_output/transcript_length.png">IDBA-Tran</a> | <a href="Shannon_output/transcript_length.png">Shannon</a> | <a href="SPAdes_output/transcript_length.png">SPAdes</a> | <a href="rnaSPAdes_output/transcript_length.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>Nx</th><th>x-aligned</th></tr></thead><tbody><tr><td><a href="comparison_output/Nx.png"><img src="comparison_output/Nx.png" width="600px"></a></br>Single plots: <a href="Trinity_output/Nx.png">Trinity</a> | <a href="Oases_output/Nx.png">Oases</a> | <a href="Trans-ABySS_output/Nx.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/Nx.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/Nx.png">Bridger</a> | <a href="BinPacker_output/Nx.png">BinPacker</a> | <a href="IDBA-Tran_output/Nx.png">IDBA-Tran</a> | <a href="Shannon_output/Nx.png">Shannon</a> | <a href="SPAdes_output/Nx.png">SPAdes</a> | <a href="rnaSPAdes_output/Nx.png">rnaSPAdes</a></td><td><a href="comparison_output/x-aligned.png"><img src="comparison_output/x-aligned.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-aligned.png">Trinity</a> | <a href="Oases_output/x-aligned.png">Oases</a> | <a href="Trans-ABySS_output/x-aligned.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-aligned.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-aligned.png">Bridger</a> | <a href="BinPacker_output/x-aligned.png">BinPacker</a> | <a href="IDBA-Tran_output/x-aligned.png">IDBA-Tran</a> | <a href="Shannon_output/x-aligned.png">Shannon</a> | <a href="SPAdes_output/x-aligned.png">SPAdes</a> | <a href="rnaSPAdes_output/x-aligned.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>x-assembled</th><th>x-matched_blocks</th></tr></thead><tbody><tr><td><a href="comparison_output/x-assembled.png"><img src="comparison_output/x-assembled.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-assembled.png">Trinity</a> | <a href="Oases_output/x-assembled.png">Oases</a> | <a href="Trans-ABySS_output/x-assembled.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-assembled.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-assembled.png">Bridger</a> | <a href="BinPacker_output/x-assembled.png">BinPacker</a> | <a href="IDBA-Tran_output/x-assembled.png">IDBA-Tran</a> | <a href="Shannon_output/x-assembled.png">Shannon</a> | <a href="SPAdes_output/x-assembled.png">SPAdes</a> | <a href="rnaSPAdes_output/x-assembled.png">rnaSPAdes</a></td><td><a href="comparison_output/x-matched_blocks.png"><img src="comparison_output/x-matched_blocks.png" width="600px"></a></br>Single plots: <a href="Trinity_output/x-matched_blocks.png">Trinity</a> | <a href="Oases_output/x-matched_blocks.png">Oases</a> | <a href="Trans-ABySS_output/x-matched_blocks.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/x-matched_blocks.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/x-matched_blocks.png">Bridger</a> | <a href="BinPacker_output/x-matched_blocks.png">BinPacker</a> | <a href="IDBA-Tran_output/x-matched_blocks.png">IDBA-Tran</a> | <a href="Shannon_output/x-matched_blocks.png">Shannon</a> | <a href="SPAdes_output/x-matched_blocks.png">SPAdes</a> | <a href="rnaSPAdes_output/x-matched_blocks.png">rnaSPAdes</a></td></tr></tbody><thead><tr><th>alignments_per_isoform</th><th></th></tr></thead><tbody><tr><td><a href="comparison_output/alignments_per_isoform.png"><img src="comparison_output/alignments_per_isoform.png" width="600px"></a></br>Single plots: <a href="Trinity_output/alignments_per_isoform.png">Trinity</a> | <a href="Oases_output/alignments_per_isoform.png">Oases</a> | <a href="Trans-ABySS_output/alignments_per_isoform.png">Trans-ABySS</a> | <a href="SOAPdenovo-Trans_output/alignments_per_isoform.png">SOAPdenovo-Trans</a> | <a href="Bridger_output/alignments_per_isoform.png">Bridger</a> | <a href="BinPacker_output/alignments_per_isoform.png">BinPacker</a> | <a href="IDBA-Tran_output/alignments_per_isoform.png">IDBA-Tran</a> | <a href="Shannon_output/alignments_per_isoform.png">Shannon</a> | <a href="SPAdes_output/alignments_per_isoform.png">SPAdes</a> | <a href="rnaSPAdes_output/alignments_per_isoform.png">rnaSPAdes</a></td><td></td></tr></tbody></table><br><br><br>

</div></body></html>